{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import json"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "{'Aachen': {'award': 'Nominated for Best Environment Project, Overgrad;Nominated for Best Hardware, Overgrad', 'medal': 'gold'}, 'Aalto-Helsinki': {'award': 'Nominated for Best Applied Design, Overgrad', 'medal': 'sivel'}, 'AFCM-Egypt': {'award': '', 'medal': 'gold'}, 'AHUT_China': {'award': '', 'medal': ''}, 'Aix-Marseille': {'award': '', 'medal': 'gold'}, 'Amazonas_Brazil': {'award': '', 'medal': 'sivel'}, 'Amsterdam': {'award': '', 'medal': 'bronze'}, 'AQA_Unesp': {'award': '', 'medal': 'gold'}, 'Arizona_State': {'award': 'Nominated for Best Presentation, Overgrad;Nominated for Best New Basic Part, Overgrad', 'medal': 'gold'}, 'AshesiGhana': {'award': 'Chairmans Prize', 'medal': 'sivel'}, 'ASIJ_TOKYO': {'award': '', 'medal': 'bronze'}, 'ASTWS-China': {'award': '', 'medal': 'bronze'}, 'Austin_UTexas': {'award': '', 'medal': 'gold'}, 'Austin_UTexas_LASA': {'award': '', 'medal': 'bronze'}, 'Baltimore_Bio-Crew': {'award': '', 'medal': 'bronze'}, 'Berlin_diagnostX': {'award': 'Nominated for Best Presentation, Overgrad;Nominated for Best Integrated Human Practices, Overgrad;Nominated for Best Education & Public Engagement, Overgrad', 'medal': 'gold'}, 'BGIC-Union': {'award': '', 'medal': 'sivel'}, 'Bielefeld-CeBiTec': {'award': 'Best Foundational Advance Project, Overgrad;Nominated for Best Foundational Advance Project, Overgrad;Nominated for Best Software Tool, Overgrad;Nominated for Best Hardware, Overgrad;Best New Basic Part, Overgrad;Nominated for Best New Basic Part, Overgrad;Nominated for Best New Composite Part, Overgrad;Nominated for Best Part Collection, Overgrad', 'medal': 'gold'}, 'Bilkent-UNAMBG': {'award': '', 'medal': 'bronze'}, 'BIT': {'award': '', 'medal': 'gold'}, 'BIT-China': {'award': '', 'medal': 'sivel'}, 'BNDS_China': {'award': 'Nominated for Best Wiki, High_school;Nominated for Best Presentation, High_school;Nominated for Best Integrated Human Practices, High_school;Nominated for Best Education & Public Engagement, High_school;Nominated for Best Model, High_school', 'medal': 'gold'}, 'BNU-China': {'award': '', 'medal': 'bronze'}, 'BOKU-Vienna': {'award': '', 'medal': 'sivel'}, 'Bordeaux': {'award': '', 'medal': 'bronze'}, 'BostonU': {'award': '', 'medal': 'sivel'}, 'BostonU_HW': {'award': 'Best Hardware Project;Nominated for Best Hardware Project', 'medal': 'gold'}, 'Botchan_Lab_Tokyo': {'award': '', 'medal': 'bronze'}, 'Bristol': {'award': '', 'medal': 'bronze'}, 'British_Columbia': {'award': 'Nominated for Best Environment Project;Nominated for Best Model', 'medal': 'sivel'}, 'Bulgaria': {'award': '', 'medal': 'sivel'}, 'Cadets2Vets': {'award': '', 'medal': 'bronze'}, 'Calgary': {'award': 'Nominated for Best Manufacturing Project', 'medal': 'gold'}, 'CAPS_Kansas': {'award': '', 'medal': ''}, 'Cardiff_Wales': {'award': 'Nominated for Best Therapeutics Project', 'medal': ''}, 'CCA_San_Diego': {'award': 'Nominated for Best Presentation, High_school;Best Poster, High_school;Nominated for Best Poster, High_school;Nominated for Best New Composite Part, High_school;Best Part Collection, High_school;Nominated for Best Part Collection, High_school', 'medal': ''}, 'CCU_Taiwan': {'award': 'Nominated for Best Entrepreneurship', 'medal': 'sivel'}, 'CGU_Taiwan': {'award': '', 'medal': 'bronze'}, 'Chalmers-Gothenburg': {'award': '', 'medal': 'sivel'}, 'CIEI-BJ': {'award': '', 'medal': ''}, 'CIEI-China': {'award': '', 'medal': ''}, 'CLSB-UK': {'award': 'Nominated for Best Wiki, High_school;Best Integrated Human Practices, High_school;Nominated for Best Integrated Human Practices, High_school;Nominated for Best Education & Public Engagement, High_school;Nominated for Best Model, High_school;Best Software Tool, High_school;Nominated for Best Software Tool, High_school;Best Hardware, High_school;Best New Composite Part, High_school;Nominated for Best New Composite Part, High_school', 'medal': 'gold'}, 'CMUQ': {'award': '', 'medal': 'bronze'}, 'ColegioFDR_Peru': {'award': '', 'medal': ''}, 'Cologne-Duesseldorf': {'award': 'Nominated for Best Foundational Advance Project, Overgrad;Nominated for Best Presentation, Overgrad;Nominated for Best Model, Overgrad', 'medal': 'bronze'}, 'ColumbiaNYC': {'award': '', 'medal': 'sivel'}, 'Cornell': {'award': 'Best Entrepreneurship;Nominated for Best Entrepreneurship', 'medal': 'gold'}, 'CPU_CHINA': {'award': '', 'medal': 'gold'}, 'CSMU_NCHU_Taiwan': {'award': 'Nominated for Best Food & Nutrition Project;Nominated for Best Wiki;Nominated for Best Entrepreneurship', 'medal': 'gold'}, 'CSU_Fort_Collins': {'award': '', 'medal': 'bronze'}, 'CU-Boulder': {'award': '', 'medal': 'sivel'}, 'Dalhousie': {'award': '', 'medal': 'bronze'}, 'Dartmouth': {'award': '', 'medal': ''}, 'DEIAGRA': {'award': '', 'medal': 'bronze'}, 'Delaware': {'award': '', 'medal': ''}, 'DTU-Denmark': {'award': '', 'medal': 'sivel'}, 'Duke': {'award': '', 'medal': ''}, 'East_Chapel_Hill': {'award': 'Nominated for Best Poster, High_school;Best Measurement, High_school;Nominated for Best Measurement, High_school', 'medal': ''}, 'ECUST': {'award': '', 'medal': 'gold'}, 'Edinburgh_OG': {'award': 'Best Therapeutics Project, Overgrad;Nominated for Best Therapeutics Project, Overgrad;Nominated for Best Wiki, Overgrad', 'medal': 'gold'}, 'Edinburgh_UG': {'award': 'Nominated for Best Measurement', 'medal': 'gold'}, 'Emory': {'award': '', 'medal': 'bronze'}, 'EPFL': {'award': 'Nominated for Best Diagnostics Project;Nominated for Best Integrated Human Practices;Best Education & Public Engagement;Nominated for Best Education & Public Engagement;Nominated for Best Software Tool;Nominated for Best New Basic Part', 'medal': 'gold'}, 'EpiphanyNYC': {'award': '', 'medal': 'bronze'}, 'ETH_Zurich': {'award': 'Nominated for Best Therapeutics Project, Overgrad;Best Poster, Overgrad;Nominated for Best Poster, Overgrad;Best Model, Overgrad;Nominated for Best Model, Overgrad;Nominated for Best New Basic Part, Overgrad', 'medal': 'gold'}, 'Evry_Paris-Saclay': {'award': '', 'medal': 'gold'}, 'Exeter': {'award': 'Best Environment Project, Overgrad;Nominated for Best Environment Project, Overgrad;Best Applied Design, Overgrad;Nominated for Best Applied Design, Overgrad', 'medal': 'gold'}, 'FAFU-CHINA': {'award': '', 'medal': 'bronze'}, 'Florida_Atlantic': {'award': '', 'medal': 'bronze'}, 'Franconia': {'award': '', 'medal': 'sivel'}, 'Freiburg': {'award': 'Nominated for Best Wiki, Overgrad;Nominated for Best New Basic Part, Overgrad', 'medal': 'gold'}, 'FSU': {'award': '', 'medal': 'bronze'}, 'Fudan': {'award': 'Best Therapeutics Project;Nominated for Best Therapeutics Project;Nominated for Best Software Tool;Nominated for Best New Composite Part;Nominated for Best Part Collection', 'medal': 'gold'}, 'Fudan_China': {'award': '', 'medal': 'sivel'}, 'Gaston_Day_School': {'award': '', 'medal': ''}, 'Georgia_State': {'award': 'Nominated for Best Integrated Human Practices, Overgrad;Chairmans Prize', 'medal': 'sivel'}, 'Gifu': {'award': '', 'medal': 'bronze'}, 'Glasgow': {'award': 'Best Food & Nutrition Project;Nominated for Best Food & Nutrition Project;Nominated for Best Wiki;Nominated for Best Measurement;Nominated for Best Applied Design;Nominated for Best Part Collection', 'medal': 'sivel'}, 'Greece': {'award': 'Nominated for Best Therapeutics Project', 'medal': 'gold'}, 'Grenoble-Alpes': {'award': '', 'medal': 'gold'}, 'Groningen': {'award': '', 'medal': 'gold'}, 'GZHS-United': {'award': '', 'medal': 'gold'}, 'Hamburg': {'award': '', 'medal': 'sivel'}, 'Harvard': {'award': '', 'medal': 'bronze'}, 'HBUT-China': {'award': '', 'medal': 'sivel'}, 'Heidelberg': {'award': 'Best Foundational Advance Project;Nominated for Best Foundational Advance Project;Best Wiki;Nominated for Best Wiki;Best Presentation;Nominated for Best Presentation;Best Poster;Nominated for Best Poster;Best Integrated Human Practices;Nominated for Best Integrated Human Practices;Nominated for Best Model;Best Software Tool;Nominated for Best Software Tool;Nominated for Best New Basic Part;Nominated for Best New Composite Part;Nominated for Best Part Collection;Finalist;Nominated for Finalist', 'medal': 'gold'}, 'HFLS_H2Z_Hangzhou': {'award': '', 'medal': 'sivel'}, 'HFUT-China': {'award': '', 'medal': 'gold'}, 'HK_SKHLPSS': {'award': '', 'medal': 'bronze'}, 'HokkaidoU_Japan': {'award': '', 'medal': 'bronze'}, 'Hong_Kong-CUHK': {'award': 'Nominated for Best Diagnostics Project', 'medal': 'gold'}, 'Hong_Kong_HKU': {'award': '', 'medal': 'bronze'}, 'Hong_Kong_HKUST': {'award': '', 'medal': 'gold'}, 'Hong_Kong_UCCKE': {'award': '', 'medal': ''}, 'HUST-China': {'award': '', 'medal': 'gold'}, 'HZAU-China': {'award': 'Nominated for Best New Basic Part', 'medal': 'gold'}, 'ICT-Mumbai': {'award': '', 'medal': 'bronze'}, 'IISc-Bangalore': {'award': 'Nominated for Best Hardware', 'medal': 'gold'}, 'IISER-Mohali-INDIA': {'award': '', 'medal': 'bronze'}, 'IISER-Pune-India': {'award': '', 'medal': 'sivel'}, 'IIT-Madras': {'award': '', 'medal': 'sivel'}, 'IIT_Delhi': {'award': '', 'medal': 'bronze'}, 'INSA-UPS_France': {'award': 'Best Information Processing Project;Nominated for Best Information Processing Project;Nominated for Best Wiki;Nominated for Best Presentation;Nominated for Best Integrated Human Practices;Nominated for Best Education & Public Engagement;Nominated for Best Entrepreneurship;Best Applied Design;Nominated for Best Applied Design;Nominated for Best New Basic Part', 'medal': 'gold'}, 'IONIS-PARIS': {'award': 'Nominated for Best Food & Nutrition Project;Nominated for Best Education & Public Engagement, Overgrad;Nominated for Best New Composite Part, Overgrad', 'medal': 'gold'}, 'ITB_Indonesia': {'award': '', 'medal': 'bronze'}, 'iTesla-SoundBio': {'award': '', 'medal': 'bronze'}, 'Jilin_China': {'award': '', 'medal': 'sivel'}, 'JNFLS': {'award': '', 'medal': 'bronze'}, 'Judd_UK': {'award': 'Nominated for Best Wiki, High_school;Best Education & Public Engagement, High_school;Nominated for Best Education & Public Engagement, High_school;Best Model, High_school;Nominated for Best Model, High_school', 'medal': 'sivel'}, 'KAIT_JAPAN': {'award': '', 'medal': 'bronze'}, 'Kent': {'award': 'Best Poster;Nominated for Best Poster', 'medal': 'bronze'}, 'Kingsborough_NY': {'award': '', 'medal': 'bronze'}, 'Kobe': {'award': '', 'medal': 'sivel'}, 'KU_Leuven': {'award': 'Best New Application Project;Nominated for Best New Application Project', 'medal': 'bronze'}, 'KUAS_Korea': {'award': '', 'medal': 'bronze'}, 'Kyoto': {'award': '', 'medal': 'sivel'}, 'Lambert_GA': {'award': '', 'medal': 'bronze'}, 'Lanzhou': {'award': '', 'medal': 'bronze'}, 'Lethbridge': {'award': 'Nominated for Best Integrated Human Practices, Overgrad;Nominated for Best Education & Public Engagement, Overgrad;Nominated for Best Software Tool, Overgrad', 'medal': 'gold'}, 'Lethbridge_HS': {'award': '', 'medal': 'sivel'}, 'Linkoping_Sweden': {'award': '', 'medal': 'sivel'}, 'LUBBOCK_TTU': {'award': '', 'medal': 'bronze'}, 'Lund': {'award': 'Nominated for Best Model, Overgrad', 'medal': 'bronze'}, 'Macquarie_Australia': {'award': 'Best Energy Project;Nominated for Best Energy Project;Nominated for Best Entrepreneurship, Overgrad', 'medal': 'gold'}, 'Manchester': {'award': 'Nominated for Best Environment Project, Overgrad', 'medal': 'gold'}, 'ManhattanCol_Bronx': {'award': '', 'medal': 'bronze'}, 'McMaster_II': {'award': '', 'medal': 'bronze'}, 'McMasterU': {'award': '', 'medal': 'bronze'}, 'Michigan': {'award': '', 'medal': 'bronze'}, 'Michigan_Software': {'award': '', 'medal': 'gold'}, 'Mingdao': {'award': 'Nominated for Best Integrated Human Practices, High_school;Nominated for Best Education & Public Engagement, High_school;Best Applied Design, High_school;Nominated for Best Applied Design, High_school;Best New Basic Part, High_school;Nominated for Best New Composite Part, High_school', 'medal': 'gold'}, 'Minnesota': {'award': '', 'medal': ''}, 'Missouri_Rolla': {'award': '', 'medal': 'sivel'}, 'MIT': {'award': '', 'medal': 'bronze'}, 'Moscow_RF': {'award': '', 'medal': ''}, 'MSU-Michigan': {'award': '', 'medal': 'sivel'}, 'Munich': {'award': 'Best Diagnostics Project, Overgrad;Nominated for Best Diagnostics Project, Overgrad;Nominated for Best Wiki, Overgrad;Nominated for Best Presentation, Overgrad;Nominated for Best Poster, Overgrad;Nominated for Best Integrated Human Practices, Overgrad;Nominated for Best Education & Public Engagement, Overgrad;Best Model, Overgrad;Nominated for Best Model, Overgrad;Nominated for Best Measurement, Overgrad;Best Applied Design, Overgrad;Nominated for Best Applied Design, Overgrad;Best Software Tool, Overgrad;Nominated for Best Software Tool, Overgrad;Best Hardware, Overgrad;Nominated for Best Hardware, Overgrad;Nominated for Best New Composite Part, Overgrad;Nominated for Best Part Collection, Overgrad;Finalist, Overgrad;Nominated for Finalist, Overgrad', 'medal': 'gold'}, 'Nagahama': {'award': '', 'medal': 'bronze'}, 'Nanjing-China': {'award': '', 'medal': ''}, 'Nanjing_NFLS': {'award': '', 'medal': ''}, 'NAU-CHINA': {'award': '', 'medal': 'gold'}, 'NAWI_Graz': {'award': '', 'medal': 'sivel'}, 'NCKU_Tainan': {'award': 'Best Environment Project;Nominated for Best Environment Project', 'medal': 'gold'}, 'NCTU_Formosa': {'award': '', 'medal': 'gold'}, 'NEFU_China': {'award': '', 'medal': 'gold'}, 'NEU-China': {'award': '', 'medal': 'bronze'}, 'Newcastle': {'award': 'Nominated for Best Poster, Overgrad;Nominated for Best Education & Public Engagement, Overgrad;Nominated for Best Measurement, Overgrad;Nominated for Best Part Collection, Overgrad', 'medal': 'gold'}, 'NIPER-Guwahati': {'award': '', 'medal': ''}, 'NJU-China': {'award': '', 'medal': 'sivel'}, 'NKU_China': {'award': '', 'medal': 'gold'}, 'NortheasternU-Boston': {'award': '', 'medal': 'bronze'}, 'Northwestern': {'award': '', 'medal': 'sivel'}, 'NPU-China': {'award': '', 'medal': 'bronze'}, 'NTHU_Taiwan': {'award': '', 'medal': 'sivel'}, 'NTNU_Trondheim': {'award': '', 'medal': 'gold'}, 'NTU_SINGAPORE': {'award': '', 'medal': 'gold'}, 'NU_Kazakhstan': {'award': '', 'medal': 'gold'}, 'NUDT_CHINA': {'award': '', 'medal': 'gold'}, 'NUS_Singapore': {'award': '', 'medal': 'gold'}, 'NWU-CHINA': {'award': '', 'medal': 'bronze'}, 'NYMU-Taipei': {'award': '', 'medal': 'gold'}, 'NYU_Abu_Dhabi': {'award': '', 'medal': 'gold'}, 'NYU_Shanghai': {'award': '', 'medal': 'bronze'}, 'OUC-China': {'award': 'Nominated for Best New Basic Part', 'medal': 'gold'}, 'Oxford': {'award': 'Best Diagnostics Project;Nominated for Best Diagnostics Project;Nominated for Best Wiki;Nominated for Best Presentation;Nominated for Best Integrated Human Practices;Nominated for Best Model;Nominated for Best Applied Design', 'medal': 'gold'}, 'Paris_Bettencourt': {'award': '', 'medal': 'gold'}, 'PASantiago_Chile': {'award': '', 'medal': ''}, 'Pasteur_Paris': {'award': '', 'medal': 'bronze'}, 'Peking': {'award': 'Nominated for Best Information Processing Project', 'medal': 'gold'}, 'Penn': {'award': '', 'medal': ''}, 'Peshawar': {'award': '', 'medal': 'sivel'}, 'Pittsburgh': {'award': '', 'medal': 'sivel'}, 'Potsdam': {'award': '', 'medal': ''}, 'Princeton': {'award': '', 'medal': ''}, 'Purdue': {'award': '', 'medal': 'sivel'}, 'Queens_Canada': {'award': '', 'medal': 'gold'}, 'RDFZ-China': {'award': '', 'medal': 'bronze'}, 'REC-CHENNAI': {'award': '', 'medal': 'bronze'}, 'RHIT': {'award': '', 'medal': ''}, 'Rice': {'award': 'Nominated for Best Environment Project', 'medal': 'gold'}, 'RPI_Troy_NY': {'award': '', 'medal': 'bronze'}, 'SCU-WestChina': {'award': '', 'medal': 'gold'}, 'SCU_China': {'award': '', 'medal': 'gold'}, 'SCUT-China_A': {'award': '', 'medal': 'gold'}, 'SCUT-FSE-CHINA': {'award': 'Nominated for Best New Application Project', 'medal': 'gold'}, 'SDSZ-China': {'award': '', 'medal': 'sivel'}, 'SDU-Denmark': {'award': 'Nominated for Best Energy Project', 'medal': 'gold'}, 'SDU_CHINA': {'award': 'Nominated for Best Therapeutics Project', 'medal': 'bronze'}, 'SECA_NZ': {'award': 'Nominated for Best Entrepreneurship', 'medal': ''}, 'Shanghaitech': {'award': 'Nominated for Best Foundational Advance Project;Nominated for Best Education & Public Engagement;Best Hardware;Nominated for Best Hardware;Nominated for Best New Composite Part', 'medal': 'gold'}, 'Sheffield': {'award': '', 'medal': 'gold'}, 'Shenzhen_SFLS': {'award': 'Nominated for Best Wiki, High_school;Best Presentation, High_school;Nominated for Best Presentation, High_school;Nominated for Best Poster, High_school', 'medal': 'bronze'}, 'SHSBNU_China': {'award': 'Nominated for Best Presentation, High_school;Nominated for Best Model, High_school;Nominated for Best Part Collection, High_school', 'medal': 'gold'}, 'SIAT-SCIE': {'award': '', 'medal': 'bronze'}, 'SiCAU-China': {'award': '', 'medal': 'bronze'}, 'SJTU-BioX-Shanghai': {'award': '', 'medal': 'gold'}, 'SJTU-Software': {'award': '', 'medal': 'bronze'}, 'SMS_Shenzhen': {'award': 'Best Presentation, High_school;Nominated for Best Presentation, High_school;Nominated for Best Poster, High_school;Nominated for Best Integrated Human Practices, High_school;Nominated for Best Education & Public Engagement, High_school', 'medal': 'gold'}, 'SSTi-SZGD': {'award': '', 'medal': 'gold'}, 'Stanford-Brown': {'award': '', 'medal': 'bronze'}, 'Stockholm': {'award': 'Nominated for Best Entrepreneurship, Overgrad', 'medal': 'gold'}, 'Stony_Brook': {'award': '', 'medal': 'bronze'}, 'Stuttgart': {'award': '', 'medal': 'bronze'}, 'SUIS_Alpha_Shanghai': {'award': '', 'medal': 'bronze'}, 'SUSTech_Shenzhen': {'award': 'Nominated for Best Foundational Advance Project;Nominated for Best Hardware;Nominated for Best New Basic Part', 'medal': 'gold'}, 'SVCE_CHENNAI': {'award': '', 'medal': 'sivel'}, 'Sydney_Australia': {'award': 'Nominated for Best Poster, Overgrad', 'medal': 'gold'}, 'SYSU-CHINA': {'award': '', 'medal': ''}, 'SYSU-Software': {'award': 'Best Software Project;Nominated for Best Software Project;Nominated for Best Model;Nominated for Best Applied Design', 'medal': 'gold'}, 'Szeged_SA_RMG': {'award': '', 'medal': ''}, 'SZU-China': {'award': 'Best Manufacturing Project;Nominated for Best Manufacturing Project', 'medal': 'gold'}, 'Tartu_TUIT': {'award': '', 'medal': 'sivel'}, 'TAS_Taipei': {'award': 'Best Wiki, High_school;Nominated for Best Wiki, High_school;Nominated for Best Presentation, High_school;Nominated for Best Poster, High_school;Nominated for Best Integrated Human Practices, High_school;Nominated for Best Part Collection, High_school;Finalist, High_school;Nominated for Finalist, High_school', 'medal': 'gold'}, 'TCFSH_Taiwan': {'award': '', 'medal': ''}, 'Tec-Chihuahua': {'award': '', 'medal': 'gold'}, 'TecCEM': {'award': '', 'medal': 'gold'}, 'TECHNION-ISRAEL': {'award': '', 'medal': 'gold'}, 'TecMonterrey_GDA': {'award': '', 'medal': 'bronze'}, 'Tel-Hai': {'award': '', 'medal': 'bronze'}, 'Tianjin': {'award': '', 'medal': 'sivel'}, 'TJU_China': {'award': '', 'medal': ''}, 'TMMU-China': {'award': '', 'medal': 'sivel'}, 'TNCR_Korea': {'award': '', 'medal': ''}, 'TokyoTech': {'award': '', 'medal': 'gold'}, 'Tongji_China': {'award': '', 'medal': 'sivel'}, 'Toronto': {'award': '', 'medal': 'bronze'}, 'TP-CC_San_Diego': {'award': '', 'medal': ''}, 'Tsinghua': {'award': '', 'medal': 'gold'}, 'Tsinghua-A': {'award': '', 'medal': 'sivel'}, 'TU-Eindhoven': {'award': 'Nominated for Best New Application Project', 'medal': 'gold'}, 'TU_Darmstadt': {'award': '', 'medal': 'sivel'}, 'TU_Dresden': {'award': 'Nominated for Best Foundational Advance Project, Overgrad;Nominated for Best Poster, Overgrad;Nominated for Best New Basic Part, Overgrad;Nominated for Best Part Collection, Overgrad', 'medal': 'gold'}, 'TUDelft': {'award': 'Best New Application Project, Overgrad;Nominated for Best New Application Project, Overgrad;Nominated for Best Wiki, Overgrad;Best Presentation, Overgrad;Nominated for Best Presentation, Overgrad;Nominated for Best Poster, Overgrad;Best Integrated Human Practices, Overgrad;Nominated for Best Integrated Human Practices, Overgrad;Best Education & Public Engagement, Overgrad;Nominated for Best Education & Public Engagement, Overgrad;Nominated for Best Model, Overgrad;Best Measurement, Overgrad;Nominated for Best Measurement, Overgrad;Best Entrepreneurship, Overgrad;Nominated for Best Entrepreneurship, Overgrad;Nominated for Best Applied Design, Overgrad;Best New Composite Part, Overgrad;Nominated for Best New Composite Part, Overgrad;Best Part Collection, Overgrad;Nominated for Best Part Collection, Overgrad;Finalist, Overgrad;Nominated for Finalist, Overgrad', 'medal': 'gold'}, 'Tuebingen': {'award': '', 'medal': 'bronze'}, 'TUST_China': {'award': '', 'medal': 'gold'}, 'U_of_Guelph': {'award': '', 'medal': ''}, 'UAlberta': {'award': '', 'medal': 'gold'}, 'UC_San_Diego': {'award': '', 'medal': 'bronze'}, 'UCAS': {'award': '', 'medal': 'gold'}, 'UCC_Ireland': {'award': '', 'medal': 'sivel'}, 'UChicago': {'award': '', 'medal': 'bronze'}, 'UChile_Biotec': {'award': '', 'medal': ''}, 'UChile_OpenBio-CeBiB': {'award': '', 'medal': ''}, 'UCL': {'award': '', 'medal': 'gold'}, 'UCLouvain': {'award': '', 'medal': 'bronze'}, 'UConn': {'award': '', 'medal': 'bronze'}, 'UCopenhagen': {'award': '', 'medal': 'bronze'}, 'UCSC': {'award': 'Nominated for Best Manufacturing Project', 'medal': 'bronze'}, 'UESTC-China': {'award': 'Best Plant Synthetic Biology;Nominated for Best Plant Synthetic Biology', 'medal': 'gold'}, 'UFlorida': {'award': '', 'medal': ''}, 'UGA-Georgia': {'award': '', 'medal': 'bronze'}, 'UiOslo_Norway': {'award': '', 'medal': 'bronze'}, 'UIOWA': {'award': '', 'medal': 'bronze'}, 'UIUC_Illinois': {'award': '', 'medal': ''}, 'ULaVerne_Collab': {'award': '', 'medal': ''}, 'UMaryland': {'award': 'Nominated for Best Hardware', 'medal': 'bronze'}, 'UNBC-Canada': {'award': '', 'medal': 'bronze'}, 'UNC-Asheville': {'award': '', 'medal': ''}, 'UNebraska-Lincoln': {'award': '', 'medal': 'gold'}, 'UNIFI': {'award': '', 'medal': 'sivel'}, 'UNOTT': {'award': '', 'medal': 'bronze'}, 'uOttawa': {'award': '', 'medal': 'sivel'}, 'UPMC_PARIS': {'award': '', 'medal': 'bronze'}, 'Uppsala': {'award': 'Best Manufacturing Project, Overgrad;Nominated for Best Manufacturing Project, Overgrad', 'medal': 'gold'}, 'UrbanTundra_Edmonton': {'award': '', 'medal': 'bronze'}, 'US_AFRL_CarrollHS': {'award': '', 'medal': 'gold'}, 'USMA-West_Point': {'award': '', 'medal': 'bronze'}, 'USNA_Annapolis': {'award': '', 'medal': ''}, 'USP-Brazil': {'award': '', 'medal': 'gold'}, 'UST_Beijing': {'award': '', 'medal': 'bronze'}, 'USTC': {'award': 'Nominated for Best New Application Project', 'medal': 'sivel'}, 'USTC-Software': {'award': '', 'medal': 'gold'}, 'UT-Knoxville': {'award': '', 'medal': ''}, 'Utrecht': {'award': 'Nominated for Best Integrated Human Practices', 'medal': 'gold'}, 'Valencia_UPV': {'award': 'Nominated for Best Poster;Nominated for Best Applied Design;Nominated for Best Software Tool', 'medal': 'gold'}, 'Vilnius-Lithuania': {'award': 'Nominated for Best Foundational Advance Project;Nominated for Best Wiki;Nominated for Best Presentation;Nominated for Best Poster;Nominated for Best Education & Public Engagement;Best New Basic Part;Nominated for Best New Basic Part;Best New Composite Part;Nominated for Best New Composite Part;Best Part Collection;Nominated for Best Part Collection;Finalist;Nominated for Finalist', 'medal': 'gold'}, 'Virginia': {'award': 'Nominated for Best Environment Project;Nominated for Best New Basic Part', 'medal': 'gold'}, 'Wageningen_UR': {'award': 'Best Wiki, Overgrad;Nominated for Best Wiki, Overgrad;Nominated for Best Presentation, Overgrad;Nominated for Best Integrated Human Practices, Overgrad;Nominated for Best Model, Overgrad;Nominated for Best Applied Design, Overgrad;Nominated for Best Hardware, Overgrad;Nominated for Best New Composite Part, Overgrad;iGEMers Prize', 'medal': 'gold'}, 'Warwick': {'award': '', 'medal': 'bronze'}, 'Washington': {'award': '', 'medal': 'sivel'}, 'WashU_StLouis': {'award': 'Nominated for Best Environment Project;Nominated for Best Hardware', 'medal': 'gold'}, 'Waterloo': {'award': '', 'medal': ''}, 'Westminster_UK': {'award': '', 'medal': ''}, 'WHU-China': {'award': '', 'medal': 'sivel'}, 'William_and_Mary': {'award': 'Nominated for Best Foundational Advance Project;Nominated for Best Presentation;Nominated for Best Poster;Nominated for Best Education & Public Engagement;Best Model;Nominated for Best Model;Best Measurement;Nominated for Best Measurement;Nominated for Best New Composite Part;Nominated for Best Part Collection;Finalist;Nominated for Finalist', 'medal': 'gold'}, 'WLC-Milwaukee': {'award': '', 'medal': 'bronze'}, 'Worldshaper-Nanjing': {'award': '', 'medal': 'bronze'}, 'Worldshaper-Wuhan': {'award': '', 'medal': 'bronze'}, 'Worldshaper-XSHS': {'award': 'Nominated for Best New Composite Part, High_school', 'medal': 'gold'}, 'WPI_Worcester': {'award': '', 'medal': 'bronze'}, 'XJTLU-CHINA': {'award': '', 'medal': 'gold'}, 'XMU-China': {'award': '', 'medal': 'gold'}, 'York': {'award': '', 'medal': 'bronze'}, 'ZJU-China': {'award': '', 'medal': 'gold'}, 'ZJUT-China': {'award': '', 'medal': 'bronze'}}\n"
     ]
    }
   ],
   "source": [
    "award_dict = json.load(open('award_2017.json','r'))\n",
    "print(award_dict)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "team_dict = json.load(open('teams_2017_basic.json','r'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "['Aachen', 'Aalto-Helsinki', 'AFCM-Egypt', 'AHUT_China', 'Aix-Marseille', 'Amazonas_Brazil', 'Amsterdam', 'AQA_Unesp', 'Arizona_State', 'AshesiGhana', 'ASIJ_TOKYO', 'ASTWS-China', 'Austin_UTexas', 'Austin_UTexas_LASA', 'Baltimore_Bio-Crew', 'Berlin_diagnostX', 'BGIC-Union', 'Bielefeld-CeBiTec', 'Bilkent-UNAMBG', 'BIT', 'BIT-China', 'BNDS_China', 'BNU-China', 'BOKU-Vienna', 'Bordeaux', 'BostonU', 'BostonU_HW', 'Botchan_Lab_Tokyo', 'Bristol', 'British_Columbia', 'Bulgaria', 'Cadets2Vets', 'Calgary', 'CAPS_Kansas', 'Cardiff_Wales', 'CCA_San_Diego', 'CCU_Taiwan', 'CGU_Taiwan', 'Chalmers-Gothenburg', 'CIEI-BJ', 'CIEI-China', 'CLSB-UK', 'CMUQ', 'ColegioFDR_Peru', 'Cologne-Duesseldorf', 'ColumbiaNYC', 'Cornell', 'CPU_CHINA', 'CSMU_NCHU_Taiwan', 'CSU_Fort_Collins', 'CU-Boulder', 'Dalhousie', 'Dartmouth', 'DEIAGRA', 'Delaware', 'DTU-Denmark', 'Duke', 'East_Chapel_Hill', 'ECUST', 'Edinburgh_OG', 'Edinburgh_UG', 'Emory', 'EPFL', 'EpiphanyNYC', 'ETH_Zurich', 'Evry_Paris-Saclay', 'Exeter', 'FAFU-CHINA', 'Florida_Atlantic', 'Franconia', 'Freiburg', 'FSU', 'Fudan', 'Fudan_China', 'Gaston_Day_School', 'Georgia_State', 'Gifu', 'Glasgow', 'Greece', 'Grenoble-Alpes', 'Groningen', 'GZHS-United', 'Hamburg', 'Harvard', 'HBUT-China', 'Heidelberg', 'HFLS_H2Z_Hangzhou', 'HFUT-China', 'HK_SKHLPSS', 'HokkaidoU_Japan', 'Hong_Kong-CUHK', 'Hong_Kong_HKU', 'Hong_Kong_HKUST', 'Hong_Kong_UCCKE', 'HUST-China', 'HZAU-China', 'ICT-Mumbai', 'IISc-Bangalore', 'IISER-Mohali-INDIA', 'IISER-Pune-India', 'IIT-Madras', 'IIT_Delhi', 'INSA-UPS_France', 'IONIS-PARIS', 'ITB_Indonesia', 'iTesla-SoundBio', 'Jilin_China', 'JNFLS', 'Judd_UK', 'KAIT_JAPAN', 'Kent', 'Kingsborough_NY', 'Kobe', 'KU_Leuven', 'KUAS_Korea', 'Kyoto', 'Lambert_GA', 'Lanzhou', 'Lethbridge', 'Lethbridge_HS', 'Linkoping_Sweden', 'LUBBOCK_TTU', 'Lund', 'Macquarie_Australia', 'Manchester', 'ManhattanCol_Bronx', 'McMaster_II', 'McMasterU', 'Michigan', 'Michigan_Software', 'Mingdao', 'Minnesota', 'Missouri_Rolla', 'MIT', 'Moscow_RF', 'MSU-Michigan', 'Munich', 'Nagahama', 'Nanjing-China', 'Nanjing_NFLS', 'NAU-CHINA', 'NAWI_Graz', 'NCKU_Tainan', 'NCTU_Formosa', 'NEFU_China', 'NEU-China', 'Newcastle', 'NIPER-Guwahati', 'NJU-China', 'NKU_China', 'NortheasternU-Boston', 'Northwestern', 'NPU-China', 'NTHU_Taiwan', 'NTNU_Trondheim', 'NTU_SINGAPORE', 'NU_Kazakhstan', 'NUDT_CHINA', 'NUS_Singapore', 'NWU-CHINA', 'NYMU-Taipei', 'NYU_Abu_Dhabi', 'NYU_Shanghai', 'OUC-China', 'Oxford', 'Paris_Bettencourt', 'PASantiago_Chile', 'Pasteur_Paris', 'Peking', 'Penn', 'Peshawar', 'Pittsburgh', 'Potsdam', 'Princeton', 'Purdue', 'Queens_Canada', 'RDFZ-China', 'REC-CHENNAI', 'RHIT', 'Rice', 'RPI_Troy_NY', 'SCU-WestChina', 'SCU_China', 'SCUT-China_A', 'SCUT-FSE-CHINA', 'SDSZ-China', 'SDU-Denmark', 'SDU_CHINA', 'SECA_NZ', 'Shanghaitech', 'Sheffield', 'Shenzhen_SFLS', 'SHSBNU_China', 'SIAT-SCIE', 'SiCAU-China', 'SJTU-BioX-Shanghai', 'SJTU-Software', 'SMS_Shenzhen', 'SSTi-SZGD', 'Stanford-Brown', 'Stockholm', 'Stony_Brook', 'Stuttgart', 'SUIS_Alpha_Shanghai', 'SUSTech_Shenzhen', 'SVCE_CHENNAI', 'Sydney_Australia', 'SYSU-CHINA', 'SYSU-Software', 'Szeged_SA_RMG', 'SZU-China', 'Tartu_TUIT', 'TAS_Taipei', 'TCFSH_Taiwan', 'Tec-Chihuahua', 'TecCEM', 'TECHNION-ISRAEL', 'TecMonterrey_GDA', 'Tel-Hai', 'Tianjin', 'TJU_China', 'TMMU-China', 'TNCR_Korea', 'TokyoTech', 'Tongji_China', 'Toronto', 'TP-CC_San_Diego', 'Tsinghua', 'Tsinghua-A', 'TU-Eindhoven', 'TU_Darmstadt', 'TU_Dresden', 'TUDelft', 'Tuebingen', 'TUST_China', 'U_of_Guelph', 'UAlberta', 'UC_San_Diego', 'UCAS', 'UCC_Ireland', 'UChicago', 'UChile_Biotec', 'UChile_OpenBio-CeBiB', 'UCL', 'UCLouvain', 'UConn', 'UCopenhagen', 'UCSC', 'UESTC-China', 'UFlorida', 'UGA-Georgia', 'UiOslo_Norway', 'UIOWA', 'UIUC_Illinois', 'ULaVerne_Collab', 'UMaryland', 'UNBC-Canada', 'UNC-Asheville', 'UNebraska-Lincoln', 'UNIFI', 'UNOTT', 'uOttawa', 'UPMC_PARIS', 'Uppsala', 'UrbanTundra_Edmonton', 'US_AFRL_CarrollHS', 'USMA-West_Point', 'USNA_Annapolis', 'USP-Brazil', 'UST_Beijing', 'USTC', 'USTC-Software', 'UT-Knoxville', 'Utrecht', 'Valencia_UPV', 'Vilnius-Lithuania', 'Virginia', 'Wageningen_UR', 'Warwick', 'Washington', 'WashU_StLouis', 'Waterloo', 'Westminster_UK', 'WHU-China', 'William_and_Mary', 'WLC-Milwaukee', 'Worldshaper-Nanjing', 'Worldshaper-Wuhan', 'Worldshaper-XSHS', 'WPI_Worcester', 'XJTLU-CHINA', 'XMU-China', 'York', 'ZJU-China', 'ZJUT-China']\n"
     ]
    }
   ],
   "source": [
    "award_dict_keys = list(award_dict.keys())\n",
    "print(award_dict_keys)\n",
    "team_dict_keys = list(team_dict.keys())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "metadata": {},
   "outputs": [],
   "source": [
    "for team_name in award_dict_keys:\n",
    "    for team_id in team_dict_keys:\n",
    "        if team_dict[team_id]['team_name'] in team_name:\n",
    "            team_dict[team_id]['award'] = award_dict[team_name]['award']\n",
    "            team_dict[team_id]['medal'] = award_dict[team_name]['medal']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Aachen\n",
      "Aachen\n"
     ]
    }
   ],
   "source": [
    "print(team_dict[team_dict_keys[0]]['team_name'])\n",
    "print(award_dict_keys[0])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'2225': {'team_id': '2225',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2225',\n",
       "  'team_name': 'Aachen',\n",
       "  'wiki': 'http://2017.igem.org/Team:Aachen',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Salt Vault\\n',\n",
       "  'abstract': 'Decreasing fresh water availability is not only a problem for desert climate regions like North-Africa and the Middle East, but also for European countries like Germany or the Netherlands, where water pollution is mainly caused by industry.\\r\\nWe are modifying the genome of Saccharomyces cerevisiae to increase the uptake of various ions into the cell and store these inside its vacuole. This will be achieved by upregulation of native (vacuolar) ion importers and knock-out of ion exporters. Furthermore, we are expressing heterologous genes from the plant Arabidopsis thaliana in yeast to improve vacuolar ion accumulation, thus creating a microbial dustbin.\\r\\nWith this newly generated library of genetically engineered yeasts, we hope to offer a novel way of treating water contaminated by industrial processes and store pollutants in our intracellular reservoir, the Salt Vault. Then this concept can be employed by industries facing problems with high pollution in their process water.\\n',\n",
       "  'award': 'Nominated for Best Environment Project, Overgrad;Nominated for Best Hardware, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2342': {'team_id': '2342',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2342',\n",
       "  'team_name': 'Aalto-Helsinki',\n",
       "  'wiki': 'http://2017.igem.org/Team:Aalto-Helsinki',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Finland',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Porifi - Purify your pores from acne bacteria\\n',\n",
       "  'abstract': 'Acne vulgaris affects 85% of adolescents at some point of their life, influencing their self-esteem negatively. Currently, antibiotics are among the most common acne treatments. However, 50% of bacteria associated with acne are already resistant to the antibiotics used. Our project aims to develop a novel treatment for acne using an antimicrobial peptide, dermcidin, naturally found in human sweat. Dermcidin has shown to be active against Propionibacterium acnes, the bacterium associated with acne.  Patients with acne also have reduced expression of dermcidin, which suggests that it might contribute to the condition. We aim to produce dermcidin tied to a cellulose-binding domain, which would enable the immobilization of the peptide on a cellulose surface. This would allow the usage of dermcidin in various skin care products, such as hydrogel masks and exfoliation pads.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Applied Design, Overgrad',\n",
       "  'medal': 'sivel'},\n",
       " '2421': {'team_id': '2421',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2421',\n",
       "  'team_name': 'ACIBADEM_Istanbul',\n",
       "  'wiki': 'http://2017.igem.org/Team:ACIBADEM_Istanbul',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Turkey',\n",
       "  'track': None,\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': ''},\n",
       " '2217': {'team_id': '2217',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2217',\n",
       "  'team_name': 'AFCM-Egypt',\n",
       "  'wiki': 'http://2017.igem.org/Team:AFCM-Egypt',\n",
       "  'region': 'Africa',\n",
       "  'country': 'Egypt',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Knock-in of Circular RNA gene in Hepatocellular Carcinoma cells via CRISPR/Cas9\\n',\n",
       "  'abstract': ' Hepatocellular Carcinoma (HCC) is the leading cause of cancer deaths worldwide & ranked first among cancers in males and next to breast cancer among females in Egypt - based upon results of National Cancer Registry Program of Egypt-. \"Grabbing the problem from the roots\" is the best way to decently describe the use of CRISPR, a special gene editing technique that we will be using to modulate a certain circRNA and adjust its gene expression, which is down-regulated in hepatocellular carcinoma. This in consequence modifies miRNA expression thus amending the mRNA gene expression; which is the visible problem in our trials. This will lead us to adopt a novel strategy for miRNA suppression by using circRNAs. This is accomplished by utilizing a synthetic circuit to give rise to a springboard in our battle against cancer. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2459': {'team_id': '2459',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2459',\n",
       "  'team_name': 'AHUT_China',\n",
       "  'wiki': 'http://2017.igem.org/Team:AHUT_China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Information Processing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Biocompass: The Maze Runner\\n',\n",
       "  'abstract': '    Our project is based on \"The Maze Runner\", where Thomas experienced the maze in constant change. How he reached the exit can be abstracted as classics of Graph Theory. Using the theory of biology, we designed experiments for stimulation and through means we found several feasible paths. It\\'s the extension of our project in 2014 and 2016, the difference is: this project is an analogy of the maze, and can be extended to many practical fields. \\r\\n    In detail, we combined experimental design with theoretical analysis. Firstly, we used computer processing for theoretical analysis of specifics in the movie. Then, we utilize biology-related instruments and provided materials to process sequence by trail and error. Finally, we extend related methods to other practical fields.\\r\\n    The project not only has theoretical values: solving a math problem, but also bears broad application prospects, so it can be considered as an alternative for information processing.\\r\\n\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2255': {'team_id': '2255',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2255',\n",
       "  'team_name': 'Aix-Marseille',\n",
       "  'wiki': 'http://2017.igem.org/Team:Aix-Marseille',\n",
       "  'region': 'Europe',\n",
       "  'country': 'France',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'KILL XYL : We have vermin to kill\\n',\n",
       "  'abstract': 'Our project, KILL XYL will provide a treatment for the disease caused by the plant pathogen Xylella fastidiosa. This disease currently causes loss of millions of acres of crops and no cure exists.\\r\\n\\r\\nKILL XYL is an efficient and green cure that works in two steps. First, we detect early symptoms of the disease using a drone equipped with an infra-red camera. We then treat the disease with our product, designed to KILL XYL. This product contains a powerful combination of specific phage-like particles and a green cocktail optimized to break-up bacterial aggregates.\\r\\n\\r\\nOur treatent will kill the bacteria, stop disease progression and unclog the trees vessels, all without using any harmful chemicals, pesticides or GMO crops thanks to synthetic biology.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2457': {'team_id': '2457',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2457',\n",
       "  'team_name': 'Amazonas_Brazil',\n",
       "  'wiki': 'http://2017.igem.org/Team:Amazonas_Brazil',\n",
       "  'region': 'Latin America',\n",
       "  'country': 'Brazil',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'CRISPeasy: Building a standard BioBrick toolbox for bacterial genome engineering \\n',\n",
       "  'abstract': 'The building block of Synthetic Biology is to turn biological systems easier to engineer. Standardization is a fundamental key to that goal. CRISPR/Cas9 machinery paves the way for precisely edit living cell genomes. Although revolutionary, SynBio community has an obstacle: CRISPR/Cas9 protocol is superficially standardized and requires a considerable amount of wet lab work due to the multi-plasmid system. We aim to build a toolbox for one-step genome engineering based on a standard BioBrick vector. To go further and beyond, we unified human practice with pattern recognition and machine learning in order to overcome boundaries in the way of SynBio advancement. We also bring the concept of computational repository for lab \"algorithms\", the protocols, developing an open and integrated platform to expand the iGEM experience. Our perspective is to leave a legacy and provide a bacterial genome editing machinery based on BioBrick parts assembly easy to engineer as A-B-C\\x85 CRISPR.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2385': {'team_id': '2385',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2385',\n",
       "  'team_name': 'Amsterdam',\n",
       "  'wiki': 'http://2017.igem.org/Team:Amsterdam',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Netherlands',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Photosynthetic magic: Producing fumarate out of thin air using cyanobacterial cell factories\\n',\n",
       "  'abstract': 'Irrespective of where you come from, we all share the global responsibility of ensuring that our societies are sustainable. We have been depleting the world\\x92s resources and filling the atmosphere with abnormal levels of CO2 for too long. Our team has decided to take on this challenge by creating photosynthetic cell factories, to directly use the pollutant CO2 as a resource for synthesizing fumarate - a versatile chemical traditionally manufactured from petroleum. We aim to stably produce, sense and export fumarate under conditions mimicking industrial settings. Stable production is achieved by activating and evolving different metabolic modules in response to natural day/night cycles. To detect fumarate, we developed a biosensor suitable for high-throughput screening. Finally, to allow optimal fumarate export, we investigate its transport mechanism. These efforts attracted attention from beyond academia, as our cell factories may help in taking CO2 out of thin air and into a bio-based economy.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2270': {'team_id': '2270',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2270',\n",
       "  'team_name': 'AQA_Unesp',\n",
       "  'wiki': 'http://2017.igem.org/Team:AQA_Unesp',\n",
       "  'region': 'Latin America',\n",
       "  'country': 'Brazil',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Insubiota: an alternative treatment to type 1 diabetes using genetically engineered probiotic\\n',\n",
       "  'abstract': 'Our project was inspired on the alarming and increasing number of diabetic people, especially diabetes mellitus type 1 patients, who are insulin dependent. The lack of less invasive treatments has motivated us to develop a new treatment based on the probiotic bacteria Lactococcus lactis, that was engineered to produce a single-chain analog insulin in human diabetic\\x92s microbiota. The bacteria will be able to produce the insulin associated with a secretion signal sequence and cell-penetrating peptides, to ensure its uptake. Moreover, the synthesis of the insulin will be controlled by the natural bacteria system of catabolite repression with regulation by a sRNA. At the presence of glucose, the insulin gene expression will be activated, and then, it will be ready to be secreted and absorbed, reaching the blood and performing its biological function. The final product could be a fermented milk or a lyophilized that could be easily ingested by patients.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2357': {'team_id': '2357',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2357',\n",
       "  'team_name': 'Arizona_State',\n",
       "  'wiki': 'http://2017.igem.org/Team:Arizona_State',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'EVR-QST: Engineering Variable Regulators for a Quorum Sensing Toolbox\\n',\n",
       "  'abstract': 'Homoserine lactones (HSLs) are a family of small molecules used to coordinate behavior in some bacteria. Group regulation is known as quorum sensing (QS) and is responsible for behaviors such as bacterial virulence, growth, and bioluminescence. Genetic components from QS systems have been modularized by synthetic biologists and incorporated into synthetic circuits. A significant hurdle to using HSLs in synthetic systems is cross-reactivity between a sender and non-target receivers. Crosstalk can be mitigated by orthogonal systems that do not communicate with each other. Our project characterizes interactions between HSL-producing sender cells and our newly developed receivers (LasR, TraR, and RpaR) using an HSL-induced GFP reporter system. We expect to add 3 new well-characterized receivers to build upon foundational advances such as the work with LuxR (BBa_F2620) by B. Canton. Our work also builds upon important research in biosafety (from our 2016 project) by using ethyl alcohol eliminate HSL bioactivity.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Presentation, Overgrad;Nominated for Best New Basic Part, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2252': {'team_id': '2252',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2252',\n",
       "  'team_name': 'AshesiGhana',\n",
       "  'wiki': 'http://2017.igem.org/Team:AshesiGhana',\n",
       "  'region': 'Africa',\n",
       "  'country': 'Ghana',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Gold FEDS: Fluorescent Emitting Devices \\n',\n",
       "  'abstract': 'The objective of our project  is to develop a bio-sensor for gold quantification providing a non-toxic approach for small scale mining to extract gold from refractory ore, and enabeling  routine monitoring of the ore before a mining endeavour is undertaken.  We will engineer E. coli  with a gold specific FRET probe using a donor part which is made up of a gold binding protein (golB) attached to a green fluorescent protein (nowGFP) and an acceptor part  made up of golB a red fluorescent protein (mRuby2).  In the presence of free gold, the two parts will be in close proximity and thus an energy transfer can take place between the donor and acceptor proteins and the red protein will be excited giving off a fluorescent signal.  Using calibration experiments we can relate the amount of fluorescence to the amount the gold present, and liberated by the organism from the ore. \\n',\n",
       "  'award': 'Chairmans Prize',\n",
       "  'medal': 'sivel'},\n",
       " '2226': {'team_id': '2226',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2226',\n",
       "  'team_name': 'ASIJ_TOKYO',\n",
       "  'wiki': 'http://2017.igem.org/Team:ASIJ_TOKYO',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Japan',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Promoting CRC Detection\\n',\n",
       "  'abstract': 'Colorectal cancer is the second most lethal cancer in the United States, often beginning as benign polyps in the colon and rectum. Despite ease of treatment, cases of CRC are usually detected in its late stages, rendering care difficult. CRC results from the mutation of multiple genes involved with the regulation of cell proliferation and DNA repair, with the ASIJ iGEM team focusing on the Wnt pathway. The activation of the Wnt pathway inhibits the degradation of beta-catenin, a protein that triggers the mutation of oncogenes and tumor suppressor genes. Through detection of these mutated genes and subsequent downstream proteins, our team aims to develop an early screening method that can be adapted to a home detection kit. Building off of a rapamycin induced split-luciferase system characterized by the 2015 Peking iGEM team, our construct consists of a promoter reporter system that looks at two downstream products, C-myc and COX-2. \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2509': {'team_id': '2509',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2509',\n",
       "  'team_name': 'ASTWS-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:ASTWS-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Smart E.coli for B Cells Capture\\n',\n",
       "  'abstract': 'Antibodies are Y-shaped proteins secreted by B lymphocytes that specifically recognize and neutralize antigens. Since the first monoclonal antibody was produced by hybridoma technology in 1975, it has become an important tool in both research and health care.\\r\\nHere we developed a smart E.coli system for capturing B lymphocytes by utilizing the high affinity interaction between antigen and antibody. A peptide fused with RSA and GFP is expressed and purified as antigens for immunization and Elisa assay respectively, and this target peptide and an affinity tag are co-expressed on the surface of E.coli. Coupled with anti-tag magnetic beads, it can be used to screen and capture the desired B cells that secret the corresponding antibodies. These B cells can be fused or gene cloned for antibody producing.\\r\\nThis system can also be applied to many aspects of enormous value, such as vaccine production and immune T-cell or tumor cell separation.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2253': {'team_id': '2253',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2253',\n",
       "  'team_name': 'Austin_UTexas',\n",
       "  'wiki': 'http://2017.igem.org/Team:Austin_UTexas',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Yo GABA GABA\\n',\n",
       "  'abstract': 'Since the discovery of the microbiota-gut-brain axis, there has been interest in using foods containing genetically engineered probiotics as alternative forms of medicine. Our project is aimed at designing a probiotic (specifically Lactobacillus plantarum) capable of producing high levels of gamma-aminobutyric acid (GABA) to treat patients with anxiety and bowel disorders. To make our probiotic overproduce GABA, we intend to overexpress gadB, which encodes glutamate decarboxylase, the enzyme that converts L-glutamate into GABA. Overexpression of gadB will be accomplished by placing it under the control of a Lactococcus lactis constitutive promoter. The efficacy of our probiotic in the human gut will be modeled using the gut of Apis mellifera. Additionally, we have created the Broad Host Range Plasmid Kit, a genetic toolset composed of part plasmids that can be quickly assembled into cassette plasmids, which can then be used to easily genetically modify Lactobacillus plantarum and other non-model organisms.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2458': {'team_id': '2458',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2458',\n",
       "  'team_name': 'Austin_UTexas_LASA',\n",
       "  'wiki': 'http://2017.igem.org/Team:Austin_UTexas_LASA',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Regulating LevaDOPA Production\\n',\n",
       "  'abstract': 'While its causes are still unknown, Parkinson\\x92s disease is currently associated with the lack of dopamine in particular regions of the brain. Presently, one of the most common treatments for Parkinson\\x92s is the prescription of levodopa tablets, which increases dopamine levels by introducing dopamine\\x92s chemical precursor, levadopa to the body. Our project involves building DNA circuits that produce and regulate levadopa using bacteria rather than pills. One plasmid focuses on the production of levadopa, E. Coli with HpaBC gene, and another regulates and monitors the first plasmid, E. Coli with a multigene assembly that includes a transcriptional factor, PP2551, which will increase levels of levadopa production when IPTG is introduced. We have a Venus fluorescent protein assembled into the sensing plasmid so that we are able to visualize and measure the amount of levadopa produced.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2436': {'team_id': '2436',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2436',\n",
       "  'team_name': 'Baltimore_Bio-Crew',\n",
       "  'wiki': 'http://2017.igem.org/Team:Baltimore_Bio-Crew',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'We\\x92re breakin\\x92 it down: degrading plastic and stigma in Baltimore through engineered E. coli\\n',\n",
       "  'abstract': 'The Baltimore BioCrew seeks to eliminate plastic pollution, specifically the pervasive issue of microplastics, through a sustainable, innovative mechanism. In comparison to other efforts, our focus is shifted from the macro to the micro by utilizing microorganisms to degrade polyethylene terephthalate (PET). Prior research has identified a Japanese bacterium, Ideonella sakaiensis, that can degrade PET plastics. We designed, synthesized and expressed two genes from I. sakaiensis encoding the enzymes, PETase and MHET, in E. coli. We are currently examining the activity of these enzymes and the ability of our engineered E. coli to degrade PET and produce benign byproducts with potential industrial uses. We plan to utilize the enzymes secreted by our engineered E. coli to degrade microplastics in controlled environments such as a bioreactor, and in doing so, ridding the Baltimore Inner Harbor of plastic pollution.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2264': {'team_id': '2264',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2264',\n",
       "  'team_name': 'Beidaqingniao',\n",
       "  'wiki': 'http://2017.igem.org/Team:Beidaqingniao',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': None,\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': ''},\n",
       " '2482': {'team_id': '2482',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2482',\n",
       "  'team_name': 'Berlin_diagnostX',\n",
       "  'wiki': 'http://2017.igem.org/Team:Berlin_diagnostX',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Wormspotter - Applying Toehold Switch Technology to Tapeworm Diagnostics\\n',\n",
       "  'abstract': 'Infections with the pork tapeworm Taenia solium present without symptoms initially, but may cause severe long-term complications like blindness and epilepsy. The disease is endemic in rural Africa, Latin America, and Asia. Currently there is no suitable diagnostic screening test available to identify patients in need of treatment. We set out to develop such a test using toehold switches that generate a visible color reaction on a paper test-strip upon binding of specific tapeworm RNA. We have established SOPs to screen toehold switches using in silico design, nested PCR, and cell-free expression. First sensor candidates show positive results and we succeeded in immobilizing our detection system on paper strips. We have organized a conference with over 150 participants, providing a forum for leaders from academia, pharmaceutical companies, NGOs, and members of the German Parliament, to raise awareness about neglected tropical diseases and to win scientific and financial support.\\n',\n",
       "  'award': 'Nominated for Best Presentation, Overgrad;Nominated for Best Integrated Human Practices, Overgrad;Nominated for Best Education & Public Engagement, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2371': {'team_id': '2371',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2371',\n",
       "  'team_name': 'BGIC-Union',\n",
       "  'wiki': 'http://2017.igem.org/Team:BGIC-Union',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'dCasentry - A guardian against Lung cancer\\n',\n",
       "  'abstract': 'Lung cancer is the most common cause of cancer death worldwide. The technique of liquid biopsy is much better than the popular biopsy as it obviates pain and complication incurred to patients by detecting various tumor markup, including short strand circulating tumor DNA (CtDNA) in the blood. However, it still requires laboratory apparatus such as PCR instrument.Thus, our project uses paired dCas9 protein links with split T7 RNA polymerase, guided by SgRNA to carry out the procedure. DCasentry not only deals with low CtDNA concentration in blood but also capable of producing various kinds of signal output. We adopt freeze-dried paper chip as our vector of detecting system to simplify its operation as well as storage. The product will be presented as a kit containing all items necessary for detection and can be widely use on the medical field. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2201': {'team_id': '2201',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2201',\n",
       "  'team_name': 'Bielefeld-CeBiTec',\n",
       "  'wiki': 'http://2017.igem.org/Team:Bielefeld-CeBiTec',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Expanding the Genetic Code\\n',\n",
       "  'abstract': 'We are exploring the application of unnatural base pairs as an expansion of the genetic code. To prevent loss of unnatural base pairs during replication, we will utilize several systems including CRISPR/Cas9. The expanded genetic code allows for the ribosomal incorporation of multiple non-canonical amino acids (ncAAs) into peptides. With their broad chemical and functional diversity, ncAAs provide a variety of promising applications including protein labeling, photocaging, structure analysis, and specific protein interactions. Therefore, our innovative toolkit for the translational incorporation of ncAAs in E. coli is a valuable contribution to iGEM. Directed evolution of tRNA/aminoacyl-tRNA synthetase pairs enables the site-specific incorporation of ncAAs into peptides. This approach results in an optimal orthogonality to the autologous translation apparatus and a high flexibility concerning the incorporation of multiple ncAAs. As proof of concept, we are developing a rapid test for prions and a new chromatography method for mild protein elution. \\n',\n",
       "  'award': 'Best Foundational Advance Project, Overgrad;Nominated for Best Foundational Advance Project, Overgrad;Nominated for Best Software Tool, Overgrad;Nominated for Best Hardware, Overgrad;Best New Basic Part, Overgrad;Nominated for Best New Basic Part, Overgrad;Nominated for Best New Composite Part, Overgrad;Nominated for Best Part Collection, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2234': {'team_id': '2234',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2234',\n",
       "  'team_name': 'Bilkent-UNAMBG',\n",
       "  'wiki': 'http://2017.igem.org/Team:Bilkent-UNAMBG',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Turkey',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'DiagNOSE Cancer - Volatile Organic Compound(VOC) Based Biosensor for Early Diagnosis of Cancer\\n',\n",
       "  'abstract': 'Cancer is one of the most common and lethal disease in the world and it is always highlighted that early diagnosis is crucial in terms of cancer treatment. It can often be invasive and painful to diagnose cancer and harmful to the body with the existing methods. At this point, breath test appears as a promising non-invasive and real-time technique which allows the monitoring of metabolic status. Volatile organic compounds(VOCs) in the exhaled breath could provide  in vitro detection, classification and discrimination of diseases and microorganisms which can alter the metabolic activities of the body. So that, concentration of VOCs in the exhaled breath changes. Due to metabolic changes that cancer causes, concentration of some specific VOCs increases in the breath of patients and our aim is to have bacteria sensed those VOCs to detect four most common cancer types: lung, breast, colorectal and prostate. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2305': {'team_id': '2305',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2305',\n",
       "  'team_name': 'BIT',\n",
       "  'wiki': 'http://2017.igem.org/Team:BIT',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'JACOB: Just A Creative assay for the detection Of Biomarkers\\n',\n",
       "  'abstract': 'To facilitate early diagnosis, we contrived a biosensor of onset-pathological biomarkers with high sensitivity, affordability, and analyte modifiability. Detection of Alpha Fetal Protein (AFP, a well-studied cancer indicator) is an exemplar demonstration of this device. Here is how it works: first, lysine is attached to DNA segment that is complementary to AFP-bonding aptamers. When AFP is bonded with aptamer, complementary chain is released; then, trypsin detaches lysine, which then enters fluorescents-producing, lysine-deficient E. coli system. Results are recorded, analyzed, visualized by independently designed device. Here are three highlights: first, biosensor (aptamer) is able to be artificially designed aimed at target biomarker; second, bioamplifier (gene knock-out, strong promoter, cycle amplifier and dual-fluorescence) realizes improvement of signal to noise ratio; third, micro-fluidic chip and fluorescence detector make it possible that the assay can work in practical. Therefore, the performance of system indicates it could be a promising assay in in-vitro diagnosis.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2368': {'team_id': '2368',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2368',\n",
       "  'team_name': 'BIT-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:BIT-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Sugar Hunter\\n',\n",
       "  'abstract': 'This story happens in a world that is parallel to ours.\\r\\nSweet receptor brothers, T1R2 and T1R3 are living in a galaxy that is under the threat of sweet monsters who want to rule the galaxy. The sweeter sweet monsters are, the more powerful they are. So the sweetness of every one of them is essential information. Although the brothers are born with the ability to swallow sweet things, they are too weak to fight with so many enemies. So they ask a battle master SC84117 who lives in planet JB486 for help. And they send their gene sequence to SC84117.\\r\\nAfter reading the sequence, SC84117 transforms into a spaceship with T1R2 and T1R3 brothers. Since then, they travel all around the galaxy to swallow sweet monsters and measure the sweetness of them. They are regard as heroes and there is a name for them, which is \\x93Sugar Hunter\\x94. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2326': {'team_id': '2326',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2326',\n",
       "  'team_name': 'BNDS_China',\n",
       "  'wiki': 'http://2017.igem.org/Team:BNDS_China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'High γ-Aminobutyric Acid Concentrated Probiotics\\n',\n",
       "  'abstract': 'γ-Aminobutyric Acid (GABA) can be found in many plants and fermented foods and plays the role of inhibitory neurotransmitter in the human brain. However, high GABA concentrated products can only be produced through the complicated procedures. Our project aims to develop GABA-enriched probiotics including species such as Lactobacillus delbrueckii subsp. Bulgaricus ATCC11482 and Escherichia coli Nissle 1917 through synthetic biology methods that may simplify the process of GABA production. In the bacterial way of GABA synthesis, two proteins, the glutamate: GABA antiporter (gadC) and the glutamate decarboxylase (gadA), form the glutamate decarboxylase (GAD) system. Though many bacteria incorporate these two genes, we select these genes from Lactobacillus brevis NCL912, a Lactic Acid Bacteria that is able to produce high amount of GABA with unique GAD sequences. In addition, we try to optimize the production of GABA in the two probiotics by adjusting the transcriptional rate between gadA and gadC.\\n',\n",
       "  'award': 'Nominated for Best Wiki, High_school;Nominated for Best Presentation, High_school;Nominated for Best Integrated Human Practices, High_school;Nominated for Best Education & Public Engagement, High_school;Nominated for Best Model, High_school',\n",
       "  'medal': 'gold'},\n",
       " '2220': {'team_id': '2220',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2220',\n",
       "  'team_name': 'BNU-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:BNU-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'MonKeYeast\\n',\n",
       "  'abstract': ' Our project aims to improve the loading capacity of yeast surface display system by displaying the microtubules and the flagellar filaments, respectively, onto the yeast surface. To start with, By using the agglutinin system, we anchored subunits of fibrous polymers which are represented here by microtubulin subunits or FilC, on the cell wall of yeast. Since the particular polymers self-assembly can take place outside cell membrane, we provide the engineered yeast with an environment rich of pre-secreted polymers\\x92 subunits. In this way, an extracellular tridimensional display system could be formed, which can enlarge the loading capacity by roughly an order of magnitudes. Moreover, we remodify the structure of FliC, substituting its D3 domain by the particular enzyme. By means of display and assemble specific combination of FilC-enzymes, the co-display ability can be greatly enhanced. As a result, a highly efficient, whole-cell biocatalyst system would be established at the meantime.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2294': {'team_id': '2294',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2294',\n",
       "  'team_name': 'BOKU-Vienna',\n",
       "  'wiki': 'http://2017.igem.org/Team:BOKU-Vienna',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Austria',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'D.I.V.E.R.T. - Directed In Vivo Evolution via Reverse Transcription\\n',\n",
       "  'abstract': 'In vivo continuous directed evolution offers significant advantages over classic in vitro methods as it drastically reduces the amount of time and actual lab work required. Most current approaches, however, are based on globally accelerated mutagenesis inevitably leading to unwanted off-target mutations. D.I.V.E.R.T. represents the concept of a new continuous in vivo evolution strategy that allows full spatial control of mutagenesis by building systems that resemble the retrotransposon life cycle: The gene of interest\\x92s mRNA is reverse transcribed in an error prone way and reintegrated into the genome at the respective locus replacing the original gene for several cycles.\\r\\n \\r\\nTo demonstrate the generalizability of the process we plan on carrying out our proof of concept experiment in yeast and E. coli. By the end of August we are still in the cloning stage. Additionally, we are working on a single plasmid system for CRISPR-enhanced chromosomal integration in E. coli.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2307': {'team_id': '2307',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2307',\n",
       "  'team_name': 'Bordeaux',\n",
       "  'wiki': 'http://2017.igem.org/Team:Bordeaux',\n",
       "  'region': 'Europe',\n",
       "  'country': 'France',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Stargate WCC\\n',\n",
       "  'abstract': '« Stargate WCC » is intended to control cell differenciation into muscular cells in the nematode C. elegans via a regulating mechanism using optogenetics and alternative splicing. We can obtain logical gates involving proteins from STAR family and the White Collar Complex which is a photoinducible system. \\r\\n\\r\\nAt the same time, a bioinformatic part has been developed in connection with scientific project. This part is intended to create a workflow to streamline the study of the alternative splicing after the use of RNAseq technology. \\r\\n\\r\\nC.elegans is currently a great model used to study many diseases through foundational research. Our project strives to give necessary tools in biomedical research connected for example to stem cells therapy, bioprinting or research about genetic diseases.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2411': {'team_id': '2411',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2411',\n",
       "  'team_name': 'BostonU',\n",
       "  'wiki': 'http://2017.igem.org/Team:BostonU',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Using Toehold Switches to Drive Recombinase-Based Logic\\n',\n",
       "  'abstract': 'The presence of specific RNAs in cells can be indicative of their state. Detecting these RNAs on a small scale allows for identification of viruses such as Zika and Ebola, however measuring larger sets of RNA remains complex. Current methods of RNA detection are time consuming and require expensive machinery and technical expertise. Toehold switches, biological devices that drive the expression of a gene when activated by trigger RNA, present an alternative with high specificity, wide dynamic range, and ease of use. We present a platform whereby trigger RNAs activate toeholds and induce recombinase expression, driving downstream genetic logic. We integrate this RNA-inducible logic within a cell-free transcription translation system to reduce the experimental burden on time and supplies. This work serves to establish functionality of RNA-inducible logic in a cell-free system, and provides a platform for future implementation in applications that require detection of multiple RNAs, including disease diagnostics.\\r\\n\\n',\n",
       "  'award': 'Best Hardware Project;Nominated for Best Hardware Project',\n",
       "  'medal': 'gold'},\n",
       " '2409': {'team_id': '2409',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2409',\n",
       "  'team_name': 'BostonU_HW',\n",
       "  'wiki': 'http://2017.igem.org/Team:BostonU_HW',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Hardware',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'MARS: Microfluidic Applications for Research in Synbio\\n',\n",
       "  'abstract': 'Microfluidics is often an underutilized technology in the field of synthetic biology because designing and using microfluidic chips requires specialized knowledge. Our project, MARS (Microfluidic Applications for Research in Synbio), aims to increase the accessibility of low-cost and easy to use microfluidic systems for researchers in synbio. MARS will comprise of designs for nine chips adapted to our low cost system that perform fundamental synthetic biology procedures, as well as video tutorials, PDFs and protocols for using these chips or fabricating your own. Furthermore, MARS includes a troubleshooting and verification framework for microfluidics allowing researchers to quantitatively grade microfluidic chip designs. Using our end to end system, microfluidics can become a more accessible and practical tool for synthetic biologists to integrate into their labs.\\n',\n",
       "  'award': 'Best Hardware Project;Nominated for Best Hardware Project',\n",
       "  'medal': 'gold'},\n",
       " '2337': {'team_id': '2337',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2337',\n",
       "  'team_name': 'Botchan_Lab_Tokyo',\n",
       "  'wiki': 'http://2017.igem.org/Team:Botchan_Lab_Tokyo',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Japan',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Amino Acid Synthesis Model from Nitrogen - The Air, New and Innovative Food\\n',\n",
       "  'abstract': 'Among some kind of nutrients, proteins are important for body formation. However, it is difficult for people in poor areas to continuously obtain protein rich foods. Therefore, in addition to these ingredients, we propose \"air\" anyone can easily take it into the body. We got the idea from Papuans living in Papua New Guinea. Despite their low-protein diets, they have muscular bodies. It is thought that nitrogen fixing bacteria in their intestines are influencing on protein nutrition. We thought to construct a pathway to synthesize amino acids from nitrogen in E.coli, introduce it into the intestine in the future. To synthesize amino acids, we first express nitrogenase to convert nitrogen to ammonia. We then express amino acid dehydrogenases to synthesize glutamate and phenylalanine from accumulated ammonia. This makes it possible to make people available to protein sources by breathing the air. This will contribute to the solution of protein shortage.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2293': {'team_id': '2293',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2293',\n",
       "  'team_name': 'Bristol',\n",
       "  'wiki': 'http://2017.igem.org/Team:Bristol',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Bristol BREATHE: BioReactor to Eliminate Atmospheric Threats to Health and the Environment\\n',\n",
       "  'abstract': 'Nitrogen oxide (NOx) gas levels in Bristol are unacceptable. Produced predominantly by car emissions, NOx causes significant morbidity and mortality; it exacerbates lung disease and in Bristol alone kills 300 people per year (8.5% of deaths). NOx is also the third most detrimental greenhouse gas. We aim to engineer E. coli to metabolise NOx using the cytochrome c nitrite reductase (NrfA), which reduces nitrite (NO2-) to ammonia, and the Nap periplasmic nitrate reductase to convert nitrate (NO3-) to nitrite. As NOx dissolves to form both nitrite and nitrate, the use of Nap will improve efficiency. The ammonia produced could be used to make fertiliser, or electricity if incorporated into a microbial fuel cell. An agent-based bacterial model will be adapted to predict system efficiency and behaviour. We aim to contain our bacteria within pods to be strategically placed in the most polluted areas, based on our atmospheric pollution modelling.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2433': {'team_id': '2433',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2433',\n",
       "  'team_name': 'British_Columbia',\n",
       "  'wiki': 'http://2017.igem.org/Team:British_Columbia',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'aGROW\\n',\n",
       "  'abstract': 'A single species of bacteria is responsible for many plant diseases - Agrobacterium tumefaciens. Infection with this bacteria causes bizarre growths and tumors on plants that repurpose the plant cell to reallocate its nutrients to the bacterial cell, causing plant death and agricultural loss. In Agrobacterium, the ability to infect a plant is contained in a tumour inducing plasmid which can spread through conjugation between cells. Here we have constructed aGROW, a strain of Agrobacterium capable of disarming pathogenic Agrobacterium and remediating infected soil. We have armed aGROW with a stable plasmid carrying a CRISPR/Cas9 system programmed to remove a key virulence region of the tumour inducing plasmid. Our plasmid employs a separate conjugation system to carry itself through a population of Agrobacterium, preventing them from becoming virulent. We envision this system to be adaptable in disarming other virulent plasmids such as those found in Shigella, Salmonella, and E. coli.\\n',\n",
       "  'award': 'Nominated for Best Environment Project;Nominated for Best Model',\n",
       "  'medal': 'sivel'},\n",
       " '2515': {'team_id': '2515',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2515',\n",
       "  'team_name': 'Bulgaria',\n",
       "  'wiki': 'http://2017.igem.org/Team:Bulgaria',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Bulgaria',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'ACE of BASE - Accelerated Crispr-based EvolutiOn For BActerial SElection\\n',\n",
       "  'abstract': 'Directed evolution has been established as an effective strategy for improving the function of biomolecules for industrial and research applications. One of the major pre-requirements for a successful directed evolution of proteins is a simple, fast and cost-efficient method for generating genetic diversity. Many technologies have been used during the years to propagate mutations. One of them is mutator strains, which carry defects in one or more of their DNA repair genes. A large number of gene knock-out, transcriptional or translational silencing methods were applied for mutator strain generation. Regardless of its great potential, the CRISPR guided dCas9 targeting to transcriptional start sites of bacterial DNA repair genes was not among them. Our project is focused on adapting this promising system to manipulate the mutation levels in E. coli in an attempt to create a novel and efficient mutator strain with controllable mutation levels and high transformation efficiency.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2248': {'team_id': '2248',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2248',\n",
       "  'team_name': 'Cadets2Vets',\n",
       "  'wiki': 'http://2017.igem.org/Team:Cadets2Vets',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Community Lab',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Affordable, Paper-based Assay For Detection Of Arsenic Contamination\\n',\n",
       "  'abstract': 'Arsenic is a toxic chemical that can be naturally occurring or a result of industrial contamination. The identification of dangerous arsenic concentrations in groundwater and soil is important because arsenic poses a serious health risk to living organisms. Unfortunately, arsenic testing is expensive, time consuming, and requires sophisticated equipment. This is especially relevant to soldiers deployed to resource-scarce, field environments. Our team is using synthetic biology to develop a more affordable, portable, and accessible means of testing. We designed a plasmid that uses ArsR, a negative transcriptional repressor, to regulate the expression of Green Fluorescent Protein (GFP). In the presence of arsenic, ArsR will undergo a conformational change that allows GFP expression. The assay is performed using a low cost paper ticket made by Edgewood Chemical Biological Center and analyzed with ultraviolet light. The development of this circuit would provide an inexpensive way to evaluate arsenic contamination.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2260': {'team_id': '2260',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2260',\n",
       "  'team_name': 'Calgary',\n",
       "  'wiki': 'http://2017.igem.org/Team:Calgary',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Astroplastic: from colon to colony\\n',\n",
       "  'abstract': 'Governments and private enterprises alike are gearing up for exploration and colonization of Mars. Two ecological and economical challenges to interplanetary travel arise: the sustainable management of waste produced on a spaceship and the astronomical cost of shipping materials to space. The iGEM Calgary 2017 team used genetically engineered E. coli (expressing genes from Ralstonia eutropha and Pseudomonas aeruginosa) to turn human waste into poly(3-hydroxybutyrate) (PHB), a bioplastic. Our engineered E. coli also secrete the PHB they produce, which improves the efficiency of the system. Our project is a start-to-finish integrated system that can be used in space to generate items useful to astronauts during early Mars missions. This will solve the problem of waste management by upcycling solid human waste into a usable product. It will also reduce astronautical costs, as fuel typically used to ship materials to space can be saved.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Manufacturing Project',\n",
       "  'medal': 'gold'},\n",
       " '2465': {'team_id': '2465',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2465',\n",
       "  'team_name': 'CAPS_Kansas',\n",
       "  'wiki': 'http://2017.igem.org/Team:CAPS_Kansas',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Manipulating Omp pores & AcrAB-TolC efflux pumps using CRISPR/dCas9 to enhance E. coli antibiotic susceptibility\\n',\n",
       "  'abstract': 'Antibiotics are a hallmark of modern medicine having saved numerous lives since the discovery of penicillin in 1928. Bacteria resistant to antibiotics emerged shortly after their initial use, and is an increasing problem as pathogenic strains of bacteria evolve resistance to multiple drugs. In an effort to increase antibiotic susceptibility, we will use CRISPR/Cas9 technologies to disrupt intrinsic resistance mechanisms in E. coli. Specifically, we plan to enhance the expression of Omp pores while also inhibiting the functioning of AcrAB-TolC efflux pumps. Mechanism for effective delivery of a functioning CRISPR system in clinical settings will be explored.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2404': {'team_id': '2404',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2404',\n",
       "  'team_name': 'Cardiff_Wales',\n",
       "  'wiki': 'http://2017.igem.org/Team:Cardiff_Wales',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'BenthBioFactory: Using plant synthetic biology to generate therapeutics for the treatment of Graves\\x92 Disease\\n',\n",
       "  'abstract': 'Our project involves expressing the human thyroid stimulating hormone antagonist (TSHantag) protein in the tobacco Nicotiana benthamiana. Using golden gate cloning, we are generating transcriptional units for transient expression of TSHantag after agrobacterium-mediated transformation in tobacco leaves. TSHantag has been used to treat hyperthyroid disorders such as Graves\\x92 Disease, but has not been produced in large amounts appropriate for therapeutic use. The TSHantag protein works by inhibiting autoimmune autoantibodies, which in turn decreases elevated thyroxine levels and reduces pathologic symptoms. \\r\\nIn addition, we are expanding the set of tools available for regulating gene expression in plant synthetic biology. Currently, only a small number of regulatory elements have been introduced into the iGEM Phytobrick standard. Therefore, we are generating novel inducible promoter Phytobricks that are responsive to jasmonic acid, salicylic acid and damage associated molecular patterns (DAMPs). The efficacy of these regulatory elements will be measured using a luciferase reporter system. \\r\\n\\n',\n",
       "  'award': 'Nominated for Best Therapeutics Project',\n",
       "  'medal': ''},\n",
       " '2491': {'team_id': '2491',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2491',\n",
       "  'team_name': 'CCA_San_Diego',\n",
       "  'wiki': 'http://2017.igem.org/Team:CCA_San_Diego',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'ImPAHct: Efficient Bacterial System for the Degradation of Polycyclic Aromatic Hydrocarbons in Crude Oil\\n',\n",
       "  'abstract': 'Crude oil is composed of polycyclic aromatic hydrocarbons (PAHs), compounds that are difficult to degrade and environmentally harmful. It is crucial to find a time and cost effective procedure that will catabolize some of the most prevalent, toxic PAHs -- fluorene, phenanthrene, and naphthalene -- to innocuous compounds, such as salicylate and phthalate, which are degradable in metabolic cycles. Certain bacteria contain intermediates in their degradative pathways that simplify gene combination and nucleotide sequencing. We proposed a novel methodology for the degradation of multiple PAHs through the implementation of these bacteria-derived pathways into E. coli. This methodology allows broad spectrum transformation of PAHs within an oil environment into safer residues. A broad host vector can be used to transfer the sequence\\x92s degradation capabilities into strains different from E. coli. These bacteria can be incorporated in oil spill remediations and bioreactor use, achieving detoxification through combinatorial genetic bioremediation.\\n',\n",
       "  'award': 'Nominated for Best Presentation, High_school;Best Poster, High_school;Nominated for Best Poster, High_school;Nominated for Best New Composite Part, High_school;Best Part Collection, High_school;Nominated for Best Part Collection, High_school',\n",
       "  'medal': ''},\n",
       " '2292': {'team_id': '2292',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2292',\n",
       "  'team_name': 'CCU_Taiwan',\n",
       "  'wiki': 'http://2017.igem.org/Team:CCU_Taiwan',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Taiwan',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': ' Carindex \\x96 Caries Index\\n',\n",
       "  'abstract': '    The WHO estimates that nearly all adults will have dental caries at some point, and the actual dental caries rate in Taiwan is 99.2%. Dental caries are easily ignored by patients since they do not hurt until they become serious. In the worst case, treatment requires tooth implantation, a painful and costly procedure. \\r\\n\\r\\n    Carindex, a system designated to evaluate the risk of dental caries through saliva, is here to solve these problems. Three parameters that correlated with dental caries: lactate level, competence stimulating peptide (CSP) concentration, pH value, will be detected in the engineered E. coli, B. subtilis and test paper, respectively. Then, machine learning will use these values to generate a prediction model. Ultimately, Carindex utilizes this model along with the detection devices to alert users to their risk of dental caries, and raise public awareness of dental caries. Expectedly, Carindex can decrease dental caries rate significantly in Taiwan.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Entrepreneurship',\n",
       "  'medal': 'sivel'},\n",
       " '2376': {'team_id': '2376',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2376',\n",
       "  'team_name': 'CGU_Taiwan',\n",
       "  'wiki': 'http://2017.igem.org/Team:CGU_Taiwan',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Taiwan',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'INKOFF: Preserve Fibers to Save Nature\\n',\n",
       "  'abstract': 'Reprocessing paper is very important for reducing deforestation, but the current manufacturing process causes chemical pollution and substantial paper fiber losses. After extensive discussions with paper-making experts, we have devised a plan to replace the chemicals used in traditional paper reprocessing and to increase recovery of paper fibers. In our model, the paper sheets undergo image analysis to determine which spots have ink printed on top and will be sprayed with Saccharomyces cerevisiae which has been engineered to express deinking enzymes upon light induction. Instead of treating all paper pulp, only the spots with ink is shone with light to induce localized deinking. This procedure can remove ink efficiently while protecting most of the paper fibers from being digested. This design extends the lifetime of the paper fibers and reduces wood-logging for making paper.\\r\\n\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2329': {'team_id': '2329',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2329',\n",
       "  'team_name': 'Chalmers-Gothenburg',\n",
       "  'wiki': 'http://2017.igem.org/Team:Chalmers-Gothenburg',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Sweden',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'BREATHtaking\\n',\n",
       "  'abstract': 'Lung cancer is the cancer form with the highest death toll. Late occurring symptoms and limited expensive detection methods leads to late detection of the disease, which is the most important reason to the high death toll.\\r\\nSpecific Volatile Organic Compounds (VOC)s are molecules that have been found in the breath of lung cancer patients. \\r\\n\\r\\nIn our project we are creating a biosensor for two of these VOCs. Receptors natively present in the membrane of yeast are replaced with xenogeneic receptors that are able to recognize the VOCs. When the VOCs are sensed the pheromone pathway is initiated and a detection system triggered, ultimately leading to that the yeast cells turns red through interruption of the ADE2 gene. \\r\\n\\r\\nThe goal of this project is to create a simple, cheap and harmless method enabling detection of lung cancer in an early stage, leading to more lives being saved. \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2281': {'team_id': '2281',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2281',\n",
       "  'team_name': 'CIEI-BJ',\n",
       "  'wiki': 'http://2017.igem.org/Team:CIEI-BJ',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Repel the mosqiutoes, no harm left\\n',\n",
       "  'abstract': 'Mosquitos are capable of transmitting severe diseases to human beings (e.g. malaria, yellow fever，and microcephaly, etc). Consequently many countries have suffered disasters caused by these epidemics, and there were roughly 1.5 million people killed in 2016. Even worse, most mosquito repellents currently used contain DEET which is detrimental to both humans and environment. Thus, our team, CIEI-BJ, decided to design a biosynthesis pathway to produce a special substance, Citronellol, which is nontoxic to human beings, environmental friendly, and can repel mosquitoes effectively. One of the most significant concern during production is to ensure that neither pollution nor toxin will be released. Therefore, our team transferred geraniol synthase gene (GES) and old yellow enzyme gene (OYE) to E.coli and yeast, then let them help us convert glucose into citronellol. Through this method, we can produce citronellol massively with high efficiency, low cost and zero toxic byproducts.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2322': {'team_id': '2322',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2322',\n",
       "  'team_name': 'CIEI-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:CIEI-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Terminator of food waste\\n',\n",
       "  'abstract': 'Disposal of food waste is a huge concern in the society. Improper disposal brings serious pollution and causes tremendous nutrition loss. Considering the high salt concentration in food waste, which greatly influences the efficiency when we use microbes to decompose the waste, our team aims to transfer salt resistant genes into the functional microbes. In this research, we decided to modify the yeast, specifically saccharomycetes cerevisiae, by inserting the E.coli glutamate synthase (gltB) gene and trehalose synthesis related genes (ScTPS1 and SpTPS1) into it, to make the yeast not only decompose the food waste efficiently but resist high salinity. These genes help produce small molecular substances (glutamate and trehalose), which enable the cells to automatically adjust the osmotic pressure in high salinity environments. In this way, we can better take care of the troublesome food waste in a more cost-effective and environmentally friendly way.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2206': {'team_id': '2206',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2206',\n",
       "  'team_name': 'CLSB-UK',\n",
       "  'wiki': 'http://2017.igem.org/Team:CLSB-UK',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Project B.A.T.M.A.N. - Biosynthetic Applications of Toehold switches - miRNAs and nonsmall cell lung cancer\\n',\n",
       "  'abstract': 'Non-small cell lung cancer (NSCLC) has a high mortality and no cheap screening test for early diagnosis. MicroRNA (miRNA) levels in body fluids are postulated to be effective, non-invasive biomarkers for many diseases. miRNAs hsa-mir-15b-5p and hsa-mir-27b-3p are differentially expressed in serum early in NSCLC. We designed two sequence specific sensors to quantify serum levels of these miRNAs using toehold switches, which regulate protein synthesis post-transcriptionally. Binding of an arbitrary RNA sequence to the toehold switch\\r\\n activates translation, producing a fluorescent reporter protein. Fluorescence intensity therefore indicates miRNA levels. This may enable miRNA level detection using toehold switches as a simple detection method for NSCLC. In future, we propose multiple switches\\r\\n used in tandem to detect differentially expressed miRNAs in multiple diseases simultaneously, or even several reporters with different emission peaks to create a multiplexing assay. This would allow for rapid and cheap diagnosis of many diseases\\r\\n from one sample.  \\n',\n",
       "  'award': 'Nominated for Best Wiki, High_school;Best Integrated Human Practices, High_school;Nominated for Best Integrated Human Practices, High_school;Nominated for Best Education & Public Engagement, High_school;Nominated for Best Model, High_school;Best Software Tool, High_school;Nominated for Best Software Tool, High_school;Best Hardware, High_school;Best New Composite Part, High_school;Nominated for Best New Composite Part, High_school',\n",
       "  'medal': 'gold'},\n",
       " '2467': {'team_id': '2467',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2467',\n",
       "  'team_name': 'CMUQ',\n",
       "  'wiki': 'http://2017.igem.org/Team:CMUQ',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Qatar',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Developing an eco-friendly approach to reduce the use of biocides for preventing Microbial Induced Corrosion.\\n',\n",
       "  'abstract': 'Sulfate Reducing Bacteria (SRB) cause significant damage to marine oil pipelines necessitating the use of biocides for reducing the Microbial Induced Corrosion (MIC) and potential for great environmental harm. Our team will focus on developing an eco-friendly approach to limit the use of biocides, which are used for targeting the SRB. SRB require high salt and low oxygen so first we plan to genetically engineer a strain of bacteria that would report the osmolarity in oil pipelines, thus act as a biosensor for salt concentrations. The ratio of salinity in the seawater flowing in versus the seawater flowing out of the well will be used to estimate SRB populations in the pipelines, thus limiting the amount of biocides added in times of high Microbial counts. In order to further reduce the environmental impact, we will inactivate the biocides prior to the oil well water being released back into the ocean.   \\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2498': {'team_id': '2498',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2498',\n",
       "  'team_name': 'ColegioFDR_Peru',\n",
       "  'wiki': 'http://2017.igem.org/Team:ColegioFDR_Peru',\n",
       "  'region': 'Latin America',\n",
       "  'country': 'Peru',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Featherase: Improving the Removal of Feather Waste in Peruvian Agriculture\\n',\n",
       "  'abstract': 'One of the most persistent, complicated problems in poultry agriculture is the removal of feather waste. Chicken feathers are mostly comprised of keratin, a strong, fibrous protein that can not be degraded by the likes of conventional proteases such as bromelain, pepsin, or papain. Failure to remove these feathers can lead to extreme health, as well as environment, related issues involving the transmission of diseases such as Avian Flu (H5N1), a disease that can wipe out chicken populations, and also has the potential for devastating effects on humans, having killed approximately 60% of all humans who have been afflicted with the disease since 1997. Our project works to develop a safe, environmentally-friendly solution to this problem involving implementing keratinases kerA & kerBPN into DH5-Alpha E. coli through transformation and assembling a prototype which allows for the degradation of the keratin in the aforementioned feathers.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2271': {'team_id': '2271',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2271',\n",
       "  'team_name': 'Cologne-Duesseldorf',\n",
       "  'wiki': 'http://2017.igem.org/Team:Cologne-Duesseldorf',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Designing a Customizable Synthetic Cell Compartment Toolbox\\n',\n",
       "  'abstract': 'Unknown, unpredictable reactions inside cells pose a major problem in synthetic biological designs as they are affecting the purposed applications. Nature solves the problems of unavoidable interactions or toxification by intermediates and by-products by separating enzymatic reactions through compartmentalization. \\r\\nThe integration of pathways into organelles leads to the concentration of enzymes and metabolites, sustains unstable intermediates and enables naturally incompatible reactions to take place simultaneously.\\r\\nWe intend to follow nature\\x92s example and engineer yeast\\x92s peroxisomes to create an intracellular space with customized properties. To do so we establish an open source toolbox and enable booting up a compartment perfectly tailored for a specific application. \\r\\nTo demonstrate the potential of this approach we relocate the nootkatone and violacein pathway into peroxisomes, modify the compartment\\x92s characteristics using our toolbox applications and thereby create an artificial compartment with optimal reaction conditions.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Foundational Advance Project, Overgrad;Nominated for Best Presentation, Overgrad;Nominated for Best Model, Overgrad',\n",
       "  'medal': 'bronze'},\n",
       " '2412': {'team_id': '2412',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2412',\n",
       "  'team_name': 'ColumbiaNYC',\n",
       "  'wiki': 'http://2017.igem.org/Team:ColumbiaNYC',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'SilenshR: Bacteria-Mediated Oncogene Silencing as Living Cancer Therapeutic\\n',\n",
       "  'abstract': 'RNA interference (RNAi) therapies modulate endogenous gene expression in target cells through introduction of exogenous short interfering RNAs (siRNA) or their precursors, short hairpin RNAs (shRNA). Challenges for efficient and cell-specific RNAi therapies abound, like rapid renal clearance, degradation by serum nucleases, traversing the lipid bilayer and escape from the intracellular endosome. Bacteria innately colonize the hypoxic and immune-privileged cores of tumors and as such have been explored as potent delivery systems for RNAi-based cancer therapeutics. We are engineering an RNAi gene therapy, utilizing recombinant E. coli that invade mammalian cells and deliver an shRNA payload targeting the aberrantly expressed receptor tyrosine kinase EGFR and transcription factor c-Myc. Bacterial uptake by mammalian cells and endosomal breakdown are mediated by a quorum-inducible Invasin-HlyA operon. We are characterizing the circuit via gentamicin protection assays in vitro using HeLa and prostate cancer lines, and assessing target oncogene knockdown through flow cytometry and qRT-PCR.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2296': {'team_id': '2296',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2296',\n",
       "  'team_name': 'Cornell',\n",
       "  'wiki': 'http://2017.igem.org/Team:Cornell',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Oxyponics\\n',\n",
       "  'abstract': 'Hydroponics is a rapidly growing area of agriculture that is projected to be a $400 million market by 2020. Despite its potential, hydroponics farmers face low crop yields due to diseases and nutrient imbalances. It is well-documented that a sufficient level of oxidative stress from reactive oxygen species (ROS) is necessary for and can enhance crop growth. To that end, Cornell iGEM developed a novel redox biosensor in E. coli that uses a redox-sensitive fluorescent protein reporter to couple an optogenetic transcriptional circuit to an external optics system. In addition to offering enhanced sensitivity to ROS, this system optimizes oxidative stress by controlling redox-sensitive transcriptional responses with greater precision through an external LED. We believe that this will be promising platform to overcome the drawbacks of hydroponics with potential applications in general agriculture.\\n',\n",
       "  'award': 'Best Entrepreneurship;Nominated for Best Entrepreneurship',\n",
       "  'medal': 'gold'},\n",
       " '2506': {'team_id': '2506',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2506',\n",
       "  'team_name': 'CPU_CHINA',\n",
       "  'wiki': 'http://2017.igem.org/Team:CPU_CHINA',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'SynNotch CAR-Tregs   V.S.   Rheumatoid   Arthritis\\n',\n",
       "  'abstract': 'Rheumatoid arthritis (RA) is a serious chronic, inflammatory and systemic autoimmune disease. It is of great essence to develop a kind of novel cell-targeted therapy for RA because there is no radical cure for RA for the time being. To solve the problems existing in the current treatment of RA, we design and build a brand new immunotherapy. FOXP3+ regulatory T cells(Tregs), which can suppress and regulate immune reactions, are modified utilizing a lentiviral vector system to express a chimeric antigen receptor(CAR) targeting inflammatory cells associated with RA. Meanwhile, we insert the Syn-Notch receptor to activate the functional stability pathway of Tregs in the inflammatory environment, which enables them to play their role of immunosuppression in lesions more efficiently and more stably. These two redirections of the two different but interrelated systems on Tregs ensure this novel therapy a promising anti-RA effect. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2494': {'team_id': '2494',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2494',\n",
       "  'team_name': 'CQU_CHINA',\n",
       "  'wiki': 'http://2017.igem.org/Team:CQU_CHINA',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': None,\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': ''},\n",
       " '2382': {'team_id': '2382',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2382',\n",
       "  'team_name': 'CSMU_NCHU_Taiwan',\n",
       "  'wiki': 'http://2017.igem.org/Team:CSMU_NCHU_Taiwan',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Taiwan',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'AFLATOXOUT\\n',\n",
       "  'abstract': 'According to the report conducted by WHO, \\x93aflatoxin\\x94 is a major contributor to the global burden of food-borne diseases. \\r\\n\\r\\nIn this project, we focus on two things: firstly, we would like to prevent people from consuming aflatoxin-contaminated food. We would create a device which could detect the amount of aflatoxin inside the food, then transmit the information on to the internet. Therefore both the public and private sector could use the device and mobile application to track contaminated food throughout the nation.  \\r\\n\\r\\nSecondly, we would like to create a enzyme-encoded yeast to eliminate the aflatoxin in the patients intestine.We would create a yeast that could produce enzymes that eliminate the aflatoxin inside the human intestine. This can help people suffering from the diarrhoea caused by aflatoxin. Additionally it can also be applied to animal foods, getting rid of the aflatoxin in the animals\\x92 bodies that could potentially contaminate our food.\\n',\n",
       "  'award': 'Nominated for Best Food & Nutrition Project;Nominated for Best Wiki;Nominated for Best Entrepreneurship',\n",
       "  'medal': 'gold'},\n",
       " '2204': {'team_id': '2204',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2204',\n",
       "  'team_name': 'CSU_Fort_Collins',\n",
       "  'wiki': 'http://2017.igem.org/Team:CSU_Fort_Collins',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Energy',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'When Life Gives You Lemons, Make Limonene\\n',\n",
       "  'abstract': 'Research done in the last 50 years or so has forced humanity to reconsider its gluttonous appetite for petrol based products and energy. The effects of this \"appetite\" have been profound, scientifically undeniable, and perhaps the biggest threat humanity has faced. The answer; synthetic biology.\\r\\n\\r\\nLike any emerging field, synthetic biology is experiencing an explosion of novel methods and ideas. We have focused our efforts on altering the metabolic process of the archaea Thermococcus kodakarensis to produce pragmatic amounts of the well-known terpenoid, limonene. \\r\\n\\r\\nLimonene is of great interest due to its myriad of industrial uses, namely as a biofuel. Usage of an archaeal organism\\'s mevalonate pathway allows for a simple incursion of one extra enzyme (limonene synthase) to convert GPP into limonene. \\r\\n\\r\\nGas chromatography and western blot methods were used to quantify limonene concentration and detect gene expression respectively, in the altered T. kodakarensis.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2249': {'team_id': '2249',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2249',\n",
       "  'team_name': 'CU-Boulder',\n",
       "  'wiki': 'http://2017.igem.org/Team:CU-Boulder',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Protein Microcompartments Engineered with a Light Activated Open/Close Switch\\n',\n",
       "  'abstract': 'The ability to control the compartmentalization of enzymes and biologics is needed for drug discovery. Bacterial Microcompartments BMCs are naturally occurring protein structures that act to either sequester harmful metabolic intermediates or concentrate critical cytosolic substrates. Light triggered release of toxic therapeutics from molecular compartments would activate them only where needed and potentially minimize their side effects. We have engineered a BMC with a light activated open/close switch that could be used for drug delivery. We did this by incorporating the light-sensitive compound azobenzene as a non-canonical amino acid into the structure of a BMC coat protein. This allows us to induce the disassembly of the BMC and release its enclosed cargo via photo-stimulation. Our work developing the first light activated BMC expands the synthetic biology toolkit and has the potential to help precisely deliver future therapeutics.\\r\\n\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2331': {'team_id': '2331',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2331',\n",
       "  'team_name': 'Dalhousie',\n",
       "  'wiki': 'http://2017.igem.org/Team:Dalhousie',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Mining the Microbiome: Using Porcupines as a Source for Cellulolytic Enzymes\\n',\n",
       "  'abstract': 'Lignocellulose, a component of a plant cell wall, is a largely untapped source for low-cost cellulosic biofuel production. First, lignocellulose must be broken down into component sugars by cellulases, hemicellulases, and various debranching enzymes. Because the primary diet of porcupines is lignified plant material, we reasoned that the porcupine microbiome would be filled with enzymes required to degrade lignocellulose. Using in silico metagenomic analysis, we mined the porcupine microbiome to identify, synthesize and characterize microbial enzymes required for cellulose and hemi-cellulose degradation. Using an orthogonal approach, we created a metagenomic library in E. coli from the porcupine microbiome, to be screened for the ability to grow on lignocellulose, or other pathway intermediates, as a sole carbon source. This work provides a solid foundation for future development of a bioreactor utilizing enzymes from the porcupine microbiome to convert lignocellulose into biofuel.   \\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2435': {'team_id': '2435',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2435',\n",
       "  'team_name': 'Dartmouth',\n",
       "  'wiki': 'http://2017.igem.org/Team:Dartmouth',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Energy',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Construction of robust bacterial plasmid for ethanol production\\n',\n",
       "  'abstract': 'Pyruvate decarboxylase (PDC) and alcohol dehydrogenase 2 (ADH2) are key enzymes for mediating ethanol production from pyruvate. A particularly well-characterized PDC-ADH system has been found in Zymomonas mobilis, and this pathway has been used to engineer a variety of bacteria for ethanol production. In some cases, this pathway works well, resulting in high levels of ethanol production. In other cases, however, the pathway does not work very well. To understand this, we aim to develop a set of BioBrick compatible expression vectors with different plasmid replicons to allow expression of the PDC-ADH pathway in a range of different bacteria. Organisms expressing the PDC-ADH pathway will be assayed for PDC and ADH activity by enzyme assay and for ethanol production by high-pressure liquid chromatography (HPLC). These new expression plasmids will be useful for other groups looking to expand their BioBrick engineering beyond the confines of current model or \"chassis\\x94 organisms.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2441': {'team_id': '2441',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2441',\n",
       "  'team_name': 'DEIAGRA',\n",
       "  'wiki': 'http://2017.igem.org/Team:DEIAGRA',\n",
       "  'region': 'Asia',\n",
       "  'country': 'India',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Bio-Beads For Removing Heavy Metal Toxicity From Industrial Effluents\\n',\n",
       "  'abstract': 'Heavy metals, also known as trace metals, are one of the most persistent pollutants in waste water. Biological remediation processes are indicated to be very effective in the treatment of heavy metal pollutants in waste water.The over-expression of metal binding proteins has been widely exploited to increase the metal binding capacity, tolerance or accumulation of heavy metals in bacteria and plants. An expression level of transgene/gene of interest depends upon strength of promoter. \\r\\nHereby in our project we are focusing on over-expressing the \"Top-4 metal binder protein\" (iGEM part BBa_K1478002) and \"Human Metallotheonin 3\" by strong synthetic promoter designed by employing bionformatics tools which would be suitable for expression in microbes and plants.  We will fabricate bio-beads by immobilizing, bacterial cells which strongly expresses our cloned metallotheonin. Our fabricated beads allows fast, easy and user handy purification of industrial waste water which can be further use for irrigation purpose.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2444': {'team_id': '2444',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2444',\n",
       "  'team_name': 'Delaware',\n",
       "  'wiki': 'http://2017.igem.org/Team:Delaware',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Narrow Spectrum Bacteriocins as Targeted Gut Microbiome Therapeutics\\n',\n",
       "  'abstract': 'The gut microbiome is an exciting field of research that has increased exponentially across the past few years. Many studies have found a correlation between imbalances in the gut microbiome and disease states. Can altering the gut microbiome at all change disease state? In order to evaluate this question, methods are currently needed that can allow researchers to make targeted manipulations to modulate the populations of specific microbes within a microbiome. \\r\\n\\r\\nThe goal of our project is to overexpress bacteriocin genes (toxins that bacteria produce that affect other bacteria) in E. coli. The rationale for our methodology this year is to answer questions similar to \"how much of a specific bacteriocin is needed to reduce the abundance of a single microbe by 60%?\"  Our intention is to generate data that will be used in computational modeling and future experimental projects towards developing gut microbiome therapeutics.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2410': {'team_id': '2410',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2410',\n",
       "  'team_name': 'Delgado-Ivy-Marin',\n",
       "  'wiki': 'http://2017.igem.org/Team:Delgado-Ivy-Marin',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': None,\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': 'Hearing loss affects millions of people worldwide.  While there have been some great advances with restoration of hearing through prosthetic devices like cochlear implants, we would like to  The goal of our project is to endow Chinese hamster ovary cells (a mammalian cell lines) in vitro with the ability to respond to sonic vibrations through the gating of an recombinant ion channel.  Recombinant DNA origami molecules, with hopefully (we don\\'t know yet!) with different resonant frequencies will be bound to the surface of the cells on the mechanical sensing ion channel to control gating.  We are hoping to achieve a proof of principle for the rational design of a artificial \"stereocillia,\" the descendant of which could be used to organically restore hearing loss in humans.\\n'},\n",
       " '2355': {'team_id': '2355',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2355',\n",
       "  'team_name': 'DTU-Denmark',\n",
       "  'wiki': 'http://2017.igem.org/Team:DTU-Denmark',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Denmark',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'DTU Snakebite Detectives: Diagnostic Tool for Venom Discrimination\\n',\n",
       "  'abstract': 'Envenomation by snakebite is one of the most neglected diseases with an estimated 5 million cases each year. These result in an estimated 100,000 deaths and 400,000 disabilities annually. The only effective treatment is animal derived antivenoms, which frequently causes adverse reactions. As a result, they are often only administered as a last resort.\\r\\n\\r\\nWe aim to develop a detection-assay that enables us to determine what type of snake a victim is envenomed by. The strategy is to target distinguishable enzymatic features in the different snake venoms by developing suitable substrate-based diagnostics. Our diagnostic tool can be useful in identifying the relative composition of specific venom components in a blood sample and thereby allow for the safest course of treatment.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2464': {'team_id': '2464',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2464',\n",
       "  'team_name': 'Duke',\n",
       "  'wiki': 'http://2017.igem.org/Team:Duke',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'An affordable HIV & Zika rapid lateral flow diagnostic using thermostable griffithsin\\n',\n",
       "  'abstract': 'Our project focuses on developing a lateral flow assay-based Rapid Diagnostic Test (RDT) for inexpensive, early-stage, point-of-care diagnosis of HIV and Zika. Treatment access and public health interventions for these diseases are seriously limited by diagnostics, as current RDTs detect either poorly-conserved viral antigens or the patients\\x92 own delayed antibody response and thus are unable to reliably diagnose early-stage infection, while more sophisticated tests are not widely accessible where the disease burden is greatest. Our novel approach enables modular, early-stage detection within an inexpensive, scalable RDT format by using a lectin to detect carbohydrates universal to a number of important viral pathogens in combination with antibodies targeting virus-specific conserved sites. Additionally, we have rationally engineered a thermostable variant of the diagnostic lectin to improve manufacturing economics and device robustness. By enabling earlier and larger-scale diagnosis, we hope this approach will help to improve interventions for these critical global health challenges.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2290': {'team_id': '2290',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2290',\n",
       "  'team_name': 'East_Chapel_Hill',\n",
       "  'wiki': 'http://2017.igem.org/Team:East_Chapel_Hill',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Developing the Fluoride Riboswitch as a Technology to Combat Excessive Water Fluoridation\\n',\n",
       "  'abstract': 'Most water sources in developing countries are decentralized. Consequently, drinking water is prone to high levels of fluoride from geological sources, corresponding to endemic fluorosis and may lead to developmental or reproductive defects. Underdeveloped regions in rapidly developing countries, such as China and India, have triple the recommended WHO limit of fluoride in their groundwater. Our project seeks to develop the fluoride riboswitch, an mRNA that can bind to fluoride and regulate the expression of downstream genes, as a technology to combat fluoride contamination in water. We developed a system where the fluoride riboswitch controls the expression of chloramphenicol acetyltransferase, allowing bacteria to grow on the antibiotic chloramphenicol in the presence of fluoride. We will use this operon to screen and select riboswitches with higher responsiveness to fluoride. We envision using engineered fluoride riboswitch systems as tools to sequester, bioremediate, or detect fluoride in a cost effective manner. \\r\\n\\n',\n",
       "  'award': 'Nominated for Best Poster, High_school;Best Measurement, High_school;Nominated for Best Measurement, High_school',\n",
       "  'medal': ''},\n",
       " '2308': {'team_id': '2308',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2308',\n",
       "  'team_name': 'ECUST',\n",
       "  'wiki': 'http://2017.igem.org/Team:ECUST',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Energy',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Light Harvester\\n',\n",
       "  'abstract': 'Traditional energy will finally come to exhaustion, and the development of new energy is extremely urgent. To produce hydrogen, which is a clean and efficient source of energy, through microbial photosynthesis is an environmentally friendly and sustainable pathway. However, the facts such as low efficiency of bio-hydrogen production, lack of large-scale application device and others hinder the application prospect of bio-hydrogen production. Therefore, this project aims to expand the absorption spectrum of photosynthetic bacteria by the way of forster resonance energy transfer (FRET) in order to improve the efficiency of the photoreaction. On top of that, we will also design a photo-reactor special for bio-hydrogen production to solve the problem of applying it on a large scale.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2330': {'team_id': '2330',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2330',\n",
       "  'team_name': 'Edinburgh_OG',\n",
       "  'wiki': 'http://2017.igem.org/Team:Edinburgh_OG',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Modular molecular toolkit for re-sensitisation of antibiotic-resistant pathogens using CRISPR delivered by a two-phage system\\n',\n",
       "  'abstract': 'The threat posed by antibiotic resistant bacteria is a pressing issue which must be addressed. It is difficult and expensive to develop new antibiotics, so our project is designed to make currently available antibiotics useful again. Our aim is to create a toolkit to re-sensitise pathogens to antibiotics using CRISPR and a two-phage system, based on work by Yosef et al. (2015). An engineered lysogenic phage will transfer a CRISPR system to its host bacterium, designed to cleave resistance genes and also confer protection from an engineered lytic phage. When this lytic phage is added to the population, it kills any bacteria that have not been successfully re-sensitised.  We chose to target genes found in the highly-resistant ESKAPE pathogens, and worked with 4 different phages - P1, lambda, T4 and T7. Our system was modelled in silico and tested empirically on a specially designed E. coli testing platform.\\n',\n",
       "  'award': 'Best Therapeutics Project, Overgrad;Nominated for Best Therapeutics Project, Overgrad;Nominated for Best Wiki, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2406': {'team_id': '2406',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2406',\n",
       "  'team_name': 'Edinburgh_UG',\n",
       "  'wiki': 'http://2017.igem.org/Team:Edinburgh_UG',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'SMORE: A Site-specific, Modular Recombination Toolkit for Genetic Engineering\\n',\n",
       "  'abstract': 'Site-specific recombination is widely utilized for genetic engineering; however, a synthetic biology approach is needed to facilitate its use in a rational and bottom-up way. Therefore, we created SMORE, a toolkit containing well-characterized recombinases, their respective target sites, and recombinase activity assay constructs, with their utility demonstrated by three constructs: a gene randomizer, logic gates, and a pulse generator. Among other applications, these could be applied to metabolic engineering to improve the industrial production of high-value compounds. Our human practices concerns the accessibility of SMORE, making it easier to use. First, we systematically reviewed iGEM team composition in previous years to determine whether diversified teams enjoyed more success; second, we built a microfluidic device and had its protocols open-sourced to promote its use outside of the discipline of engineering; lastly, we adjusted the readability of our promotional material and created animations to facilitate the understanding of the concept of SMORE.\\n',\n",
       "  'award': 'Nominated for Best Measurement',\n",
       "  'medal': 'gold'},\n",
       " '2487': {'team_id': '2487',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2487',\n",
       "  'team_name': 'Emory',\n",
       "  'wiki': 'http://2017.igem.org/Team:Emory',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Phosphate removal and detection for wastewater treatment\\n',\n",
       "  'abstract': 'The WaterHub at Emory University is a wastewater treatment plant on campus that utilizes bacteria and plants to recycle 400,000 gallons of water a day for non-potable use. The Emory iGEM team has utilized synthetic biology to try to help optimize their system by developing a phosphate accumulating organism that can help them combat the high orthophosphate levels in their wastewater.  We have created a number of experimental strains of bacteria with variations in polyphosphate kinase and exopolyphosphatase expression. Their phosphate assimilation rates and growth rates in waterhub sewage with an added carbon source were tested. We have also created a phosphate sensor that takes advantage of a natural phosphate sensing mechanism regulated by promoters called pho regulon. This sensor will allow the cells to indicate when phosphate rates have been lowered by our engineered bacteria.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2203': {'team_id': '2203',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2203',\n",
       "  'team_name': 'EPFL',\n",
       "  'wiki': 'http://2017.igem.org/Team:EPFL',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Switzerland',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'aptasense\\n',\n",
       "  'abstract': 'The emerging field of cell-free synthetic biology promises to significantly improve molecular diagnostics. Cell-free systems for engineering and implementation allows for fast testing cycles, ready-to-use detection devices, better biosafety, as well as cheap and easy transport and storage. Previously, Pardee et al. developed a cell-free system for detecting viral RNA in patients by engineering toehold switches coupled to lacZ for signal generation.\\r\\n Since many clinical tests rely on the detection of protein biomarkers, we developed a novel scheme for detecting proteins by coupling aptamer based affinity reagents to the toehold-switch concept in cell-free system. We furthermore improved on various aspects of the original Pardee cell-free system to create a tool that is more effective, cheaper and easier to use. Because the system can be rapidly engineered and deployed, and all parts can be modified to recognize different protein or RNA molecules, it\\x92s a highly modular and novel diagnostic tool.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Diagnostics Project;Nominated for Best Integrated Human Practices;Best Education & Public Engagement;Nominated for Best Education & Public Engagement;Nominated for Best Software Tool;Nominated for Best New Basic Part',\n",
       "  'medal': 'gold'},\n",
       " '2511': {'team_id': '2511',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2511',\n",
       "  'team_name': 'EpiphanyNYC',\n",
       "  'wiki': 'http://2017.igem.org/Team:EpiphanyNYC',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Tackling Huntington\\x92s Disease with RNA Strand Displacement\\n',\n",
       "  'abstract': \"Huntington's disease (HD) is an autosomal dominant disorder that causes the progressive breakdown of nerve cells in the brain, and currently has no cure. HD is usually adult-onset, and includes symptoms such as amnesia, involuntary movements, and physical incoordination, rendering the patient with a lifespan of a mere ten years after onset. The main cause is a trinucleotide repeat of CAG in the huntingtin (HTT) gene, where a repeat of 40 or more can cause the disease to manifestation. Our goal is to create a synthetic RNA strand displacement technology that targets and blocks endogenous faulty mRNA and releases a corrected RNA strand for proper protein synthesis of the HTT protein. For this, we aim to generate a plamsid with an identical sequence to the human mutant form of Huntington's disease. Ultimately, once technologies are developed to overcome endocytosis and blood brain barrier traversal, an injectable cure can be produced.\\n\",\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2500': {'team_id': '2500',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2500',\n",
       "  'team_name': 'ETH_Zurich',\n",
       "  'wiki': 'http://2017.igem.org/Team:ETH_Zurich',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Switzerland',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'CATE - Cancer-Targeting E. coli\\n',\n",
       "  'abstract': 'Treating cancer remains a challenge for modern medicine. Current treatments are often invasive, time-consuming and limited by toxic effects on healthy tissue.\\r\\n\\r\\nAn ideal therapeutic should be effective, safe and targeted. We engineered a non-pathogenic strain of E. coli to approach this goal. The combination of autonomous targeting, visualization and externally controlled toxin release provides a novel non-invasive, quick and safe approach to treating cancer. \\r\\n\\r\\nSpecifically, following intravenous administration, our E. coli preferentially populates solid tumors due to the properties of the tumor microenvironment. Intracellular accumulation of the cytotoxin azurin and the MRI contrast agent bacterioferritin is controlled by an AND gate which is activated once a population threshold is reached AND lactate, a tumor marker, is present. However, azurin release happens only if the physician confirms the correct localization of the bacteria via MRI and subsequently heats them via focused ultrasound, which activates a thermo-inducible cell lysis system.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Therapeutics Project, Overgrad;Best Poster, Overgrad;Nominated for Best Poster, Overgrad;Best Model, Overgrad;Nominated for Best Model, Overgrad;Nominated for Best New Basic Part, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2448': {'team_id': '2448',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2448',\n",
       "  'team_name': 'Evry_Paris-Saclay',\n",
       "  'wiki': 'http://2017.igem.org/Team:Evry_Paris-Saclay',\n",
       "  'region': 'Europe',\n",
       "  'country': 'France',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Psicose\\n',\n",
       "  'abstract': 'The World Health Organisation recognized obesity as a serious illness so we decided to consider ways to combat it\\x92s spread and effects. This is how we came across D-Psicose, which is a non-toxic rare sugar and a sweetener with several measurable health effects. \\r\\nCurrently, its industrial production is based on purification from organic matter or on epimerization of more common sugars but it remains difficult and expensive. To improve the bioproduction process, we decided to construct a biosensor to screen the best epimerase for the conversion of Fructose into Psicose and corroborate the fluorescence quantifications with HPLC measurements.\\r\\nTo be efficient any bioproduction process requires screening of the most efficient enzyme(s). We are bringing to iGEM a new enzyme screening process based on biosensors, which, in the presence of the compound of interest, emit a fluorescent signal and can be used to identify the enzyme that gives the optimal production.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2324': {'team_id': '2324',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2324',\n",
       "  'team_name': 'Exeter',\n",
       "  'wiki': 'http://2017.igem.org/Team:Exeter',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Pili+: modified cell surface pili as a new vogue for bacterial bioremediation\\n',\n",
       "  'abstract': 'Heavy metal ion pollutants have significant effects on local flora and fauna and their leaching has implications for human health. Current treatment methods are energetically costly and, in the case of lime dosing, environmentally detrimental. We aim to investigate E. coli type 1 pili adhesion mechanisms and repurpose the involved structures to bind heavy metal ions in water. Pili are hair-like structures found on bacteria that attach to cell surface mannose molecules, using their terminal pili protein, FimH. Our aim is to fuse a variety of metal binding proteins to the FimH protein by modification of the fimH gene. The modified bacteria will be contained in a fluidised media reactor filter system used in conjunction with a hydrocyclone to prevent GMO release. Our modular cloning strategy, allows us to develop a toolkit for a wide number of pili applications and further future developments.\\r\\n\\n',\n",
       "  'award': 'Best Environment Project, Overgrad;Nominated for Best Environment Project, Overgrad;Best Applied Design, Overgrad;Nominated for Best Applied Design, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2384': {'team_id': '2384',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2384',\n",
       "  'team_name': 'FAFU-CHINA',\n",
       "  'wiki': 'http://2017.igem.org/Team:FAFU-CHINA',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Synergism of Phosphate-solubilizing bacteria-plant interactions for bioremediation of metalliferous soils\\n',\n",
       "  'abstract': 'We hope to establish a sustainable, regulable and reusable project to solve the soil polluted by heavy metals. Phosphate-solubilizing bacteria have important functions. In our project, we used the mechanism of the alliance between microbe and plant, by manufacturing Bacillus megatherium, which is a kind of phosphate-solubilizing microorganism exists in the root system, forcing it enhance the plant remediation from two aspects, accumulating heavy metals and defend adversity stress. To achieve the goal of spatial specificity, we make most of the expression system which is regulated by root organic acid. We also used MBP (metal binding peptide) to make accumulation of heavy metal in root system\\x92s soil success. Finally, heavy metals will transfer to plant itself. A series of transformation we made in Phosphate-solubilizing microorganism in our project will solve the weakness of hyper-accumulators, therefore, the remediation method will be put into use widely.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2523': {'team_id': '2523',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2523',\n",
       "  'team_name': 'Florida_Atlantic',\n",
       "  'wiki': 'http://2017.igem.org/Team:Florida_Atlantic',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Software',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Welcome to the Machine: Developing a Novel Biosensor for Artemisinin\\n',\n",
       "  'abstract': 'Counterfeit drugs are one of the most serious problems in medicine. Not only do they prevent patients from receiving the treatment that they need, they are difficult for consumers to identify and can promote drug resistance. In order to address this problem, our team is developing a biosensor platform that can detect the presence of the antimalarial drug artemisinin. First, a genetic machine was devised that generates a large amount of protein and then rapidly lyses a cell, releasing that protein into solution. Then, a machine learning protocol was designed to determine if a protein can perform a specific function (in this case, bind artemisinin). Finally, a novel synthetic protein was added to the genetic machine to create a cheap, effective, and relatively foolproof biosensor. Because this system is not limited to artemisinin, it can be used to rapidly create a biosensor for different medications or compounds of interest. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2419': {'team_id': '2419',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2419',\n",
       "  'team_name': 'Franconia',\n",
       "  'wiki': 'http://2017.igem.org/Team:Franconia',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Team name changed from Erlangen_Nuremberg on 2017-7-10\\n',\n",
       "  'title': 'B.E.A.M. - Biocompatible Elastic Artificial Muscle\\n',\n",
       "  'abstract': 'The human muscular system enables supremely precise body movements to perform everyday tasks. While robotic devices improve these capabilities for industrial purposes, currently used medical prostheses can only mimic basic functions of the muscular system. Therefore, we will develop biological synthetic muscles to provide ecological friendly tissue compatible with the human body. At first, we will fabricate biopolymers with integrated molecular machines that will form a tissue capable of a muscle-like contraction. Named molecular machines are based on azo dyes, which can contract by light irradiation. These will be attached to the biopolymer matrix fabricated by Escherichia coli via biotin based catcher-tag systems. Another approach is to express both conductive and elastic P-Pili from Geobacter sulfurreducens and Escherichia coli. Alternating layers of these biopolymers form the dielectric elastomer actuator and the resulting biological tissue can respond to an applied voltage. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2295': {'team_id': '2295',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2295',\n",
       "  'team_name': 'Freiburg',\n",
       "  'wiki': 'http://2017.igem.org/Team:Freiburg',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'CARtel\\x99 - Chimeric Antigen Receptor on T cells Expressed Locally in the Tumor Microenvironment\\n',\n",
       "  'abstract': 'A new approach to cancer treatment is the chimeric antigen receptor (CAR) therapy, which shows promising results fighting tumors in clinical trials. It consists of autologous isolated T cells modified with a chimeric receptor based on the T-cell receptor combined with the recognition domain of an antibody. Upon reinjection, CAR T-cells exhibit cytotoxicity with high affinity towards cells displaying the antigen. However, clinical trials have shown that as tumor antigens are not solely expressed on tumor cells, but also on healthy tissues, grave off-target effects like the Graft-versus-Host-Disease may occur.\\r\\nIn order to avoid such side-effects, we engineer CAR T-cell lines specifically activated by factors of the tumor microenvironment. Controlled by a genetic AND-gate system the T cells need two input signals in order to express CAR. This would allow highly localized cytotoxic activity of T cells and provide safer cell-based cancer immunotherapy especially for solid tumors.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Wiki, Overgrad;Nominated for Best New Basic Part, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2367': {'team_id': '2367',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2367',\n",
       "  'team_name': 'FSU',\n",
       "  'wiki': 'http://2017.igem.org/Team:FSU',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Bartii: Cell-Based Therapy for Celiac Disease\\n',\n",
       "  'abstract': 'Celiac Disease (CD) is an autoimmune disorder that affects approximately 75,000,000 people around the world. The disease is triggered by the ingestion of gluten, a protein found in wheat. Common symptoms are abdominal discomfort, diarrhea, and malnutrition. The current practice for treating Celiac Disease is strict adherence to a gluten-free diet. Gluten-free diets are difficult to maintain, diminish the quality of life of CD patients, and on average are 242% more expensive than diets that contain gluten. We are proposing a new approach in the prevention and management of CD where patients are able to enjoy foods that contain gluten through population of the gut with engineered therapeutic cells.  The cells will be able to: 1) Degrade gliadin, a peptide in gluten that elicits an inflammatory response 2) Sequester gliadin by binding it to the surface of the therapeutic cells 3) Sequester zonulin, a hormone responsible for enhancing inflammation.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2446': {'team_id': '2446',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2446',\n",
       "  'team_name': 'Fudan',\n",
       "  'wiki': 'http://2017.igem.org/Team:Fudan',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Fudan_stripe Team\\n',\n",
       "  'title': 'SwordS: antigen density targeting with customized therapeutic responses via SynNotch-Stripe system\\n',\n",
       "  'abstract': 'Antigen density on tumor cells\\x92 surface is heterogeneous, which constrains current solid tumor immunotherapy to target only highly expressed tumor associated antigens (TAAs). We designed an antigen density targeting immunotherapy platform, SwordS (SynNotch-Stripe system), that is capable of generating non-monotonic therapeutic responses. To demonstrate the concept, we took advanced-stage hepatocellular carcinoma (HCC), one of the most common malignant tumors and a leading cause of cancer-related death in China, as a potential target of disease treatment by SwordS. In combination with HCC associated antigen responding SynNotch, our genetically engineered cells would generate an antigen density-dependent, triple HCC therapeutic response patterns. Our approach is aimed to reduce the on-target/off-tumor effects and greatly expand the diversification and combined potency of tumor therapy. We believe that SwordS is promising to become a brand new generation\\x92s customized therapy developing platform for optimizing cellular immunotherapy against cancerous diseases.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2460': {'team_id': '2460',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2460',\n",
       "  'team_name': 'Fudan_China',\n",
       "  'wiki': 'http://2017.igem.org/Team:Fudan_China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Information Processing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'MemOrderY: A sequential memory device that monitors the changing of  signals\\n',\n",
       "  'abstract': 'Biological memory can be defined as a sustained cellular response to a transient stimulus. Although the existing memory devices can be highly diverse and delicate, they can only record the static state at the instant the recording action happens, therefore, they do not have the ability to monitor the dynamic changing process of one signal. This year, we want to development the concept of cellular memory, and to realize the monitoring of one changing signal. So, we design a unique memory device with sequential structure using recombinases. After we measure the orthogonality and efficiency of our recombinases, we try to engineer our E.coli population to record several static states of the target signal at different time point. Thus, by putting these results together in the right time order with our recombinase-based sequential system, we can finally get an idea of how the signal changes as time goes by.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2221': {'team_id': '2221',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2221',\n",
       "  'team_name': 'Gaston_Day_School',\n",
       "  'wiki': 'http://2017.igem.org/Team:Gaston_Day_School',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Energy',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Engineering E.coli to Improve Alcohols Resistance For Biofuel Production\\n',\n",
       "  'abstract': 'During industrial production of alcohols, large-scale bacterial fermentation is often used to yield more product. However, when the alcohol level in the growth medium nears the toxic concentration for the bacteria, the bacteria will start to break down the alcohol they have created. Therefore, the toxic threshold determines the maximum alcohol production. This year, we have started a new project to increase alcohol resistance in bacteria, which should also increase the toxic threshold. This, in turn, would allow the bacteria to produce higher concentrations of the desired alcohol. E.coli is a good platform for biofuel production because the pathways to produce alcohols such as ethanol, isobutanol and isopropanol already exist. We plan to clone several genes in E.coli, such as GlmY and GlmZ, that are associated with alcohol resistance. \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2517': {'team_id': '2517',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2517',\n",
       "  'team_name': 'GBioS_ESALQ',\n",
       "  'wiki': 'http://2017.igem.org/Team:GBioS_ESALQ',\n",
       "  'region': 'Latin America',\n",
       "  'country': 'Brazil',\n",
       "  'track': None,\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': ''},\n",
       " '2524': {'team_id': '2524',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2524',\n",
       "  'team_name': 'Georgia_State',\n",
       "  'wiki': 'http://2017.igem.org/Team:Georgia_State',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'The Novel Synthesis of Factor C: A Story of Blood and Venom\\n',\n",
       "  'abstract': 'Endotoxin contamination is a concern for pharmaceutical and medical devices that are intended to encounter human blood. The most widely accepted test for detection is the Limulus Amebocyte Lysate (LAL), which is produced by harvesting the blood of horseshoe crabs and this has devastated their population. LAL detects concentrations as low as one part per trillion of endotoxins by using the natural clotting mechanism of the horseshoe crab blood in the presence of lipopolysaccharides (LPS) found on the surface of gram-negative bacteria. Factor C is a component of LAL that self-cleaves in the presence of LPS to initiate clotting. We used the cleaving property of factor C to design a novel contamination biosensor. Producing a recombinant fusion of factor C with human chorionic gonadotropin (hCG), will allow us to detect our system using over-the-counter pregnancy tests creating a cheaper and more efficient way to detect endotoxins.\\n',\n",
       "  'award': 'Nominated for Best Integrated Human Practices, Overgrad;Chairmans Prize',\n",
       "  'medal': 'sivel'},\n",
       " '2438': {'team_id': '2438',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2438',\n",
       "  'team_name': 'Gifu',\n",
       "  'wiki': 'http://2017.igem.org/Team:Gifu',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Japan',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Sake Sommelier\\n',\n",
       "  'abstract': 'Our team will try to make a new technology called LAM (Lactic acid mediated) communication utilizing GAR+ prion. GAR, or glucose associated repression controls the usage of glucose of yeast. In 2016, it was proved that this pathway is managed by the certain amount of lactic acid. Lactic acid produced by lactic acid bacteria, affects the prion formed gar- and induces the formation of GAR+. Considering this mechanism and terminators of fission yeast and regulating the amount of mRNA of GFP and anti-GFP, we will enable yeast to communicate each other beyond the species of gram-negative bacteria and make a biosensor to sense hiochi.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2442': {'team_id': '2442',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2442',\n",
       "  'team_name': 'Glasgow',\n",
       "  'wiki': 'http://2017.igem.org/Team:Glasgow',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Campylocator: Detection of Campylobacter jejuni for the prevention of food poisoning \\n',\n",
       "  'abstract': 'Food contamination and improper handling of raw poultry is the leading cause of food poisoning in the U.K. Campylobacter jejuni is native to poultry but is highly pathogenic to humans and cross-contamination of surfaces and other foods is common. The current detection methods are time consuming and costly and therefore we aimed to create a new cheaper, faster system for detection of this bacteria using synthetic biology. We designed and engineered a dual input biosensor using sensory aspects including campylobacter-specific sugars and quorum sensing to identify any C. jejuni present on a swabbed surface. We also developed an understanding of the legal aspects of using GMO biosensors within the EU. In the future, our biosensor is a potential solution in reducing the risk of food poisoning from Campylobacter.\\r\\n\\n',\n",
       "  'award': 'Best Food & Nutrition Project;Nominated for Best Food & Nutrition Project;Nominated for Best Wiki;Nominated for Best Measurement;Nominated for Best Applied Design;Nominated for Best Part Collection',\n",
       "  'medal': 'sivel'},\n",
       " '2364': {'team_id': '2364',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2364',\n",
       "  'team_name': 'Greece',\n",
       "  'wiki': 'http://2017.igem.org/Team:Greece',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Greece',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'pANDORRA \\x96 Engineering and delivering modular RNAi-based logic circuits to treat colorectal cancer\\n',\n",
       "  'abstract': 'Engineering of transcriptional/post-transcriptional synthetic circuits and optimization of delivery systems to transfer them into mammalian cells could revolutionize the development of programmable cellular responses towards a plethora of applications in health and disease. We are developing pANDORRA, an assembly platform for modular RNAi-based logic circuits, capable of integrating multiple endogenous inputs to classify a cell by its miRNA expression profile and trigger a biological actuation. As a proof-of-principle, we are employing this strategy to build a cell-type classifier for colorectal cancer cells inducing fluorescence and/or triggering apoptosis. Simultaneously, in order to design a multi-pronged treatment approach for colorectal neoplasia, we are engineering a novel anti-cancer E.coli agent, facilitating selective adhesion and cell density-dependent invasion into cancer cells, as the delivery module for our classifier circuit. In the future, we envision our system to be employed in a variety of pathological conditions generating reporter signals or protein outputs of therapeutic potential.\\n',\n",
       "  'award': 'Nominated for Best Therapeutics Project',\n",
       "  'medal': 'gold'},\n",
       " '2299': {'team_id': '2299',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2299',\n",
       "  'team_name': 'Grenoble-Alpes',\n",
       "  'wiki': 'http://2017.igem.org/Team:Grenoble-Alpes',\n",
       "  'region': 'Europe',\n",
       "  'country': 'France',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'SnapLab, our portable kit that will detect cholera\\n',\n",
       "  'abstract': 'We are designing a pathogen sensitive detector, allowing an easier and more specific diagnostic of cholera.\\r\\nAfter extracting DNA from the feces and performing the analysis, a nucleic acid sequence will be detected if the patient is infected with cholera. Once these sequences are introduced into the bacteria, fluorescence will be emitted and captured by the camera of a smartphone. The application that we will design will perform the analysis and do the image processing. Plus, the temperature will continuously be monitored within the kit.\\r\\nThe greatest advantage of this device will be its capability to communicate the results to its user and to map the cases of cholera. That way, we will be able to know which area is the most affected by the epidemic. \\r\\nGoing forward, this kind of device will be replicable for different pathogens, thus allowing to widen the spectrum of this kit\\x92s use.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2361': {'team_id': '2361',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2361',\n",
       "  'team_name': 'Groningen',\n",
       "  'wiki': 'http://2017.igem.org/Team:Groningen',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Netherlands',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'IMPACT - a programmable bacteriophage detection system\\n',\n",
       "  'abstract': 'During the fermentation process of dairy products, such as cheese and yogurt, fermentation bacteria are under constant threat of bacteriophages. Severe infections can result in inferior product quality, wasted processing time and expensive countermeasures. Our team has developed IMPACT: Integrated Modular Phage-Activated CRISPR Tracer. Built in Lactococcus lactis, this bacteriophage detection system consists of two specialized orthogonal CRISPR mechanisms. The first continuously surveys incoming bacteriophage DNA and isolates spacers from it, while the other enables transcription of a signal-gene when a spacer is matched to the pre-programmed target array. To facilitate use, we have developed a closed detection cartridge and bioinformatic algorithms for determining new target sequences. Our product offers rapid on-site detection without techniques that require highly trained personnel. Our goal is to have a positive impact on industries all over the world that depend on fermentation processes and contribute to a better understanding of bacteriophage infections in general.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2415': {'team_id': '2415',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2415',\n",
       "  'team_name': 'GZHS-United',\n",
       "  'wiki': 'http://2017.igem.org/Team:GZHS-United',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Mos Quit X Cry\\n',\n",
       "  'abstract': 'Mosquito-related diseases, such as malaria, dengue fever, cause thousands of casualties and tremendous economic loss every year. In most of the cases, Aedes and Culex are the major source of suffering, and governments around the world take diminishing Aedes and Culex as priority in mosquito-related diseases control.\\r\\n\\r\\nBacillus thuringiensis israelensis (Bti) and Bacillus sphaericus (Bs) are natural-occurring biological larvicides that are widely used in urban area because they are environmental-friendly and economical. \\r\\n\\r\\nIn our project, two mosquito toxic proteins, Cry4Ba and Mtx1, which are derived from Bti and Bs respectively, are coexpressed via transcriptional and translational fusion. The recombinant protein has higher efficiency in larvae elimination and is more extensive insecticidal spectrum than any single of them.\\r\\n\\r\\nOur project has created a new larvicide that specifically target against the larvae of mosquito, and we hope our product could keep people away from the suffering caused by mosquitoes.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2401': {'team_id': '2401',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2401',\n",
       "  'team_name': 'Hamburg',\n",
       "  'wiki': 'http://2017.igem.org/Team:Hamburg',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Resistent Germs And How To Fight Them\\n',\n",
       "  'abstract': 'Estimates from the World Health Organization predict a resurgence of bacteria to their long-lost top spot among the most devastating human diseases. How did it come to that? Since the discovery of Alexander Fleming, the first scientist that invented and proved the concept of antibiotics, over the years scientists have discovered a vast number of antibiotics to effectively treat bacterial infections. We could fight most bacterial threats for a long time which lead to a notion of false comfort that everything is under control! However, with the current overuse of antibiotics and acquired resistances among bacteria we are about to enter a new era, the post antibiotic era.\\r\\nWe are now reaching the tipping point, at which new strategies are required to overcome multiresistance and prevent a resurgence of devastating infections. Our Project will help to combat multiresistant-infections with a Trojan-Horse-Approach utilizing gallium-loaded siderophores as a resistance-resistant therapy.\\r\\n\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2251': {'team_id': '2251',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2251',\n",
       "  'team_name': 'Harvard',\n",
       "  'wiki': 'http://2017.igem.org/Team:Harvard',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Optimizing Curli Fiber Production as a Scalable Materials Manufacturing Platform\\n',\n",
       "  'abstract': 'Curli fibers, the main proteinaceous component in E. coli biofilms, polymerize extracellularly to form a macroscopic agglomeration of material when isolated in sufficient bulk. It has been demonstrated that functional peptide domains can be fused to the self-polymerizing units of curli to form a variety of functional materials. Thus, curli fibers present a promising platform for the scalable production of programmable materials.\\r\\n\\r\\nOur project focuses on optimizing curli production on two fronts. First, we aim to increase curli export efficiency by optimizing the stoichiometric ratio of proteins involved in the curli pathway. Second, we aim to optimize the conditions for protein-producing cell cultures by growing them in a bioreactor. Our work along these two lines will inform the further development of the curli system as a feasible biosynthetic platform for producing materials at industrially relevant yields.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2304': {'team_id': '2304',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2304',\n",
       "  'team_name': 'HBUT-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:HBUT-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Nickel Hunter: a biosensor that detects nickel ions in the natural environment\\n',\n",
       "  'abstract': \"China is the world's largest producer and consumer of nickel ions. However, nickel and its compounds are one of the primary pollutants in China. They mainly derive from electroplating, nickel metal processing and battery manufacturing. Nickel and its compounds released from the industry  have detrimental effects on the environment and hence threaten human health.  Therefore, nickel is one of the heavy metals that are often monitored in the environment and food chains. The traditional nickel detecting method is to implement chemical measurement of nickel-containing samples in the laboratory, which is time-consuming, expensive, and difficult to operate. Thus, we expect to create a cheap, user-friendly E. coli biosensor, a type of bacterium that produces fluorescence protein when it detects nickel ions in the environment. Consequently, we can find out the amount of nickel ions existing in a certain environment by examining the intensity of fluorescence produced by the probing bacteria. \\n\",\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2398': {'team_id': '2398',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2398',\n",
       "  'team_name': 'Heidelberg',\n",
       "  'wiki': 'http://2017.igem.org/Team:Heidelberg',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'THE PHAGE AND THE FURIOUS \\n',\n",
       "  'abstract': 'Darwinian evolution is an enormously powerful concept that drove biology towards astonishing complexity and beauty. This year, the iGEM team Heidelberg aims at harnessing this power to accelerate the engineering of biomolecules for human benefit. To this end, we built upon the PACE (phage-assisted continuous evolution) method: Synthetic biological circuits couple the survival of quickly mutating phages carrying a protein of interest to directed selection within E. coli hosts. Thereby, proteins with improved functions can be evolved within hours.\\r\\nWe designed a standardized toolbox that highly simplifies PACE and expands its utility towards various new areas of application, including the development of enzymes for pharmaceutical production. Additionally, we applied machine learning techniques to evaluate and systematically improve protein sequences to serve a desired purpose. Taken together, we provide a new foundational advance by introducing a unique combination of in silico and in vivo directed evolution to Synthetic Biology.\\r\\n\\n',\n",
       "  'award': 'Best Foundational Advance Project;Nominated for Best Foundational Advance Project;Best Wiki;Nominated for Best Wiki;Best Presentation;Nominated for Best Presentation;Best Poster;Nominated for Best Poster;Best Integrated Human Practices;Nominated for Best Integrated Human Practices;Nominated for Best Model;Best Software Tool;Nominated for Best Software Tool;Nominated for Best New Basic Part;Nominated for Best New Composite Part;Nominated for Best Part Collection;Finalist;Nominated for Finalist',\n",
       "  'medal': 'gold'},\n",
       " '2346': {'team_id': '2346',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2346',\n",
       "  'team_name': 'HFLS_H2Z_Hangzhou',\n",
       "  'wiki': 'http://2017.igem.org/Team:HFLS_H2Z_Hangzhou',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Biosolution to Nitrite Degradation in Industrial Pickle Production\\n',\n",
       "  'abstract': 'Pickles have long been an important part in the worldwide diet. But the nitrite generated in the production of pickles has always been a concerning problem to health and food safety, due to its conversion to carcinogenic nitrosamine once taken up by humans. Currently, most industrial production of pickles relies on the natural decomposing process of nitrite, which is inefficient and often takes months to complete. To make an improvement, we designed a bio-device to make the degradation of nitrite more rapid and environmental-friendly. Our device consists of a positive feedback loop device followed by a fusion enzyme which can reduce nitrite to nitrogen and oxygen. The feedback loop is regulated by a nitrite/nitrate specific promoter to ensure the expression of downstream function proteins only in pickles environment. Our solution drastically shorten the production period and increases profit margin for the multibillion-dollar pickled vegetables industry. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2466': {'team_id': '2466',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2466',\n",
       "  'team_name': 'HFUT-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:HFUT-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Software',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'BioDesigner Dolphin\\n',\n",
       "  'abstract': '    We developed BioDesigner Dolphin, an information portal which integrates papers, previous teams and BioBricks information along with various algorithms and designs of assistant devices. Users can search for team information, and they will obtain extra information about similar teams as well. Search results are classified into different categories so users will acquire various information efficiently on project overview, description, team awards and so on. The searching engine of BioDesigner Dolphin also allows users to find gene information and explore gene relationship. Besides information acquisition, users are also able to design biological parts with the help of recommendation. We hope BioDesigner Dolphin will improve the working efficiency of researchers and give inspirations to iGEM teams when deciding theme and methodology of their projects.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2219': {'team_id': '2219',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2219',\n",
       "  'team_name': 'HK_SKHLPSS',\n",
       "  'wiki': 'http://2017.igem.org/Team:HK_SKHLPSS',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Hong Kong',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'A Self-Assembled DNA Nanocube for the Diagnosis of H3N2 Influenza\\n',\n",
       "  'abstract': \"DNA is always used a genetic material for information transfer over generations. It's programmability also allows the application in fabricating various objects for diagnostics and therapeutics as the field of DNA nanotechnology. This year, we used DNA to fabricate a three-dimensional cube that is responsive to the presence of H3N2 influenza mRNA biomarker. We used DNA and RNA oligos of the same sequence of the target for detection. From the result, we found that the DNA nanocube specifically responses to the presence of target leading to the opening of the lid. This results in the disassemble of the quadruplex formation and reduction in hemin-mediated peroxidase activity. We proved that a three-dimensional nanodevice can be used for quick diagnosis within 30 minutes and it is applicable for the detection of different biomarkers and we wish to largely produce this diagnostic device with engineered bacteria.\\n\",\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2394': {'team_id': '2394',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2394',\n",
       "  'team_name': 'HNU_China',\n",
       "  'wiki': 'http://2017.igem.org/Team:HNU_China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': None,\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': ''},\n",
       " '2414': {'team_id': '2414',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2414',\n",
       "  'team_name': 'HokkaidoU_Japan',\n",
       "  'wiki': 'http://2017.igem.org/Team:HokkaidoU_Japan',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Japan',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'E.co Circle\\n',\n",
       "  'abstract': 'In stock-breeding, animals are fed with grains that contain phytic acids though many livestock are not capable of producing phytase, an enzyme that decomposes phytic acid, and remaining phytic acid is excreted in the excrement. Excreted phytic acid flows into rivers and leads to eutrophication that causes problems such as red tide, which exerts adverse effects on ecosystem and fishery.\\r\\nUnder the status quo, people try to cope with this problem by adding phytase to the feed of livestock however, its enzyme activity is thought to be lowered when it is heated during its production. Moreover, the low pH in the stomach of livestock may also lower its activity.\\r\\nIn our project, we aim to increase stability against heat and extreme pH by circularizing enzyme using self-assembling peptides (SAPs) which could decrease the amount of phosphate in the excrement and enhance nutrition absorption since phytic acids chelate minerals.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2254': {'team_id': '2254',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2254',\n",
       "  'team_name': 'Hong_Kong-CUHK',\n",
       "  'wiki': 'http://2017.igem.org/Team:Hong_Kong-CUHK',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Hong Kong',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': ' DR. SWITCH (Disease-associated RNA Switch)\\n',\n",
       "  'abstract': 'Influenza A is a rapid changing disease that causes 5,000,000 of death annually worldwide. Among different subtypes, highly pathogenic avian influenza has the highest mortality rate. Challenges of disease control in the modern world with high population mobility remains at the speed and accuracy of diagnosis. However, nowadays influenza A subtyping method rely greatly on RT-PCR, which requires long time, expertise and laboratory space. Meanwhile, a novel type of riboswitch, namely toehold switch, shows its potential in subtyping Influenza A with quicker detection and lower production cost. Our project focus on developing an on-site subtypting method for Influenza A virus subtype H5N1 and H7N9 using toehold switches. An online software program was also developed for designing toehold switch.\\r\\n\\r\\n\\r\\n \\r\\n\\n',\n",
       "  'award': 'Nominated for Best Diagnostics Project',\n",
       "  'medal': 'gold'},\n",
       " '2202': {'team_id': '2202',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2202',\n",
       "  'team_name': 'Hong_Kong_HKU',\n",
       "  'wiki': 'http://2017.igem.org/Team:Hong_Kong_HKU',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Hong Kong',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Disease Diagnosis Using 3D Functional DNA Nanostructures produced in-vivo\\n',\n",
       "  'abstract': \"DNA nanotechnology utilizes DNA\\x92s chemical properties to produce nanostructures with novel functions. DNA diagnostic nanodevices are produced using the predictable and programmable properties of DNA binding. In our project, we will produce biobricks encoding a 3D nano device which detects diseases, like Huntington's disease and cancers, that produces mRNA biomarkers. The presence of target would induce a conformational change of our nanostructure, which we can be assessed through gel electrophoresis, colorimetric and fluorescence assay. Lastly, we will transform the construct with our biobrick , consisting of our oligos, and reverse transcriptase into E. coli that can facilitate the synthesis of our nanostructure inside the cell. By using E. coli, this will help us to mass produce a diagnostic tool for a specified disease.\\n\",\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2240': {'team_id': '2240',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2240',\n",
       "  'team_name': 'Hong_Kong_HKUST',\n",
       "  'wiki': 'http://2017.igem.org/Team:Hong_Kong_HKUST',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Hong Kong',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Genetic Containment Strategy: Preventing the Replication of unintentionally released Genetically Modified Materials through Recombinase-based Deletion\\n',\n",
       "  'abstract': 'Unintentional release of genetically engineered organisms (GMOs) has always been a major safety concern in the field of synthetic biology. Different approaches of genetic containment strategy such as kill-switches and auxotrophic mutants pose some issues of horizontal gene transfer of modified genes and metabolic cross-feeding that may limit the effectiveness of those approaches. To circumvent the shortcomings, we present an alternative safety circuit which is capable of sensing and amplifying a signalling molecule under a time-delay mechanism. After the time-delay, the circuit will trigger the production of a recombinase that splices out the genetically modified segment from the origin of replication (ORI) of the carrier vector, thus preventing replication of the genetically modified segment in the host. Through this genetic containment approach, we hope to create a standardized design to lower the risk of leakage of genetically modified materials to the environment and alleviate public concerns.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2197': {'team_id': '2197',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2197',\n",
       "  'team_name': 'Hong_Kong_UCCKE',\n",
       "  'wiki': 'http://2017.igem.org/Team:Hong_Kong_UCCKE',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Hong Kong',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'A meter and medication of Gout: uric acid detector, disintegration and deployment\\n',\n",
       "  'abstract': 'Knowing that about 1% population of the world suffers from gout, which causes extreme pain daily, we would like to use genetic engineering techniques to help those patients. Gout is formed due to uric acid crystallizes and accumulated at synovial fluid. Thus, we wish to develop a rapid detection method to confirm whether he is suffering from gout by measuring the concentration of uric acid in their blood. The cure or relieve gout by preventing the increase of uric acid concentration to a certain threshold is also in progress. Several parts have been designed to achieve the above goals. Assays are also designed to test and demonstrate the sensing or curing ability of our parts. \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2527': {'team_id': '2527',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2527',\n",
       "  'team_name': 'HSL_WEST_VIRGINIA',\n",
       "  'wiki': 'http://2017.igem.org/Team:HSL_WEST_VIRGINIA',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': None,\n",
       "  'kind': 'Community Lab',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': ''},\n",
       " '2336': {'team_id': '2336',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2336',\n",
       "  'team_name': 'HUST-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:HUST-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'REEBOT\\n',\n",
       "  'abstract': 'The rare-earth elements (REEs), essential in many high-tech products, are becoming increasingly important. However, mining, refining, and recycling of them may yield hazard by-product if not properly managed. Therefore, we have attempted to apply synthetic biology strategies for its proper usage. Here we have proposed REEBOT, the REE robot, which could sense lanthanide ions by its membrane protein, and then catch those ions with the artificial peptides on its surface. After its enriching process, REEBOT would be adsorbed to silicon with the help of Si-tag, enabling us to recycle it. To enhance its efficiency and specificity, we have also designed an awakening system: only under the presence of rare earth elements could REEBOT start its enriching and recycling system, which could also ensure its safety.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2381': {'team_id': '2381',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2381',\n",
       "  'team_name': 'HZAU-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:HZAU-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Synchronizer-4C: Computer-Controlled Cell Cycles\\n',\n",
       "  'abstract': 'Cell cycle synchronization is highly needed in many fields and the traditional synchronization methods have different limitations; for example, chemical methods usually have severe side-effects on the natural physiological states of cells while physical methods (like cell sorting) cannot sustain long time synchronization. In this project, we proposed a cell synchronization approach based on a synthetic circuit of light-controlled CRISPR system targeted at OriC to block the initiation of chromosome replication in E. coli cells. Under the control of computer, the cell cycles can be synchronized with tuned lightening patterns. Our synthetic biological approach of cell synchronization has multiple advantages than traditional methods and may become a fundamental tool for biological research.\\n',\n",
       "  'award': 'Nominated for Best New Basic Part',\n",
       "  'medal': 'gold'},\n",
       " '2479': {'team_id': '2479',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2479',\n",
       "  'team_name': 'ICT-Mumbai',\n",
       "  'wiki': 'http://2017.igem.org/Team:ICT-Mumbai',\n",
       "  'region': 'Asia',\n",
       "  'country': 'India',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'DyEODORANT: Giving ammonia the blues\\n',\n",
       "  'abstract': 'Ammonia released from hydrolysis of urine is a major reason for the stench in public toilets. Apart from being a put-off, ammonia is also hazardous. Current approaches to tackle this problem include flushing water, using microbes (BioBlocks®) that break down urea in urine to prevent its hydrolysis, and using air fresheners. The first approach requires using copious amounts of water, which is not abundantly available in many parts of the world, while the second is not cost-effective. Finally, spraying air fresheners is not a solution, as it does not get rid of ammonia. We propose to engineer Escherichia coli to assimilate ammonia and convert it into indigoidine, a blue colored compound. Ammonia from air can be dissolved in an aqueous medium and can act as a nitrogen source for the engineered cells that maybe housed in a cassette; the synthesized indigoidine will indicate when the cassette has to be replaced.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2319': {'team_id': '2319',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2319',\n",
       "  'team_name': 'IISc-Bangalore',\n",
       "  'wiki': 'http://2017.igem.org/Team:IISc-Bangalore',\n",
       "  'region': 'Asia',\n",
       "  'country': 'India',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'iFLOAT: a multifaceted approach to cluster bioengineered gas vesicles in vitro and enhance their flotation\\n',\n",
       "  'abstract': 'Gas vesicles (GVs) are hollow protein nanostructures synthesized by phototrophic haloarchaea and cyanobacteria to regulate their flotation in aquatic habitats. Bioengineered GVs have been genetically modified for diverse purposes including ultrasonic molecular imaging, gauging cellular turgor pressures, and vaccine delivery \\x97 harnessing unique acoustic, mechanical, and surface properties of GVs \\x97 but none of their current applications exploits their most fundamental characteristic: buoyancy. Our modelling indicates that clusters of GVs float several orders of magnitude better than individual GVs, as buoyancy scales with volume while Stokes\\x92 drag scales with effective radius. Our project iFLOAT aims to improve the flotation of gas vesicles by clustering them using three distinct methods \\x97 charge-based flocculation, biotin-streptavidin interaction, and SpyCatcher-SpyTag heterodimerization \\x97 and simultaneously develop robust, reproducible flotation assays. Potential future applications of buoyant clusters of bioengineered gas vesicles include bioremediation of oil spills and flotation-based separation and purification of specific targets from mixtures.\\n',\n",
       "  'award': 'Nominated for Best Hardware',\n",
       "  'medal': 'gold'},\n",
       " '2320': {'team_id': '2320',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2320',\n",
       "  'team_name': 'IISER-Mohali-INDIA',\n",
       "  'wiki': 'http://2017.igem.org/Team:IISER-Mohali-INDIA',\n",
       "  'region': 'Asia',\n",
       "  'country': 'India',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'gEco: Paper based bio-synthetic system for detection and capture of noxious gases and phenolic compounds\\n',\n",
       "  'abstract': 'Pollution is a major problem worldwide and more severe in developing countries like India. According to WHO database, more than 80% of the people around the world are exposed to pollutants. These are measured by instruments like GC-FID, PTRMS which are costly and cumbersome to install. Therefore, alternative and cost effective approaches are required. So, we would like to develop a detector and capture system for pollutants by designing synthetic circuit in E. coli. For detecting and reducing/capturing salicylate, we designed a synthetic circuit and simulated it for its feasibility based upon kinetic equations. From simulation results, bacteria is showing color gradation from yellow to green on the basis of levels of salicylate present in the environment. We cloned a part of circuit for producing yellow color in bacteria. The cloning of other part for producing gradation of color and developing it as a paper based system is going on.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2258': {'team_id': '2258',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2258',\n",
       "  'team_name': 'IISER-Pune-India',\n",
       "  'wiki': 'http://2017.igem.org/Team:IISER-Pune-India',\n",
       "  'region': 'Asia',\n",
       "  'country': 'India',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'TB or not TB? A diagnostic tool\\n',\n",
       "  'abstract': 'Cheap and easily available diagnostic tool for tuberculosis is the need of the hour. Problems faced during TB diagnosis include the long culturing time and expensive microscopy. We aim to create an inexpensive device which can be used by diagnosticians on a day-to-day basis without using microscopic equipment.\\r\\nOur project has three modules: The Hijack module will facilitate faster growth of M. tuberculosis by increasing the frequency of oscillations of certain cell cycle proteins. The Detection module will make the bacteria express chromoproteins, which can be seen by the eye. The Termination module will ensure that the fast growing bacteria are killed after they reach a required population using quorum sensing based killer gene expression.\\r\\nThe device will be delivered using a bacteriophage delivery system, and this entire diagnostic process will be carried out in a closed, handy device which we have designed.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2318': {'team_id': '2318',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2318',\n",
       "  'team_name': 'IIT-Madras',\n",
       "  'wiki': 'http://2017.igem.org/Team:IIT-Madras',\n",
       "  'region': 'Asia',\n",
       "  'country': 'India',\n",
       "  'track': 'Software',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'A digital information catalogue of host organisms for synthetic biology, with supportive software tools\\n',\n",
       "  'abstract': 'With regard to  construction of synthetic biological systems, it is observed in several iGEM projects and literature that E. coli could not express quite a number of interesting phenotypes that are native to specific organisms. In such cases it is advantageous to use those organisms and introduce synthetic circuits into them. Hence, it is desirable to have a catalogue of alternative host organisms with specific characters,  and a  database of protocols, parts and software tools to work with them. \\r\\n\\r\\nOur database will consist of a collection of organisms used as hosts, their physical attributes, protocols involved in culturing and transformation, and host-wise listing of biobrick parts and other vectors. It shall serve as a platform for synthetic biologists to exchange all relevant information about host organisms.\\r\\n\\r\\nAlong with the database, a set of necessary computational tools such as codon optimisation will also be developed.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2525': {'team_id': '2525',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2525',\n",
       "  'team_name': 'IIT_Delhi',\n",
       "  'wiki': 'http://2017.igem.org/Team:IIT_Delhi',\n",
       "  'region': 'Asia',\n",
       "  'country': 'India',\n",
       "  'track': 'Information Processing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'BLAST - Basic Logic Assessment and Signalling Tools\\n',\n",
       "  'abstract': 'Lack of digital responses in Synthetic Biology have inhibited the diverse potential that accompanies the digitization of biological circuits. \\r\\n\\r\\nThis year we aim to develop synthetic modules for signal processing in biological systems, in the form of elements of specialized logic gates based on transcriptional regulation. We move from developing near digital logic gates with sharp responses, to more specialized collapsible and reconfigurable circuits which can perform various operations like developing square pulses. Further, to realize this aim of making a square wave generator, we engineered a five node repression based ring network to give digital oscillations. Quantitative computational modelling would be used to tailor the cellular environment and observe period, steepness, noise and amplitude variations. \\r\\n\\r\\nOur project poses to be an integral element in genetic networks intended to solve scientific challenges for years to come, ranging from making light sensitive frequency modulators and bacterial memory storage systems.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2278': {'team_id': '2278',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2278',\n",
       "  'team_name': 'INSA-UPS_France',\n",
       "  'wiki': 'http://2017.igem.org/Team:INSA-UPS_France',\n",
       "  'region': 'Europe',\n",
       "  'country': 'France',\n",
       "  'track': 'Information Processing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Croc\\x92n Cholera\\n',\n",
       "  'abstract': 'Synthetic biology projects are usually based on a unique strain of microorganism. New exciting possibilities are expected to emerge by using synthetic microbial consortia. This way, the most interesting properties of microorganisms (e.g. detection, production, resistance) can be combined and modified to achieve new desired tasks. This is what we did in our iGEM project. As a proof of concept, we create a synthetic microbial consortium to fight against cholera disease. The players are modified strains of (1) Escherichia coli producing a quorum sensing (QS) molecule to mimic the presence of pathogenic Vibrio cholerae bacteria; (2) Vibrio harveyi producing diacetyl in response to QS molecule; and (3) Pichia pastoris yeast producing antimicrobial crocodile peptides in response to diacetyl. By crossing the border of the cell-to-cell communication between prokaryote-eukaryote, our project brings not only new perspectives to circumvent cholera epidemia but also to design new synthetic biology approaches using microbial consortia.\\n',\n",
       "  'award': 'Best Information Processing Project;Nominated for Best Information Processing Project;Nominated for Best Wiki;Nominated for Best Presentation;Nominated for Best Integrated Human Practices;Nominated for Best Education & Public Engagement;Nominated for Best Entrepreneurship;Best Applied Design;Nominated for Best Applied Design;Nominated for Best New Basic Part',\n",
       "  'medal': 'gold'},\n",
       " '2282': {'team_id': '2282',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2282',\n",
       "  'team_name': 'IONIS-PARIS',\n",
       "  'wiki': 'http://2017.igem.org/Team:IONIS-PARIS',\n",
       "  'region': 'Europe',\n",
       "  'country': 'France',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'SofterShock : a thermo-adaptive solution for fighting extreme climatic conditions on crops\\n',\n",
       "  'abstract': 'Recently displaced from its leading position among global wine producers with a 12% recession last year, France suffers from climate changes and their negative impacts on vineyards. The appearance of extreme temperature events threatens the agricultural economy and farmers have to deal with the unsatisfactory current solutions. Using synthetic biology, we are developing a thermo-adaptive biological product \\x93SofterShock\\x94 for protecting grapevines against climatic hazards.\\r\\nWe are engineering a microorganism to make it respond differently according to the temperatures. Below 15°C, anti-freeze proteins will prevent ice-crystal growth and above 37°C, light-reflecting compounds will limit evapotranspiration. Once applied on crops, our solution will possess a double protection: anti-drought and anti-frost.\\r\\nWe elaborate a \\x93killswitch\\x94 strategy to limit microorganism dissemination and DNA transfer. In order to not alter plant biodiversity or wine characteristics, we will select the best chassis to combine with novel properties and bring an innovative approach to global crop protection.\\n',\n",
       "  'award': 'Nominated for Best Food & Nutrition Project;Nominated for Best Education & Public Engagement, Overgrad;Nominated for Best New Composite Part, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2378': {'team_id': '2378',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2378',\n",
       "  'team_name': 'ITB_Indonesia',\n",
       "  'wiki': 'http://2017.igem.org/Team:ITB_Indonesia',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Indonesia',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Dewaruci - Removal of Microplastic Pollution in the Ocean Using Biofilm-based Polyethylene Terephthalate (PET) Degradation\\n',\n",
       "  'abstract': 'Polyethylene terephthalate (PET)-based plastic pollution in the ocean is a very concerning environmental issue. These plastics are extremely difficult to degrade. Moreover, harsh ocean environment breaks down these plastics into tiny fragments called microplastics. Microplastics are commonly ingested by marine life, causing poisoning which could lead to deaths.  While larger-sized plastics are easy to collect for recycling, microplastics are impossible to collect, making them an untreatable pollution. ITB_Indonesia team will create a synthetic bacterium which has the ability to remove plastic pollution from the ocean efficiently. This bacterial machine will do its action in four main steps: 1. detection of microplastics; 2. attachment to the microplastics through biofilm formation; 3. plastic degradation using PETase enzyme; 4. conversion of PET-degradation products into nutrition source. Additionally, this bacterium will be designed to survive marine conditions. Through this breakthrough, microplastic pollution would be treated and marine wildlife would be saved.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2363': {'team_id': '2363',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2363',\n",
       "  'team_name': 'iTesla-SoundBio',\n",
       "  'wiki': 'http://2017.igem.org/Team:iTesla-SoundBio',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Community Lab',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Eliminating PCB pollution in the Puget Sound by genetically modifying E. coli\\n',\n",
       "  'abstract': 'Polychlorinated biphenyls (PCBs) are a class of man-made organic chlorine contaminants. Although their manufacture has been banned, they remain in the environment today. PCBs are probable carcinogens and toxic; cause immune system and thyroid defects; and its biomagnification up the food chain in the Puget Sound has been particularly detrimental to orcas. Though they persist because they are highly nonreactive, it has been known for several decades that PCBs slowly degrade in the environment. Recently, it was discovered that the bacterium Dehalococcoides mccartyi can break them down with a variety of enzymes, the genes for which were sequenced in 2014 by Wang. However, D. maccartyi is anaerobic and obtains energy through organohalide respiration.\\r\\n\\r\\nWe planned to transform these genes into easier-to-work-with E. coli for potential PCB cleanup operations. The end goal was a process using the produced enzymes or technology containing the genetic pathway for use in PCB clean-up operations.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2317': {'team_id': '2317',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2317',\n",
       "  'team_name': 'Jilin_China',\n",
       "  'wiki': 'http://2017.igem.org/Team:Jilin_China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'GeneGuard: A modular plasmid system designed for chlorophenol degradation.\\n',\n",
       "  'abstract': '2017Jilin_China takes biosafety into our major concern. Our research interst has been attracted to Chlorophenol, one important raw material in organic synthesis. Though widely used in industrial and agricultural production, the water pollution resulted from the leakage or non-compliance of the emissions becomes a huge threaten to human health. Therefore, we designed a self-regulated circuit with the protein DmpR as the sensor to identify chlorophenol signals and its derivation, and to initiate expression of the two critical enzymes-- monooxygenase and dioxygenase in two E.coli strains for degradation of chlorophenol in water. Additionally, we added a toxin-antitoxin (TA) system for the self-regulation of E.coli growth. In the absence of pollutants, the engineered bacteria will express toxin that can inhibit their own growth. When exposed to contaminants, the expressing of antitoxin will be initiated and block the toxicity, bringing bacteria back to normal, so that the purification mission will be completed. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2358': {'team_id': '2358',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2358',\n",
       "  'team_name': 'JNFLS',\n",
       "  'wiki': 'http://2017.igem.org/Team:JNFLS',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Executioner of colon cancer\\n',\n",
       "  'abstract': 'Cancer mortality rate is very high. The biggest characteristic of cancer cells is rapid cell division, resulting in the hypoxia microenvironment of solid tumor. Our project is to construct three plasmids, which are transported to the solid tumor using the probiotics Nissle 1917 as gene delivery carrier. When the extracellular environment lacks of oxygen, a plasmid in Nissle 1917 can express invasin and Hyl proteins, making the Nisssle 1917 invade into tumor cells, and avoid cell lysosomes endocytosis. After the Nissle 1917 enter solid tumor cells, another plasmid will express TAT protein, inducing cell apoptosis. The specialty of our project is that if the Nissle 1917 enter the nontumorous cells by mistake, the third plasmid will start suicide program, which will not pose a threat to normal cells, achieving the safety of gene therapy.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2245': {'team_id': '2245',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2245',\n",
       "  'team_name': 'Judd_UK',\n",
       "  'wiki': 'http://2017.igem.org/Team:Judd_UK',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'IonIron - a home-testing kit to keep an eye on your iron levels\\n',\n",
       "  'abstract': 'We are aiming to genetically engineer E.coli to detect iron concentrations in saliva as a biological sensor for iron deficiency and overdose. This would ultimately be developed into a cell-free, paper based system.\\r\\nAccording to the WHO, it is estimated that 1 billion people worldwide as well as over 50% of pregnant women in LEDCs are iron deficient. Iron supplementation programmes exist in 90 countries worldwide; however, most of these are not systematically monitored, implemented or evaluated.  Excess iron supplementation can result in an increased risk of developing chronic conditions such as diabetes and arthritis. \\r\\nMonitoring these conditions requires a time consuming method involving expensive equipment and trained personnel which are difficult to find in LEDCs. \\r\\nOur construct will change colour depending on the iron levels in patients\\x92 saliva samples. This would make our test inexpensive, unobtrusive and easy to use to help monitor these conditions across the globe.\\n',\n",
       "  'award': 'Nominated for Best Wiki, High_school;Best Education & Public Engagement, High_school;Nominated for Best Education & Public Engagement, High_school;Best Model, High_school;Nominated for Best Model, High_school',\n",
       "  'medal': 'sivel'},\n",
       " '2335': {'team_id': '2335',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2335',\n",
       "  'team_name': 'KAIT_JAPAN',\n",
       "  'wiki': 'http://2017.igem.org/Team:KAIT_JAPAN',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Japan',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'CAR: Cure Allergic Rhinitis (Hay Fever in Japan)\\n',\n",
       "  'abstract': 'Hay fever in Japan is most commonly caused by pollen from Cryptomeria japonica and Japanese cypress. Which are native Japanese tree species. About 20% of the population currently suffer from this type of seasonal hay fever in Japan. The general symptoms are, sneezing, runny nose, nasal congestion and itchy eyes. There is an opinion that the balancing of helper T cells which controls the immune system is involved. Helper T cells transmitｓ antigen information to B cells, which are antibody-producing cells. And depending on the type of cytokine produced, it is roughly divided into type 1(Th1) and type 2(Th 2). Our aim is to create a system that produces IL12 to maintain the balance between Th1 and Th2 cells.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2340': {'team_id': '2340',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2340',\n",
       "  'team_name': 'Kent',\n",
       "  'wiki': 'http://2017.igem.org/Team:Kent',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'LuCas: the unique Cas13a mediated mRNA localization tool  \\n',\n",
       "  'abstract': 'Cas13a is a RNA guided endonuclease which degrades RNAs based on alignment of its CRISPR derived guide RNA. To determine the sub-cellular localization of messenger RNA (mRNA) we fluorescently tagged a Cas13a that cannot cleave its target RNA (dCas13a). This was achieved by using four mutations, two in HEPN1 and two in the HEPN2 domains, which although ablating nuclease activity, still permitted dCAS13a to bind the target RNA sequence. In tandem we designed guide RNAs for targeting mRNA of four cytosolic proteins. Visualization of the dCAS13a was achieved by fusion with a GFP, and to enhance target detection, a nuclear localization sequence sequestered the dCas13a-GFP in the nucleus until needed in the cytosol, thus reducing background noise. We have performed verification tests, constructed mathematical models and developed multiple tools for public engagement. Our findings will lead to the development of a new tool for research and potentially in diagnostic laboratories.\\n',\n",
       "  'award': 'Best Poster;Nominated for Best Poster',\n",
       "  'medal': 'bronze'},\n",
       " '2268': {'team_id': '2268',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2268',\n",
       "  'team_name': 'Kingsborough_NY',\n",
       "  'wiki': 'http://2017.igem.org/Team:Kingsborough_NY',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Microbes of the Night\\n',\n",
       "  'abstract': \"In synthetic biology, its essential to implement control mechanisms that disable microbes that are unintentionally released into the enviroment. Teams have developed 'kill-switches', or toxic genes are activated under selected conditions. We decided to create a light-inducible kill-switch, based on the design by Wageningen 2016. This complements our previous project (which we are continuing to explore); enhancing E. coli metabolism for wastewater treatment, in dark conditions. The kill-switch is based upon the pDawn system, activating transcription in the presence of blue light. It features MazF, a mRNA-targeting endonuclease. We have designed controls to verify the expression pattern. Furthermore, we introduced additional layers of regulation to prevent premature cell death. Completion of our previous work, together with our kill-switch, is a step towards an environmentally friendly and safe bacterium for nitrogen removal. Moreover, the creation of an effective light inducible kill-switch is a useful addition to the synthetic biology toolkit.\\n\",\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2233': {'team_id': '2233',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2233',\n",
       "  'team_name': 'Kobe',\n",
       "  'wiki': 'http://2017.igem.org/Team:Kobe',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Japan',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Tea Ceremony Master Bug: Evaluating Tea Quality with a Biosensor for L-theanine\\n',\n",
       "  'abstract': 'The aim of our project is to create a biosensor for evaluating the concentration of L-theanine.\\r\\n \\r\\nTheanine is an amino acid primarily found in particular plants, especially in tea plant (Camellia sinensis). Theanine boosts alpha brain waves, promoting relaxation. In addition, theanine is one of the taste ingredients of green tea, which is called as \\x93umami\\x94 in Japanese, meaning pleasant savory taste. For these reasons, green tea containing the more theanine is considered to have the higher quality. \\r\\n\\r\\nCurrently, the theanine content in green tea is measured by instrumental analysis depending on the expensive equipment. To reduce the cost and special technique required for the measurement, we aimed at developing a bacterial sensor to evaluate the theanine content in green tea based on identification of the genes in Bacillus subtillis that are induced in the presence of theanine.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2437': {'team_id': '2437',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2437',\n",
       "  'team_name': 'KUAS_Korea',\n",
       "  'wiki': 'http://2017.igem.org/Team:KUAS_Korea',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Korea, Republic Of',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'POO-robiotics: Check Your BOwel and rebalance the Gut (CYBOrG)!\\n',\n",
       "  'abstract': 'Everybody poops. A stool analysis is a cheap and useful pre-diagnostic tool for many human diseases by chemical and biological examinations of daily stools. For instance, the colon cancer can be pre-screened by checking the presence of occult blood in feces. However, the procedure of collecting samples and delivering it to the laboratory often makes people cumbersome and therefore causes a delay of the examination. This year at iGEM, team KUAS_Korea presents \\x91POO-robiotics\\x92, a do-it-yourself (DIY) stool analysis at home. We employ Lactobacillus plantarum L67 as a chassis, a probiotic with antiallergic activity and used as a yogurt starter. The designed genetic circuit will convert bowel conditions into color pigments. As a proof of concept, we focused on the screening of occult blood, pH variation, and bile salt residues. After ingesting our \\x91POO-robiotics\\x92, individuals can identify their own bowel conditions at home by simply checking the color of their feces.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2263': {'team_id': '2263',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2263',\n",
       "  'team_name': 'KU_Leuven',\n",
       "  'wiki': 'http://2017.igem.org/Team:KU_Leuven',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Belgium',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'HEKcite: Measuring frequency change of an artificial rhythm in HEK cells for therapeutic drug monitoring\\n',\n",
       "  'abstract': 'Lack of continuous therapeutic drug monitoring is associated with increased health costs, morbidity and mortality. A more dynamic sensing system could improve the therapy and quality of life of patients taking drugs with a narrow therapeutic range. Here, an electrically oscillating system consisting of genetically modified Human Embryonic Kidney (HEK) cells is proposed. After transfecting HEK cells with genes encoding key ion channels, an intrinsic rhythm of subsequent de- and repolarization was obtained. Molecular substrates, such as drugs, can open or close these ion channels and thus affect the frequency of the rhythm. This change in frequency can then be measured in vivo by use of a multi-electrode array and correlated to the concentration of the drugs in the blood. Additionally, substrate specificity can be chosen by integration of a certain substrate-sensitive ion channel into the oscillating system. This multipurpose sensor could be developed for medical and biotechnological applications. \\r\\n\\n',\n",
       "  'award': 'Best New Application Project;Nominated for Best New Application Project',\n",
       "  'medal': 'bronze'},\n",
       " '2403': {'team_id': '2403',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2403',\n",
       "  'team_name': 'Kyoto',\n",
       "  'wiki': 'http://2017.igem.org/Team:Kyoto',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Japan',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'B.xylophilus Busters-the attempt to kill pine-wood nematodes by feeding RNAi\\n',\n",
       "  'abstract': \"Pine-wilt disease caused by Bursaphelenchus xylophilus causes tremendous damage to pines in Asia and Europe. However, the existing counter-measures are not only insufficient, but also cause environmental and ecological damage. \\r\\n We engineered a solution to killing nematodes effectively and specifically by focusing on the nematode's diet: yeast living inside pines. We selected target genes which were both nematode-specific and predicted to be essential for survival. We then engineered yeast which express dsRNA targeting these genes, with the goal of feeding the engineered yeast to the nematodes to knock down their genes (feeding RNAi). Furthermore, we predict that engineered yeast may be used to colonize healthy pines, functioning as a preventive measure against nematode infestation. \\r\\n The behavior of dsRNA in yeast is still not well understood, and establishment of a feeding RNAi system in pine-wood nematodes is unprecedented. Thus, our method will also support fundamental research on RNAi and pine-wood nematodes.\\n\",\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2353': {'team_id': '2353',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2353',\n",
       "  'team_name': 'Lambert_GA',\n",
       "  'wiki': 'http://2017.igem.org/Team:Lambert_GA',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': \"CLiP'd: Characterizing non-Lysosomal Inducible Protein Degradation\\n\",\n",
       "  'abstract': \"In the development of genetic circuits, researchers often face issues with the overlap of protein expression. As a result, the 2017 Lambert iGEM team aimed to develop a clean way to \\x93switch\\x94 off protein expression by further characterizing a proteolytic mechanism known as ClpXP. An inducible genetic construct was made to express tsPurple (a chromoprotein) and degrade via ClpXP upon induction of varying levels of IPTG, resulting in correlating amounts of protein degradation. Data was collected on the team\\x92s engineered Chrom-Q, a 3-D printed camera-device that supports a constant light source for centrifuged cells; in turn the data was analyzed using Lambert iGEM's self-constructed software app to determine HSL values. The purpose and goal for this technology was to promote scientific research under any financial circumstance to quantify data in standardized conditions. Measuring relative strengths of protein degradation using self-engineered products will allow an economic approach in characterizing non-lysosomal proteolysis. \\n\",\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2377': {'team_id': '2377',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2377',\n",
       "  'team_name': 'Lanzhou',\n",
       "  'wiki': 'http://2017.igem.org/Team:Lanzhou',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'A novel method in controling weeds and pests by tandem RNA Interference\\n',\n",
       "  'abstract': 'Weeds and pests are most important damages to the crop yield in the world. Traditional ways like using herbicides and pesticides will easily cause resistance and pollution problems. In addressing this challenge we focus on RNA interference (RNAi), a novel molecular technology that used for gene knockdown, which has been showing great potential in agriculture field especially for insects control, Meanwhile we notice that many weeds are the hosts or intermediate hosts of pests. Based on these two observations, we are aimed at using synthetic biology to control weeds and pests at the same time. We selected model organism Arabidopsis as basic plant verification system which has clear genetic background and field pests Aphidoidea are chosen to be discussed as well in our experiment.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2443': {'team_id': '2443',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2443',\n",
       "  'team_name': 'Lethbridge',\n",
       "  'wiki': 'http://2017.igem.org/Team:Lethbridge',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Next vivo: Cell-Free Synthetic Biology for the Masses\\n',\n",
       "  'abstract': 'We aim to develop a standardized, modular, and open-source ex vivo transcription and translation (TX/TL) system available for research and teaching communities worldwide. Ex vivo systems provide several advantages to cell-based platforms, including: simple input and output, non-proliferation, precise control of molecular interactions, and incorporation of unnatural amino acids. Thus, ex vivo systems are highly useful tools in making synthetic biology accessible to novices, providing enthusiasts with inexpensive cell-free synthesis, and empowering experts with modular control over robust expression systems. With our end-users in mind, we have designed, modelled, and built a plasmid-based multi-protein parts collection for the stoichiometric expression of all required biomachinery for coupled TX/TL reactions, as well as a novel purification method that makes these parts readily available from an inexpensive single-step elution. Lastly, we have created a software tool to preempt biosecurity challenges associated with genetic recoding in ex vivo systems.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2481': {'team_id': '2481',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2481',\n",
       "  'team_name': 'Lethbridge_HS',\n",
       "  'wiki': 'http://2017.igem.org/Team:Lethbridge_HS',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'SynthetINK: environmentally friendly pigment production \\n',\n",
       "  'abstract': 'Printer ink is a commodity that is used on a daily basis. The current pigments used to colour conventional inks are environmentally harmful to produce. Our goal is to produce biological pigments by putting the biosynthesis pathway genes encoding for black, cyan, magenta, and yellow pigments into Escherichia coli. Using synthetic biology to produce pigments may provide a cleaner and safer alternative to conventional pigment manufacturing. Costs of manufacturing methods can be compared and pigment production yields can be modeled mathematically. Pigments purified from bacteria are combined with a solvent and resin to create an ink mixture. Consultation with potential stakeholders indicated that pigments and inks produced by this method would need to be tested for bleaching, bleeding and colour consistency, among other characteristics. Additional human practices include analyzing the environmental impact of standard pigment production versus bacterial production and presenting iGEM and STEAM initiatives to members of our community.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2474': {'team_id': '2474',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2474',\n",
       "  'team_name': 'Linkoping_Sweden',\n",
       "  'wiki': 'http://2017.igem.org/Team:Linkoping_Sweden',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Sweden',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Forgetful Folding\\n',\n",
       "  'abstract': 'Aggregation prone peptides, such as Amyloid Beta and Tau which are found in Alzheimer\\x92s disease tend to form plaque and tangles in the brain. Proteins known as chaperones can help these peptides in the folding process and hinder aggregation. The four chaperones called GroEL, GroES, DnaK and Trigger Factor are going to be studied, both on their own and in different combinations. These chaperones will be overexpressed in Escherichia coli together with Tau and Amyloid Beta fused with the fluorescent proteins eGFP and mNeonGreen. These fusion proteins will also be expressed in different conditions where parameters such as temperature, expression time and induction strength is varied. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2478': {'team_id': '2478',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2478',\n",
       "  'team_name': 'LUBBOCK_TTU',\n",
       "  'wiki': 'http://2017.igem.org/Team:LUBBOCK_TTU',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Measurement',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Characterization and Standardization of a Thermal Actuator and Calcium-Sensitive Reporter in Saccharomyces cerevisiae.\\n',\n",
       "  'abstract': 'The most commonly used input signals in iGEM generally include chemicals and light. Temperature-based input signals are not used as frequently and remain a reservoir of potential for future synthetic biology tools. Our project explores the characterization and standardization of thermosensitive ion channels, known as TRPV1, to function as a thermal actuator and regulate calcium-sensitive gene expression in Saccharomyces cerevisiae. At the activation temperature, gating of the thermal actuator allows for an influx of extracellular calcium ions, which initiate the calmodulin-calcineurin signaling pathway, and promote the expression of genes regulated by Crz1p-sensitive promoters. The characterization of the thermal actuator was conducted in S. cerevisiae because yeast offers an accessible eukaryotic chassis for other iGEM teams to work with. By varying the temperature, the thermal actuator can be used to control genetic circuits that function as temperature-sensitive kill switches, drug delivery devices, or biosensors.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2266': {'team_id': '2266',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2266',\n",
       "  'team_name': 'Lund',\n",
       "  'wiki': 'http://2017.igem.org/Team:Lund',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Sweden',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Construction of a tripartite split-GFP biosensor for detection of microplastics\\n',\n",
       "  'abstract': 'In light of the recent discussions regarding the realisation of the UN development goal number 14 concerning the conservation and sustainable use of marine resources, iGEM Lund has set out to aid in the world-wide engagement against microplastic pollution. Microplastic pollution is an ever-growing problem that stems from wasteful lifestyles with propagating adverse effects throughout the entire food chain. Upwards of 10 million metric tons of plastic has been estimated to enter the ocean every year. Participating in the iGEM competition for the first time, team Lund intends to rapidly and accurately determine the presence of microplastics through the design and implementation of a genetic circuit into E. coli. A logic AND-gate will be constructed utilising the conformational change of a heterologously expressed hER-alpha to associate a split fluorescent reporter. \\n',\n",
       "  'award': 'Nominated for Best Model, Overgrad',\n",
       "  'medal': 'bronze'},\n",
       " '2300': {'team_id': '2300',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2300',\n",
       "  'team_name': 'Macquarie_Australia',\n",
       "  'wiki': 'http://2017.igem.org/Team:Macquarie_Australia',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Australia',\n",
       "  'track': 'Energy',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'HydroGEM - Producers of Pollution-Free Energy\\n',\n",
       "  'abstract': 'To seek a clean and sustainable fuel resource, we sought to imitate the most efficient natural mechanism for the transduction of energy - photosynthesis. By engineering the photosynthesis system into E.coli, \\x91green\\x92 hydrogen fuel can be produced simply from sunlight and water. To do this, we focused on the assembly of the hydrogenase enzyme complex, the final component of the photosynthesis pathway. We successfully added an [FeFe] Hydrogenase enzyme, from Chlamydomonas reinhardtii, ferredoxin, ferredoxin NADP+ reductase (FNR) and maturation enzymes (HydEFG), which together, work cohesively to produce our desired \\x91green\\x92 hydrogen gas product. Our modeling and human practice approaches provided an assessment of the viability of the production of hydrogen on an industrial scale. We have also developed strategies to educate future generations about the potential applications of synthetic biology for providing alternative green energy solutions which can address the global energy crisis.\\r\\n\\r\\n\\n',\n",
       "  'award': 'Best Energy Project;Nominated for Best Energy Project;Nominated for Best Entrepreneurship, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2213': {'team_id': '2213',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2213',\n",
       "  'team_name': 'Manchester',\n",
       "  'wiki': 'http://2017.igem.org/Team:Manchester',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Phosphostore\\n',\n",
       "  'abstract': 'Phosphate supplementation is essential for maximizing plant yields in many agricultural settings. However, phosphate runoff from fields often causes eutrophication, leading to the deterioration of aquatic ecosystems.  Moreover phosphate rock is a finite and increasingly scarce resource. Together, these two issues suggest a closed-loop solution. IGEM Manchester aims to address this issue using a bioengineered device: Phosphostore, genetically-modified bacteria that accumulate phosphate from wastewater in protein-based microcompartments for future recycling. \\r\\nIn interviews with a wide range of stakeholders, we explore the economic and regulatory constraints that would influence the implementation of this system in real-world water treatment plants, as well as utilize a Design of Experiments approach to design a system with the required properties (e.g. the correct shell protein ratios for proper microcompartment formation) that would make our solution feasible on a larger scale.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Environment Project, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2238': {'team_id': '2238',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2238',\n",
       "  'team_name': 'ManhattanCol_Bronx',\n",
       "  'wiki': 'http://2017.igem.org/Team:ManhattanCol_Bronx',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Energy',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'E(lectro) coli and the GOxLDEN nANODE\\n',\n",
       "  'abstract': 'We are maximizing the efficiency of a bioanode in a three-fold approach. First, we stably express the MtrCAB operon from Shewanella oneidensis, an electric bacterium, in E. coli. MtrCAB is responsible for the production of membrane bound cytochromes and known to generate bacterial nanowires. We anticipate that the MtrCAB system will allow for an electric E. coli that produces nanowire connections between the anode and bacterium for direct electron transfer. Second, we utilize variants of the Aspergillus niger enzyme, glucose oxidase (GOx), that are engineered for stability and affinity to gold (via the addition of thiol fusion tags). Finally, we synthesized a gold nanowire anode to increase the surface area for GOx affinity, bacterial nanowire connections and electron deposition. We envision a system that utilizes each approach concurrently in an effort to increase the electron shuttling of a biofuel cell at the anode. \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2470': {'team_id': '2470',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2470',\n",
       "  'team_name': 'McMasterU',\n",
       "  'wiki': 'http://2017.igem.org/Team:McMasterU',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'An RNA-cleaving fluorogenic DNAzyme probe for simple detection of bacterial pathogens\\n',\n",
       "  'abstract': 'Current point-of-care diagnostics for bacterial infections are costly, time-consuming, and offer limited strain specificity. These challenges contribute to improper antibiotic usage, accelerating the propagation of antimicrobial-resistant (AMR) bacteria. We are developing DNAzymes - catalytically active ssDNA generated via in vitro selection - to serve as inexpensive and sensitive probes for the rapid detection of AMR bacteria. In the presence of targeted strains, the DNAzyme cleaves a fluorophore-RNA-quencher motif at the RNA site, generating fluorescence. As a proof-of-concept, we have adapted a known E. coli K12 DNAzyme for use in a plate-based assay, and are generating a novel DNAzyme to detect resistant strains of C. difficile. Simultaneously, we are leveraging machine learning techniques to predict potential DNAzymes, and are developing kinetic models to describe DNAzyme behaviour. Our project addresses the need for novel approaches within AMR detection and active antimicrobial stewardship \\x96 issues widely recognized by the experts in this field.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2493': {'team_id': '2493',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2493',\n",
       "  'team_name': 'McMaster_II',\n",
       "  'wiki': 'http://2017.igem.org/Team:McMaster_II',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'C12 Mediated Cancer Treatment: Dual Functionality as a Cytotoxic and Signalling Agent\\n',\n",
       "  'abstract': 'Current chemotherapeutic agents trigger apoptosis non-specifically and induce cell death within healthy tissues. We propose a novel therapeutic using a bacterial vector that expresses acyl-homoserine lactone C12, the Pseudomonas aeruginosa signalling molecule known to induce cancer cell apoptosis through a Bcl-2 independent pathway. Upon detection of hypoxia and low pH, known tumour microenvironment markers, C12 will be released by the bacteria containing our genetically engineered circuit to act as: (1) a cytotoxic agent that induces cancer cell death, and (2) a signalling molecule that coordinates actions between bacterial colonies. Furthermore, the quorum sensing properties of C12 have been manipulated to activate a kill mechanism that arrests bacterial growth upon destruction of the tumour and the ensuing loss of its microenvironment. Thus, we propose constructing a genetic circuit that not only removes tumours, but also self-regulates itself to mitigate the risk of bacterial infection in the absence of cancerous cells.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2395': {'team_id': '2395',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2395',\n",
       "  'team_name': 'Melbourne',\n",
       "  'wiki': 'http://2017.igem.org/Team:Melbourne',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Australia',\n",
       "  'track': 'Manufacturing(P)',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': 'StarScaffold: A novel method for accelerating biosynthetic pathways in vivo\\n',\n",
       "  'abstract': 'Proteins have been well-attested as natural scaffolds, with notable examples including histones compacting DNA, glycogenin acting as the core for the polymer glycogen, and AKAP co-localising kinases, phosphatases, and substrates at the correct intracellular location. We have furthered this idea to design a four-pointed star-shaped scaffold protein using a single disulphide bond, as well as split inteins which can then be associated with different enzymes to increase modularity and allow a greater application potential. We have improved on the design by Melbourne iGEM 2016 by introducing more stable linkers between inteins and adjusting the DNA sequence to have greater accuracy and precision in expression and purification. We also propose a wide range of possible applications, specifically with respect to increasing the rate of synthesis of chemicals critical in physiological homeostasis, including dopamine, vitamin D and mevalonate.\\r\\n\\n'},\n",
       " '2301': {'team_id': '2301',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2301',\n",
       "  'team_name': 'Michigan',\n",
       "  'wiki': 'http://2017.igem.org/Team:Michigan',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Thermolyze: A Temperature Controlled Kill-Switch for Containment of Pathogenic Bacteria in Research Labs\\n',\n",
       "  'abstract': 'Genetically modified and/or pathogenic bacteria used in research pose a concern to the public due to the potential consequences if they escape into the environment. Efforts to address this have led to the design of bacterial kill switches: biocontainment systems pairing environmental sensing mechanisms with circuit-based control of viability. Though successful switches exist, most require specific molecular signaling mechanisms, limiting their use. We designed a temperature-controlled kill switch activated below 34C using the temperature-dependent repressor TlpA36. Bacteria are lysed by constitutively expressed holin and endolysin unless counteracted by antiholin, which is under the control of TlpA36 and therefore only expressed in a narrow temperature range. This system should function with any gram-negative bacteria. We implemented our switch in E. coli as proof of concept. Promoters of varying strengths were tested to fine-tune the time until lysis, giving researchers a buffer period to work with cells at low temperatures.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2274': {'team_id': '2274',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2274',\n",
       "  'team_name': 'Michigan_Software',\n",
       "  'wiki': 'http://2017.igem.org/Team:Michigan_Software',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Software',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'ProtoCat - Collaborating Towards Success\\n',\n",
       "  'abstract': 'Choosing apt and reliable protocols for new experiments is a problem that wet labs routinely face due to the difficulty in anticipating which protocols will produce the best results. Experimental practices may differ immensely across laboratories and precise details of these practices may be lost or forgotten as skilled faculty or students leave the lab to pursue other endeavors. Furthermore, there are few well-defined protocols that are generally agreed upon by the scientific community, in part due to the lack of a system that can supply a measure of a protocol\\x92s acceptance. In order to address these problems, we set out to build a database that integrates a crowd-sourced ratings and comments system to serve as a protocol curator that enables lab investigators to compare various protocol efficacies, quantify a protocol\\x92s acceptance within the scientific community, and provide an avenue through which experiential knowledge can be communicated and shared.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2230': {'team_id': '2230',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2230',\n",
       "  'team_name': 'Mingdao',\n",
       "  'wiki': 'http://2017.igem.org/Team:Mingdao',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Taiwan',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Sugar Crush \\x97  Probiotic-based glucose retrieval system\\n',\n",
       "  'abstract': '    According to research, in 2015, about 7 adults in 10 suffer the problem of overweight or even obesity, and it is expected that more people are going to get into this kind of unhealthy problem. One of the most major factors that lead to obesity is \\x93sugar\\x94, which could be easily found in beverages, dessert, and almost any kind of food. Thus, it is quite difficult not to absorb too much amount of sugar in our daily lives.\\r\\n\\r\\n    In hope of preventing our bodies from absorbing too much glucose, our team aims to engineer bacteria that absorb glucose more efficiently. Our team constructed glucose active transporters on the membrane of E.coli, enabling the glucose uptake to become faster than intestine cells. On the other hand, our team has designed a suicide circuit ensuring the bacteria don\\x92t absorb all the sugar and the body can get adequate amount of sugar.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Integrated Human Practices, High_school;Nominated for Best Education & Public Engagement, High_school;Best Applied Design, High_school;Nominated for Best Applied Design, High_school;Best New Basic Part, High_school;Nominated for Best New Composite Part, High_school',\n",
       "  'medal': 'gold'},\n",
       " '2375': {'team_id': '2375',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2375',\n",
       "  'team_name': 'Minnesota',\n",
       "  'wiki': 'http://2017.igem.org/Team:Minnesota',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'The Use of Cytolysin FitD Overexpression and Biocontainment Systems for the Control of Zebra Mussels\\n',\n",
       "  'abstract': 'Zebra mussels have a large impact locally in Minnesota as well as nationally. Zequanox is a new effective treatment composed of heat-killed Pseudomonas fluorescens, and it is very expensive. The Minnesota iGEM team has proposed a novel synthetic system that is based upon a toxin released by this Pseudomonas fluorescens strain, cytolysin FitD, that could be released into lakes. This designed system entails the creation of a modified Escherichia coli strain that expresses FitD and a biomolecular control system to allow for the release of live bacteria into the lake. With the continuous production of the FitD toxin, an increased number of zebra mussels will be killed, limiting the number of treatments needed. Minnesota iGEM experimented with several biocontainment strategies, including a thymidine auxotrophy system, purine auxotrophy system, SacB control system, and toxin activation system. These control mechanisms help to create robust and safe engineered biological systems for environmental applications.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2362': {'team_id': '2362',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2362',\n",
       "  'team_name': 'Missouri_Rolla',\n",
       "  'wiki': 'http://2017.igem.org/Team:Missouri_Rolla',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Detecting groundwater and soil pollutants using plant biosensors\\n',\n",
       "  'abstract': 'Plant-based biosensors have immense benefits over analytical chemistry or potentiometric techniques because they continuously sample a large volume of the environment, provide warning to laypeople, and achieve the amazing specificity and sensitivity of biomolecules. \\r\\nWe are developing two approaches to biosensing contaminants with plants. Both systems are based on important developments in biosensors, namely the creation of synthetic signal transduction systems in bacteria and plants and the redesign of natural periplasmic binding proteins for the detection of new ligands. Taken together, these advances could allow a computationally-designed periplasmic binding protein which binds a contaminant of interest extracellularly to transfer the signal through a phosphorylation cascade and produce a transcriptional response. We will create circuits to implement these synthetic signal transduction systems, attempt to computationally design periplasmic binding proteins for new ligands, and test the efficacy of our two biosensing approaches.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2429': {'team_id': '2429',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2429',\n",
       "  'team_name': 'MIT',\n",
       "  'wiki': 'http://2017.igem.org/Team:MIT',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Splice and Dice: Artificial Control of Alternative Splicing via RNA Binding Proteins\\n',\n",
       "  'abstract': 'Alternative splicing is the eukaryotic mechanism that selects which exons will be included in processed mRNA. It increases the diversity of the human proteome by allowing more than one protein isoform to be produced from the same gene. The inclusion or exclusion of particular exons is regulated by splicing factors which bind to specific motifs in the pre-mRNA. Our project aims to control alternative splicing by using the RNA-binding proteins dCas13a and Ms2 to interfere with these splicing factors\\x92 binding, allowing us to control which exons are included in the final processed mRNA.\\r\\n\\r\\nBy choosing between different protein isoforms, this control method may allow synthetic genes to dynamically change functionality. It also has therapeutic potential for diseases that arise from aberrant splicing, including an aggressive form of breast cancer where one splice variant produces a nonfunctional tumor suppressor, contributing to tumor formation.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2488': {'team_id': '2488',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2488',\n",
       "  'team_name': 'Moscow_RF',\n",
       "  'wiki': 'http://2017.igem.org/Team:Moscow_RF',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Russian Federation',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Community Lab',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Phytases piggy bank\\n',\n",
       "  'abstract': 'The project is aimed at solving the problem of thermal destruction of phytase of  Citrobacter braakii during production of granulated compound feeds for farm animals. As producers serving as capsules for phytase molecules and protecting them from high temperatures we will use Yarrowia lipolytica. In order to protect phytase from degradation when affected by enzymes of yeast cells we will modify the protein attaching another protein \\x96 phytochelatin \\x96 to one of the ends of the protein chain. Phytochelatin has a very stable and compact structure resistant to yeast enzymes and will serve as a barrier between phytase and such destructive enzymes.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2405': {'team_id': '2405',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2405',\n",
       "  'team_name': 'MSU-Michigan',\n",
       "  'wiki': 'http://2017.igem.org/Team:MSU-Michigan',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Biosensing Water Contaminants with Genetically Engineered Shewanella oneidensis MR-1 using Single-Chambered Bioelectrochemical Systems\\n',\n",
       "  'abstract': 'Pollutants in fresh water such as pharmaceuticals, hormones and heavy metals are rarely monitored and the need to detect and remove these compounds in an inexpensive way is what motivates this project. The marine bacterium Shewanella oneidensis MR-1 could be a part of the solution through its unique ability to transport electrons to an external acceptor such as an anode through its external electron transport chain (Mtr pathway). This can be utilized in bioelectrochemical systems to make a biosensor by removing the mtrB gene which allows electricity production then turning this gene back on in the presence of water contaminants. This biosensor will be engineered to be manufactured on a large scale to be used for research, education, humanitarian efforts and even consumer use. Although the proof of concept is currently tested in a single chambered bioelectrochemical system, an affordable and portable paper microbial fuel cell system is being developed.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2323': {'team_id': '2323',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2323',\n",
       "  'team_name': 'Munich',\n",
       "  'wiki': 'http://2017.igem.org/Team:Munich',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'CascAID ( Cas13a Controlled Assay for Infectious Diseases )\\n',\n",
       "  'abstract': 'The ongoing crisis of increasing antibiotic resistance demands innovative preventive strategies. Recently, the RNA-targeting protein CRISPR-Cas13a has been used for highly sensitive DNA and RNA detection, promising diverse applications in point-of-care diagnostics. We integrated Cas13a in the detection unit of CascAID, our GMO-free diagnostic platform. CascAID combines an automated microfluidic device for rapid lysis and extraction of nucleic acids with a paper-based readout system. We demonstrated the performance of our device by targeting the 16S RNA from E. coli. We improved the detection limit of our platform, using simulations to optimize our amplification scheme and the final readout. Conceived as a distributable platform for rapid point-of-care diagnostics, CascAID can be used to distinguish between bacterial and viral infections, thus minimizing the widespread use of antibiotics. Furthermore, Cas13a allows the fast design of target sequences, making our system adaptive to the emergence of new viral outbreaks or fast mutating pathogens.\\r\\n\\n',\n",
       "  'award': 'Best Diagnostics Project, Overgrad;Nominated for Best Diagnostics Project, Overgrad;Nominated for Best Wiki, Overgrad;Nominated for Best Presentation, Overgrad;Nominated for Best Poster, Overgrad;Nominated for Best Integrated Human Practices, Overgrad;Nominated for Best Education & Public Engagement, Overgrad;Best Model, Overgrad;Nominated for Best Model, Overgrad;Nominated for Best Measurement, Overgrad;Best Applied Design, Overgrad;Nominated for Best Applied Design, Overgrad;Best Software Tool, Overgrad;Nominated for Best Software Tool, Overgrad;Best Hardware, Overgrad;Nominated for Best Hardware, Overgrad;Nominated for Best New Composite Part, Overgrad;Nominated for Best Part Collection, Overgrad;Finalist, Overgrad;Nominated for Finalist, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2256': {'team_id': '2256',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2256',\n",
       "  'team_name': 'Nagahama',\n",
       "  'wiki': 'http://2017.igem.org/Team:Nagahama',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Japan',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Design of nutritious  food based on \\x93Funazushi\\x94\\n',\n",
       "  'abstract': 'Our team wants to make nutritious foods that can be stored for a long time to help people suffering from hunger.\\r\\nStarvation is caused by the fact that not only calories but also nutrients such as protein are short.\\r\\nTherefore, we propose nutritious food that can be stored for a long time by fermentation mainly using fish which is a protein source which does not cost money to raise.\\r\\nWe focused on the fermented food called Funazushi which rooted in the area where we live in.\\r\\nWe will develop nutritious food by recombining yeast which was dominant species of Funazushi and making yeast produces nutrients lacking in the area.\\r\\nBecause Funazushi is high salt concentration and low pH state during fermentation ,we will try to create yeast which make necessary nutrients for nation which is troubled by starvation while have resistance to high salt concentration and low pH.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2257': {'team_id': '2257',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2257',\n",
       "  'team_name': 'Nanjing-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:Nanjing-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Three Microbial Whole-Cell Sensing Systems to Detect Gas Molecules in the Environment\\n',\n",
       "  'abstract': 'Formaldehyde, hydrogen sulfide and hydrogen pose great danger to our lives. Traditional methods for detecting them are mainly based on irreversible redox reactions in expensive devices with short life spans, whereas whole-cell bacterial biosensors have major advantages over traditional analyses with regard to specificity, sensitivity and portability.\\r\\n\\r\\nWe attempt to develop whole-cell systems detecting these gases. We have managed to transfer gene clusters encoding translational suppressors or activator into E. coli to respond to different gas molecules by up regulation of specific downstream genes. In order to visualize the gases\\x92 existence and concentration, FP genes are inserted in the expression system after the promoter influenced by the suppressor or activator. At present, results have shown sensibility at a µM level.\\r\\n\\r\\nAs long as proper containers are designed, we will be able to construct portable sensors by which users can perform instant on-site monitoring with higher sensitivity, stability but lower costs. \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2209': {'team_id': '2209',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2209',\n",
       "  'team_name': 'Nanjing_NFLS',\n",
       "  'wiki': 'http://2017.igem.org/Team:Nanjing_NFLS',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Fortified Aniline Killer - the Identification, Recombination, and Verification of Aniline Compound Degradation Genes\\n',\n",
       "  'abstract': 'The biodegradation solution to toxic substances in the environment has received general consent and considerable attention. Our project aims at the biodegradation of toxic aniline and its compounds (a type of chemical raw material and the intermediate metabolite of aniline herbicides). Currently the spectrum for aniline biodegradation genes is too narrow to effectively recover polluted soil and waterbody environments. Based on previously discovered relevant genes, we attempt to search for crucial genes for aniline degradation in Acinetobacter sp. strain YAA, and perform gene recombination and expression, meanwhile construct recombinant genetic engineering bacteria, and test the bioremediation effects in contaminated waterbodies and soil samples. Results show that recombined target gene could efficiently express in E. coli, and that the engineered bacteria acquired from three-parent hybridization express a wider range of substrate spectrum, thus laying the foundation for further bioremediation research.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2365': {'team_id': '2365',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2365',\n",
       "  'team_name': 'NAU-CHINA',\n",
       "  'wiki': 'http://2017.igem.org/Team:NAU-CHINA',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Wheat Guard system\\n',\n",
       "  'abstract': 'Fusarium head blight（FHB）is one of the most destructive global disease of cereal，which poses a significant threat to the wheat production and the safety of food. Fusarium Graminearum predominates among several Fusarium species that can cause FHB. Deoxynivalenol (DON) and zearalenone (ZEN) compounds, produced by Fusarium that are common contaminants which pose a threat to human and animal health. So the main purpose of our project is to prevent FHB，not only inhibit growth of the F. graminearum，but also degrade DON and ZEN . We choose to have mass production of biological pesticides\\x97antifungal peptide using yeast system and engineer Yeast-based biosensor to detect. Considering biosafety，our project also includes a kill switch, which is responsible for preventing gene contamination. The mammalian Bax protein will confer a lethal phenotype when expressed in yeast. This reassures that the yeast removes itself when it was exposed to the environment.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2348': {'team_id': '2348',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2348',\n",
       "  'team_name': 'NAWI_Graz',\n",
       "  'wiki': 'http://2017.igem.org/Team:NAWI_Graz',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Austria',\n",
       "  'track': 'Information Processing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'ColiBot \\x96 a robot-bacteria interface\\n',\n",
       "  'abstract': 'The aim of project \\x93ColiBot\\x94 is to create a robot-bacteria interface in which information processing is done by a bacterial culture that forms a feedback loop with a mobile robot.\\r\\n\\r\\nEscherichia coli is cultivated in a bioreactor, which provides stable conditions. Communication from bacterial cells to robot, the output signal, is achieved by fluorescence proteins. The expression of those proteins is influenced by certain environmental conditions. For measurement, a small culture sample will be transported through a modular system to a measurement chamber that detects the fluorescence wavelengths of the culture. This procedure is repeated over and over again, while the robot moves accordingly through an arena. \\r\\n\\r\\nOur first approach of this project is based on thermosensitive bacteria for information processing. Until now, this project already shows satisfactory results. \\r\\nNow, we are primary focusing on the second part of our project, trying to achieve information processing by pH shift. \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2275': {'team_id': '2275',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2275',\n",
       "  'team_name': 'NCKU_Tainan',\n",
       "  'wiki': 'http://2017.igem.org/Team:NCKU_Tainan',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Taiwan',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'NO Problem- A Biological Approach to Nitrate Sensing and Regulation\\n',\n",
       "  'abstract': 'Aquaculture is expected to become major fishery consumption for human upon the depletion of ocean resources, and to reach USD 202 billions by 2020 in a commercial sight. Keeping water unpolluted is crucial to aquaculture industry, with Nitrate the main target when monitoring water condition. Nevertheless, as to control its concentration, changing water continuously, the most common and affordable way for fish-farmers, is undoubtedly a burden for earth. So how to make monitoring and controlling of Nitrate inexpensive, precisely sensitive and with less manpower? Now, NCKU_Tainan brings a new idea \\x97 \"NO Problem: A Biological Approach to Nitrate Sensing and Regulation.\" With a plasmid composed of  PyeaR promoter and GFP for nitrate sensing, genes constructed for four enzymes \\x97 by which Nitrate transformed into glutamine, a harmless and valuable amino acid \\x97 and our own device for both sensing and regulation process, we turn traditional aquaculture into a sustainable industry.\\n',\n",
       "  'award': 'Best Environment Project;Nominated for Best Environment Project',\n",
       "  'medal': 'gold'},\n",
       " '2262': {'team_id': '2262',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2262',\n",
       "  'team_name': 'NCTU_Formosa',\n",
       "  'wiki': 'http://2017.igem.org/Team:NCTU_Formosa',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Taiwan',\n",
       "  'track': 'Software',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Parabase \\x96 A Simple and Applicable Peptide Prediction System with Validation of Artificial Intelligence\\n',\n",
       "  'abstract': 'In the era of explosive information, the power of using a huge quantity of information is pivotal, so comes in the genesis of Parabase, which integrates A.I. based on Scoring Card Method into databases to achieve drug repurposing by the cross match of collection data and the quick prediction of the unknown in vast data sets. The system can be highly applicable in different topics, and we take fungal diseases for example. We have discovered new antifungal peptides and done experiments to validate their functions. After the antifungal peptides are set, IoT then gathers weather information responsible for the changing situation of reality and tallies out the spore germination prediction model. With the Parabase system composed of Database and IoT, farmers can easily opt for the antifungal peptides to eradicate the pathogens and the exact spraying timing.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2302': {'team_id': '2302',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2302',\n",
       "  'team_name': 'NEFU_China',\n",
       "  'wiki': 'http://2017.igem.org/Team:NEFU_China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Coordinated Grease Eraser: Build a microbial trap for grease\\n',\n",
       "  'abstract': '    Civil sewage contains enriched grease with low biodegradability. To develop a novel approach for grease treatment, we designed a microbial cooperation system based on serine-mediated attraction and leucine-mediated repellent of bacteria.\\r\\n\\r\\n    Our system consists of 3 Leaders (LA, LB, LC) and 2 Followers (FD, FE) that are engineered microorganisms. LA inducibly secretes serine, LB constitutively secretes leucine, and LC makes acyl-CoA synthetase long chain (ACSL) to metabolize fatty acids. Serine and leucine produced by Leaders can attract or repel FD/FE, respectively. FD secretes lipase to dissolve sewage grease into fatty acids and FE expresses a fatty acid binding protein on its membrane to carry fatty acids. When Followers are attracted to Leaders, a fatty acid-enriched microenvironment can be built to let ACSL from LC metabolize fatty acids.\\r\\n\\r\\n    Thus, we developed a novel grease-degrading system consisting of differentially engineered microbial groups, similar to an assembly line in a human society.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2288': {'team_id': '2288',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2288',\n",
       "  'team_name': 'NEU-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:NEU-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'iSmeller: An intensified cellular odor sensor system based on CRISPR activation technology\\n',\n",
       "  'abstract': 'Odor molecule can be sensed via the specific binding with its cognate olfactory receptors and the following elicited signaling cascade within the cell. Our research aims to build a mammalian cell-based biological odor detection system (iSmeller, intensified Smeller) with enhanced sensitivity by employing the CRISPR/Cas9 activation technology. Once an odor binds to the specific olfactory receptors on the cell membrane, it leads to a series of signaling rally and consequently generates a flux of cAMP (cyclic AMP) which could be detected by a cAMP-activated reporter gene system. We implement CRISPR activation apparatus to simultaneously amplify several core components endogenously along the olfactory signal cascade to enhance power and sensitivity of the cellular odor sensor. We will evaluate the performance of iSmeller with two odor/receptor pairs: β-citronellol/OR1A1 and bourgeonal/OR1D2. The iSmeller holds great promise in various applications such as cancer diagnosis, environmental pollution evaluation and food quality control. \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2205': {'team_id': '2205',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2205',\n",
       "  'team_name': 'Newcastle',\n",
       "  'wiki': 'http://2017.igem.org/Team:Newcastle',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Sensynova - a new era of biosensors\\n',\n",
       "  'abstract': 'Biosensor applications are wide-reaching, but their development is complex, inefficient, and uptake limited. We propose a new foundational paradigm for biosensor development; a modular, multicellular development platform, Sensynova. Biosensor designs were reviewed, identifying design-patterns of commonly used subcomponents and configurations. Separating biosensor subcomponents as modules, in different bacterial cells, promotes \\x92off-the-shelf\\x92 reuse. New sensors are developed simply by mixing proportions of these cells which communicate using small molecules. A synthetic biology dialogue helped derive designs for new modules (adaptors, detectors, processors, and reporters), and a proof-of-concept system. Cell-free, automation, developer requirements, target-specific knowledge, robustness, legislation, and end-user uptake were also investigated. \\n',\n",
       "  'award': 'Nominated for Best Poster, Overgrad;Nominated for Best Education & Public Engagement, Overgrad;Nominated for Best Measurement, Overgrad;Nominated for Best Part Collection, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2473': {'team_id': '2473',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2473',\n",
       "  'team_name': 'NIPER-Guwahati',\n",
       "  'wiki': 'http://2017.igem.org/Team:NIPER-Guwahati',\n",
       "  'region': 'Asia',\n",
       "  'country': 'India',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Development of synthetic Bio-conjugates for targeted down-regulation of Oncogenes.\\n',\n",
       "  'abstract': 'Our iGEM-2017 work involves evolution of Bcl-2 mRNA-binding peptides and DNAzymes for Bcl-2 mRNA cleaving activity, which can eventually down regulate the expression of Bcl-2 anti-apoptotic protein in a very specific manner.\\r\\n\\r\\nThe second part of the project is to evolve cell permeating peptides specific for skin cancer cells (A375) through phage-surface display random peptide library. We will eventually fuse the novel Bcl-2 mRNA binding peptides and mRNA cleaving DNAzymes with Cell-specific Cell Permeating Peptides (CPPs) to develop Bioconjugates which can target cancer cell through the CPP and inhibit the Bcl-2 protein expression through the novel Bcl-2 mRNA binding peptide and mRNA cleaving DNAzyme to induce apoptosis of the cancer cells.\\r\\nThe work has a special significance as we try to boost the global efforts of developing \"Complete\" Drug molecules/Bio-pharmaceuticals; which can inhibit the target (Drug Action), guided by a CPP to Deliver the Oncogene-Inhibitor inside Cancer cells (Drug Delivery).\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2208': {'team_id': '2208',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2208',\n",
       "  'team_name': 'NJU-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:NJU-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'FFF(free from fat)\\n',\n",
       "  'abstract': 'Obesity has become increasingly disturbing and common these years. However, a perfect treatment of obesity has not appeared so far. The goal of our project is to develop a strategy to treat obesity, with building a transplantable system targeting a specific molecule that functions in white fat tissue. We packed siRNA into exosomes (nano-sized vesicles secreted by human cells). Then we modified our exosome with a certain peptide to act as white-fat-tissue-specific targeting tool. The siRNA will function to bring out the apoptosis of fat tissue. Our validation experiments will be carried out at the level of cells and animals (mice) to prove both the targeting and function of siRNA. Eventually, we expect to see a specific accumulation of the siRNA in the mice\\x92s white fat tissue and the decrease of mice\\x92s fat and their weights. This project may provide new insights into future treatment of obesity.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2280': {'team_id': '2280',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2280',\n",
       "  'team_name': 'NKU_China',\n",
       "  'wiki': 'http://2017.igem.org/Team:NKU_China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Energy',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Traffic Police in the Reservoir -- engineered microbes that control water flooding \\n',\n",
       "  'abstract': 'In the process of oil recovery by water injection, large water channels will be formed. Water mainly flows through these channels, avoiding brushing the oil stuck in the minor channels, so that the oil remains unexplored. Our project is aiming to engineer Enterobacter sp. FY-07 (which is separated from oilfield produced water and able to produce cellulose in anaerobic conditions) using toggle switch, so as to produce rhamnolipid and cellulose in an controllable way. The engineered Enterobacter sp. FY-07 acts as the traffic police controlling the oil flux. In large channels, cellulose is produced to clog the flowing water and help it enter into the minor channels and wash out the oil within them. In minor channels, rhamnolipid is produced to emulsify oil, so the oil can be easily washed out. Our project aims at improving oil recovery rate in the oil harvesting process using the engineered bacteria.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2431': {'team_id': '2431',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2431',\n",
       "  'team_name': 'NortheasternU-Boston',\n",
       "  'wiki': 'http://2017.igem.org/Team:NortheasternU-Boston',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Expansion of Cell-Free Manufacturing with Post-Translational Modification\\n',\n",
       "  'abstract': 'Production of recombinant proteins has enabled or improved the treatment of numerous diseases including anemia, diabetes, and cystic fibrosis. However, recombinant protein production requires industrial infrastructure and the biologics produced in this manner typically have stringent storage requirements, including refrigeration. Many areas of the world lack infrastructure for local production or cold-chain infrastructure for effective delivery of recombinant proteins. Cell-free manufacturing based on freeze dried protein expression reaction pellets may be a solution. These pellets contain the molecular machinery to manufacture proteins and can withstand variations in temperature and humidity. Cell-free expression in this context has limited ability to produce complex proteins. Our project uses anti-microbial peptides as model molecules in order to characterize limits of cell-free expression of functional molecules. We then attempt to expand the production of functional molecules by introducing proteins for post-translational modification of expressed anti-microbial peptides to rescue functionality and demonstrate improved cell-free expression.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2396': {'team_id': '2396',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2396',\n",
       "  'team_name': 'Northwestern',\n",
       "  'wiki': 'http://2017.igem.org/Team:Northwestern',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'VesiCure: Designing pathways for integrating functional Cas9 protein into outer membrane vesicles \\n',\n",
       "  'abstract': '\\r\\nInappropriate use of antibiotics has escalated the growing problem of antibiotic resistance in many threatening diseases. In 2014, the World Health Organization classified antibiotic resistance as a global epidemic. Inactivating resistance genes via Cas9 nuclease-mediated cleavage has been shown to be an effective means of combating this epidemic; however, methods of in vivo delivery are currently limited. Our team aims to deliver Cas9 to antibiotic-resistant, pathogenic bacteria through submicron bacterial outer membrane vesicles (OMVs) as a companion re-sensitization therapeutic to antibiotic treatment. OMVs are naturally produced by all Gram-negative bacteria and are used for crosstalk, stress responses, and nutrient acquisition. Their ability to be modified and directed with relative ease makes them an ideal carrier of CRISPR-Cas9. Aiding conventional antibiotic treatment, our technology will model a complete protein delivery system and transport functional Cas9 to target cells.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2347': {'team_id': '2347',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2347',\n",
       "  'team_name': 'NPU-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:NPU-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Glycerol-based  Acrylic Acid Cell Factory\\n',\n",
       "  'abstract': '  Acrylic acid is a bulk chemical raw material,whose excellent polymerization capacity is widely used in many fields.This year, we chose to use glycerol as carbon source to achieve the all green production of acrylic acid. Based on the core enzyme Ceas2, high-yield acrylic acid cell factory was built through the Part,Pathway,System and Process.\\r\\n\\r\\n  We designed the Ceas2 mutant by utilizing the AEMD platform. HPLC and HTS techniques were applied in screening for Ceas2 mutants of high catalytic efficiency.We devised GDC pathway, achieving complete synthesis from glycerol to acrylic acid. In addition, we add a reduction power module for this pathway.We constructed a new pathway in E. coli and S. cerevisiae respectively,and after prediction via metabolic flux modeling,we optimized the cell metabolism by using the RED and CRISPR-Cas9 technique. The cell production process was hereby bolstered by optimizing the fermentation process and screening the carbon source,Buffer,temperature,pH and other experimental conditions.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2354': {'team_id': '2354',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2354',\n",
       "  'team_name': 'NTHU_Taiwan',\n",
       "  'wiki': 'http://2017.igem.org/Team:NTHU_Taiwan',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Taiwan',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'EDCs Terminator\\n',\n",
       "  'abstract': 'This year, our goal is to build a system that can detect and degrade EDCs.\\r\\nIn many developing countries, factories spew toxic water into rivers, and farmers nearby may accidentally utilize it. EDCs can interfere with endocrine systems, causing harmful effect on organisms. In our project, we focus on two common kinds of EDCs, BPA and NP.\\r\\nFor detection, we modified E.coli to express EDC receptor, ERα and GFP. Next, we assembled monobodies on gold surface. We can measure the fluorescent intensity of GFP or information of SPR via interaction between ERα and monobody, thus estimate the concentration of EDCs.\\r\\nFor degradation, we modified E.coli to produce target enzymes which can degrade BPA and NP. We integrated target enzymes with activated carbon and bioreactor system to eliminate EDCs.\\r\\nIn conclusion, our project aims to solve water safety issue for farmers in developing countries and create a more healthy agricultural environment.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2265': {'team_id': '2265',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2265',\n",
       "  'team_name': 'NTNU_Trondheim',\n",
       "  'wiki': 'http://2017.igem.org/Team:NTNU_Trondheim',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Norway',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Phage Age: Combating antibiotic resistant bacteria through bacteriophage mutation and selection\\n',\n",
       "  'abstract': 'Antibiotic resistance is poised to become one of the greatest dangers of our time. Widespread overuse of antibiotics coupled with minimal investment in new treatments have allowed pathogenic bacteria to develop resistances to many antibiotics. Our team sought to develop a platform that utilizes genetically modified E. coli DH5α to quickly evolve bacteriophages (phages). These phages should be capable of infecting the target bacteria in our coupled chemostat system. We have created a cheap, fast and simple system as well as a mathematical model for understanding its regulation. Photosensors capable of measuring real-time concentration of bacteria and phages were constructed. A plasmid to increase the mutation rate of host bacteria and thus also of invasive phages was designed. Additionally, a protocol for selecting and purifying phages from environmental samples has been developed by our team. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2316': {'team_id': '2316',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2316',\n",
       "  'team_name': 'NTU_SINGAPORE',\n",
       "  'wiki': 'http://2017.igem.org/Team:NTU_SINGAPORE',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Singapore',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Improving CRISPR-Cas-based technologies\\n',\n",
       "  'abstract': 'The CRISPR-Cas system is a wonderful tool for mankind to manipulate the genetic code of an organism. Originally functioning as bacterial adaptive immune systems, CRISPR-Cas-based technologies have been extensively studied and improved upon in recent years. \\r\\n\\r\\nOur team is interested in overcoming current limitations of CRISPR-Cas technologies, which may hinder their efficiency and deployment in living cells and organisms. There are three projects that we are working on. First, we are interested in enhancing the efficiency of HDR repair, which is utilized by scientists to precisely edit certain genes of their interest, by fusing the Cas9 enzyme with a HDR protein. Second, we are trying to deploy the enhanced design to correct a non-small-cell lung cancer mutation. Third, we are optimizing the dCas9 DNA-targeting scaffold by exploring potential deletions and introducing several mutations that will render it to be more compact and better than the traditional dCas9.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2440': {'team_id': '2440',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2440',\n",
       "  'team_name': 'NUDT_CHINA',\n",
       "  'wiki': 'http://2017.igem.org/Team:NUDT_CHINA',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'MiRNA Locker: A Modularized DNA Assembly As miRNA Inhibitors\\n',\n",
       "  'abstract': 'Nowadays, sequence-specific microRNA (miRNA) inhibitors have been extensively demanded in miRNA loss-of-function studies and gene therapies, whereas current developed inhibitors still suffer from high cost and poor scalability. Seen new approaches needed, our project attempts to demonstrate a novel design of miRNA inhibitor, named as miRNA locker, which can be easily assembled using modularized DNA parts from a set of chemically synthetic oligo DNA library. The microRNA lockers we assembled were proven to be able to bind miRNAs in an Ago2 dependent manner, and were able to trigger gene expression and phenotypic changes consistent to commercialized miRNA inhibitors while decreasing costs significantly. We also demonstrate a computer-aided designing software to facilitate the lockers design and optimization. With its unique advantage and potential on multi-targeting and convenience, we believe that our design might provide an alternative approach for miRNA inhibiting for research, diagnostic and therapeutic uses.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2447': {'team_id': '2447',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2447',\n",
       "  'team_name': 'NUS_Singapore',\n",
       "  'wiki': 'http://2017.igem.org/Team:NUS_Singapore',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Singapore',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Making engineering of customised kill switches easier!\\n',\n",
       "  'abstract': 'Many SynBio groups are engineering microbes that could one day be useful in detecting diseases, fighting cancer and monitoring heavy metals in rivers. However, engineered microbes may leak into non-designated environment, posing threats to our natural ecosystem. This is a major hurdle towards the commercialization of engineered microbes. To address this, we need effective kill switches to prevent engineered microbes from escaping into the environment. However, existing kill switches have limitations and, more importantly, it is difficult to readily tailor make kill switches for different applications.\\r\\n\\r\\nTeam NUSgem aims to make engineering of customised, effective kill switches easier. To this end, we are developing a library of characterized sensors, a killing and verification module which can be used in a computer aided design tool (e.g., Cello) and can be readily modelled. As a proof of concept, we focus on developing kill switch for engineered probiotics for human health.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2516': {'team_id': '2516',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2516',\n",
       "  'team_name': 'NU_Kazakhstan',\n",
       "  'wiki': 'http://2017.igem.org/Team:NU_Kazakhstan',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Kazakhstan',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Bioremediation of hexavalent chromium \\n',\n",
       "  'abstract': 'Chromium is a well-known toxin and carcinogen with wide industrial use. Pollution with chromium is a serious environmental concern in Kazakhstan since it is the 2nd largest chromium manufacturer in the world. Chromium primarily exists in two redox forms: trivalent and hexavalent. The former is poorly soluble and less toxic compared to the latter form. Hexavalent chromium is bioavailable and readily crosses membranes through sulfate transporters. The goal of our project is to collect Cr(VI) from wastewater, reduce it to trivalent form and store inside the microalgae C.reinhardtii.  We are introducing chromate reductase which converts Cr(VI) to Cr(III) and oligopeptide chromodulin which tightly binds 4 Cr(III) ions. To increase chromate uptake into the cell, we are exploiting natural ability of C.reinhardtii to upregulate sulfate channels when starved from sulfur. Our safety system is represented by photosensitizing protein SuperNova. It generates ROS when exposed to 585 nm wavelength of light.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2399': {'team_id': '2399',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2399',\n",
       "  'team_name': 'NWU-CHINA',\n",
       "  'wiki': 'http://2017.igem.org/Team:NWU-CHINA',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Alkane Biosensor\\n',\n",
       "  'abstract': 'Our project for this year is developing a biosensor for alkane. As present methods dealing with oil spill cannot meet the requirement of environmental recovery, our team think that the best way solving oil spill problem is preventing it.\\r\\n   We separated oil degradation bacteria from areas contaminated by oil, then we confirmed this bacteria is Pseudomonas aeruginosa strain by measuring its 16S rRNA and named it DN1. For oil degradation is an alternative metabolic pathway in bacteria, there is a manipulator for oil degradation gene, such as GntR. It codes GntR protein, which can bind with promoter on upstream of alkB2. When alkane exists, GntR will unbind with the promoter, and the RFP gene we added on downstream will express to show a signal. We used P.a DN1 and DH5α as our chassis to explore which bacteria will be a better chassis for our device. \\r\\n \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2350': {'team_id': '2350',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2350',\n",
       "  'team_name': 'NYMU-Taipei',\n",
       "  'wiki': 'http://2017.igem.org/Team:NYMU-Taipei',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Taiwan',\n",
       "  'track': 'Energy',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Smart AlgaEnergy\\n',\n",
       "  'abstract': 'Facing the threatening energy crisis, scientists are craving for alternative energy sources. Taking both clean energy productivity and other factors under consideration, we have decided to target our project on increasing the oil accumulation in microalgae by multiple approaches. On the one hand, we have determined to make microalgae undergo nitrogen starvation to increase its oil accumulation by creating a co-culturing system of microalgae and NrtA-transformed Escherichia coli that can deprive microalgae of nitrogen source. On the other hand, we have changed the color of microalgae by transforming pigmentation functions from other species into microalgae cells to enhance its efficiency of photosynthesis. By combining these two approaches, we can develop a new intelligent system which can enhance bio-energy production and contribute to the needs of renewable clean energy.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2495': {'team_id': '2495',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2495',\n",
       "  'team_name': 'NYU_Abu_Dhabi',\n",
       "  'wiki': 'http://2017.igem.org/Team:NYU_Abu_Dhabi',\n",
       "  'region': 'Asia',\n",
       "  'country': 'United Arab Emirates',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'E. coLAMP: A portable device for rapid detection of Shiga toxin-producing Escherichia coli\\n',\n",
       "  'abstract': 'Shiga toxin-producing Escherichia coli (STEC) is one of the leading causes of food-borne illnesses. Shiga toxin\\x92s mode of action involves the inhibition of protein synthesis, consequently leading to cell death. While most countries have stringent food safety regulations to prevent the sale of contaminated foods, small scale manufacturers often do not have the access, time, or resources to ensure the safety of their food. Therefore, the aim of this project was to design a cost-effective, portable device that can readily detect the presence of STEC. The device functions by lysing the bacterial cell wall and amplifying a STEC-specific gene sequence using loop-mediated isothermal amplification. It is envisioned that the use of this device in the developing world would be an effective means of reducing the incidence of food-borne illnesses.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2400': {'team_id': '2400',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2400',\n",
       "  'team_name': 'NYU_Shanghai',\n",
       "  'wiki': 'http://2017.igem.org/Team:NYU_Shanghai',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Methyltransferase-1-Mediated Consumption of Methanol in Fake Alcohol \\n',\n",
       "  'abstract': 'In China and Europe, substitutes for drinking alcohol, known as fake alcohol, have caused dangerous health effects in the community. One such substitute is methanol, known to cause blindness and/or death. The methyltransferase pathway exists in Methanosarcina Acetivorans, converting methanol into methane. By taking advantage of this pathway and two of its essential component, mtaB and corrinoid protein mtaC, methanol may be reduced in fake alcohol solution without affecting the concentration of ethanol. Since methane is a greenhouse gas, we stopped the biological pathway halfway at its intermediate methyl-mtaC. After transforming mtaB and mtaC into DH5α strain of E.coli, the effectiveness of the pathway remained undetermined due to limitations of our gas chromatography equipment and to large fluctuations in our magenta sulfite colorimetric results. Further tests, including clonogenic assay and fluorescence counterstaining, suggest that the possibility of the E.coli being unable to survive in methanol could be ruled out. \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2314': {'team_id': '2314',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2314',\n",
       "  'team_name': 'OUC-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:OUC-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'A bottle of algae Wine\\n',\n",
       "  'abstract': 'Algae Outbreak is a serious marine disaster for ocean life, which threatens economic interests and health of human. The periodically outbreak of Enteromorpha on the coastline has been a stubborn environmental problem in ShanDong, China. Here, we aim to utilize the cellobiose and xylose from waste algae and turn them into ethanol as healthy and tasty algae wine with resveratrol.\\r\\nAdditionally, we achieved a new synthetic biology platform for artificial interspecific cooperation. E.coli and S. cerevisiae are engineered to organize together as multi-cell device. The co-cultured E. coli works as surface-display system of S. cerevisiae for enhancing its biological function. Simultaneously, we built a mini transcriptional unit of standardized promoters and terminators with concise structure in Yeast, providing more potential for large-scale SynBio operations. Our project can contribute to local environmental issue and enrich synthetic biology toolbox by novel interspecific cooperation platform and transcription regulatory elements.\\n',\n",
       "  'award': 'Nominated for Best New Basic Part',\n",
       "  'medal': 'gold'},\n",
       " '2450': {'team_id': '2450',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2450',\n",
       "  'team_name': 'Oxford',\n",
       "  'wiki': 'http://2017.igem.org/Team:Oxford',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'See cruzi: Cell-free Protease Detection to Diagnose a Neglected Tropical Disease\\n',\n",
       "  'abstract': 'Our project seeks to find a synthetic biology solution to the problem of diagnosing Chagas disease, a neglected tropical disease. Caused by the parasite, Trypanosoma cruzi, it is endemic to much of Latin America. Chagas disease claims over 12,000 lives per year, yet remains difficult to diagnose in its treatable acute phase. Existing diagnostics require highly-trained personnel and expensive equipment. Our solution is to develop an accurate, low cost, and portable cell-free diagnostic kit that detects the presence of cruzipain, a protease specific to T. cruzi. We have developed, modelled, and tested two designs: (i) a cell-free DNA circuit-based diagnostic utilising the TetR repressor system and (ii) a protein-based outer membrane vesicle (OMV) diagnostic using the SpyTag/SpyCatcher system for localisation of our parts to the outer membrane and a split-protease-based protein circuit to amplify the signal. We believe this method will be applicable to numerous pathogens with specific proteases.\\n',\n",
       "  'award': 'Best Diagnostics Project;Nominated for Best Diagnostics Project;Nominated for Best Wiki;Nominated for Best Presentation;Nominated for Best Integrated Human Practices;Nominated for Best Model;Nominated for Best Applied Design',\n",
       "  'medal': 'gold'},\n",
       " '2510': {'team_id': '2510',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2510',\n",
       "  'team_name': 'Paris_Bettencourt',\n",
       "  'wiki': 'http://2017.igem.org/Team:Paris_Bettencourt',\n",
       "  'region': 'Europe',\n",
       "  'country': 'France',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Medusa: Bringing control to the 3rd dimension\\n',\n",
       "  'abstract': 'Accurate spatial-temporal response is fundamental to synthetic biology. Optogenetics has emerged as a powerful tool for genetic control and Medusa brings optogenetics to the next level. By engineering E. coli to respond to multiple light inputs, creating a logical AND gate, we aim to achieve both spatial and temporal control of gene expression. Photosensory transmembrane proteins as well as photoswitchable protein caging were investigated to further expand the existing library of optogenetic tools.  For spatial control at the subcellular level, we explored the use of a novel synthetic RNA organelles to manipulate enzymatic activity. Finally, in an effort to promote synthetic biology, we sought the input of the DIY community and chose to illustrate the power of our system by 3D-printing biomaterials.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2454': {'team_id': '2454',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2454',\n",
       "  'team_name': 'PASantiago_Chile',\n",
       "  'wiki': 'http://2017.igem.org/Team:PASantiago_Chile',\n",
       "  'region': 'Latin America',\n",
       "  'country': 'Chile',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Blueberi: Saving Lives\\n',\n",
       "  'abstract': 'Our project is directed mainly to radiologists and medical technologists due to they are constantly exposed to ionizing radiation, which is harmful to the body and accumulates over the years, causing from skin problems to different types of cancer.\\r\\n\\r\\n\"Blueberi\" is a biological option to dosimeters that currently work to measure how much radiation the specialist was exposed. The problem with these devices is that the results can be affected by different external factors as well as adulterations when they are sent to the National Health Service.\\r\\n\\r\\nOur project will give an immediate response to the exposure to high ionizing radiation levels , thanks to the use of genetically modified bacterias that will measure the mutation of DNA, it\\'s going to change color and release a lemon scent, so the specialist has a double warning that the professional is in a potential risk.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2198': {'team_id': '2198',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2198',\n",
       "  'team_name': 'Pasteur_Paris',\n",
       "  'wiki': 'http://2017.igem.org/Team:Pasteur_Paris',\n",
       "  'region': 'Europe',\n",
       "  'country': 'France',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'æther : an innovative air-purifying biomaterial\\n',\n",
       "  'abstract': 'Indoor air pollution is a worldwide threat and existing solutions are limited as stated by experts we met. Our aim is to design an air purifying device. Not only will it capture toxic compounds present in the air, but it will also degrade them using enzymes. As a proof of concept, we have chosen Polycyclic Aromatic Hydrocarbons (PAHs) as targets, since they are some of the most dangerous volatile air pollutants. The enzymatic pathway we have elaborated will degrade PAHs into pyruvate, a physiological compound. Our system and its coating will not release any GMOs. After meetings and discussions with the general public, health specialists, air quality regulators, law and political actors, we have designed an affordable and user-friendly device, æther. Indeed, it consists of an energetically autonomous DIY kit based on simple materials, so that people with low income and restricted access to electricity can also benefit from it.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2243': {'team_id': '2243',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2243',\n",
       "  'team_name': 'Peking',\n",
       "  'wiki': 'http://2017.igem.org/Team:Peking',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Information Processing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'pekingigem@gmail.com\\n',\n",
       "  'title': 'Genetic Sequential Logic Programming\\n',\n",
       "  'abstract': 'Complex gene regulation requires responses that depend not only on the current levels of input but also on signals received in the past, enabling temporal variation in cell behavior. In digital circuit theory, this information-processing paradigm refers to sequential logic.\\r\\n\\r\\nWe developed recombinase parts that can stably edit DNA sequences and demonstrated their capability of implementing sequential logic in cell. The core of sequential logic is the memory module to store past events. We built a bio-flip-flop with similar function as its electronic counterpart, which can store one bit of memory. By incorporating repressilator into the system, we aim to trigger cell state transition with an intracellular oscillation clock signal. We also developed an automated method to generate genetic sequential logic circuits according to customized specification of inputs and responses in different temporal states, namely the 3D truth table.\\n',\n",
       "  'award': 'Nominated for Best Information Processing Project',\n",
       "  'medal': 'gold'},\n",
       " '2456': {'team_id': '2456',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2456',\n",
       "  'team_name': 'Penn',\n",
       "  'wiki': 'http://2017.igem.org/Team:Penn',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Hardware',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'KAM-Spec: An open-source, cost-effective dispersion-based microplate reader\\n',\n",
       "  'abstract': 'Microplate readers are important tools used for multiplexed assays in synthetic biology. However, the current market price for such devices creates a high barrier of entry for many institutions. To make this key technology more accessible, the Penn iGEM team (2017) strives to create an open-source and cost-effective solution. The KAM-Spec is a ~$2K dispersion-based plate reader (up to 96 wells), in which full absorbance and emission spectra are rapidly resolved on a CCD line-array camera. KAM-spec automation and analysis is performed in Python. KAM-spec performance will be benchmarked using commercial fluorophores and common fluorescent bacterial strains used in synthetic biology and optogenetics.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2518': {'team_id': '2518',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2518',\n",
       "  'team_name': 'Peshawar',\n",
       "  'wiki': 'http://2017.igem.org/Team:Peshawar',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Pakistan',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Bio-Reporter Fish: Detection of heavy metal contamination in freshwater through novel BioBricks-based devices \\n',\n",
       "  'abstract': 'Estimates from national and international studies indicate an alarming concentration of heavy-metals in the water resources of Pakistan, with 60 million Pakistanis affected by arsenic alone. This highlights the negative impact of heavy-metal contamination on environment, food, and health. To solve this problem, we are developing novel bio-brick devices which will enable a fish to detect Arsenic, Cadmium, Mercury, Nickel, Copper and Zinc. Using metal inducible promoters we aim to construct six devices which express specific chromo-proteins for heavy-metal detection. All devices with prokaryotic parts will initially be tested in E.coli to establish their proper functioning. Next, we will express heavy-metal reporter devices in fish which will in principle be able to provide colorimetric detection of heavy-metals in fresh water ponds. This novel bio-reporter system will ultimately help in taking timely actions to ensure the safety of water and fish, impacting the environment, food and livelihood of millions of Pakistanis.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2476': {'team_id': '2476',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2476',\n",
       "  'team_name': 'Pittsburgh',\n",
       "  'wiki': 'http://2017.igem.org/Team:Pittsburgh',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Droving with Dronpa: Rapid, Reversible Control of Escherichia Coli Motility Using Light\\n',\n",
       "  'abstract': 'Real-time controllable nanorobots are an attractive goal of engineering and robotics. However, current nanorobots are limited by the precision of control: most utilize magnetic or electric field based steering. Yet, light-based control through laser technology, offers precise spatial resolution at the diffraction limit. \\r\\n\\r\\nWe seek to modify the chemotactic machinery of E. coli by modulating the activity of the critical chemotaxis protein CheY. The GFP-related protein Dronpa has been demonstrated to reversibly block kinase activity through light-induced dimerization [Science 2017, 355, 836-842]. Our team has designed a novel Dronpa-CheY fusion protein to precisely control the movement of E. coli with different wavelengths of light. Since Dronpa controls CheY activity directly, the effects after exposure to light will be rapid, compared to cases controlled by gene regulation. The control of E. coli that can be precisely \\x91driven\\x92 on the micrometer-scale may revolutionize targeted medical treatment, bioanalytical sensing, and nanomanufacturing. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2483': {'team_id': '2483',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2483',\n",
       "  'team_name': 'Potsdam',\n",
       "  'wiki': 'http://2017.igem.org/Team:Potsdam',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Two novel approaches to metabolic channeling to increase the efficiency of the Indole-3-acetic acid pathway\\n',\n",
       "  'abstract': 'With metabolic channeling, the speed of the reaction of a metabolic pathway is raised by putting the enzymes close together, thus the diffusion distances for the intermediates are reduced. Therefor we have two approaches. \\r\\n\\r\\nIn the first approach, we are utilizing the DNA-binding property of the dCas9-protein to put the enzymes next to each other on a DNA-scaffold. The proteins bind via Aptamers and specific binding proteins to dCAS9. This approach is tested in E. coli while the second one will be implemented in yeast. \\r\\n\\r\\nThe second approach works with liquid-liquid-phase separation. In this process, membraneless organelles are formed, induced by specific variable domains of Ddx4. We want to investigate, whether fusing the enzymes for the pathway with Ddx4 enables droplet formation and thereby induces metabolic channelling.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2432': {'team_id': '2432',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2432',\n",
       "  'team_name': 'Princeton',\n",
       "  'wiki': 'http://2017.igem.org/Team:Princeton',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Engineering the Microbiome of Drosophila melanogaster \\n',\n",
       "  'abstract': 'Tremendous potential lies in engineering the microbiome, like recognizing and metabolizing toxic compounds the host cannot. Using the Drosophila gut as our model host and E.coli as our microbiome component, we developed techniques to engineer the gut microbiome. The E. coli was engineered to fluoresce GFP or RFP when induced by arabinose, its localization in the gut confirmed by fluorescent microscopy of dissected gut. To increase uptake of e. coli, it was mixed with baker\\x92s yeast, thus coexistence of 3 species was required to deliver the bacteria to fly gut, When continuously fed, the bacteria remained in the gut, but removing bacteria from food led to a gut residence time of E. coli of 24 hours. The work showed the resistance of the host to establish a foreign persistent microbiome; future work would involve engineering a more native bacterial species such as Lactobacillus to increase gut residence time.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2359': {'team_id': '2359',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2359',\n",
       "  'team_name': 'PSI_engineers',\n",
       "  'wiki': 'http://2017.igem.org/Team:PSI_engineers',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Korea, Republic Of',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': ''},\n",
       " '2228': {'team_id': '2228',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2228',\n",
       "  'team_name': 'Purdue',\n",
       "  'wiki': 'http://2017.igem.org/Team:Purdue',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Engineering the human lung microbiome to degrade inhaled carcinogens\\n',\n",
       "  'abstract': 'Benzene, an inhaled carcinogen linked to leukemia and lymphoma, is found in consumer goods and industrial byproducts. To reduce exposure-related illnesses, we engineered bacteria to degrade benzene into safe metabolites in a nine-enzyme pathway. This prophylactic Benzene REduction THERapy (BREaTHER) may be introduced to the lung microbiome. As proof-of-concept we evaluated the ability of engineered E. coli to degrade benzene via gas chromatography and measured improvements in the benzene tolerance of engineered strains. We also demonstrated that nebulizers can efficiently deliver BREaTHER into the lungs. To define the conditions under which BREaTHER is safe and effective, we developed a mathematical model. We envision BREaTHER using a variety of lung microbes and have designed \\x91universal\\x92 promoters and ribosome binding sites via analysis of various Gram positive and negative bacteria. BREaTHER may one day provide additional protection for concerned individuals and reduce the occupational exposure of high risk workers such as firefighters.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2484': {'team_id': '2484',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2484',\n",
       "  'team_name': 'Queens_Canada',\n",
       "  'wiki': 'http://2017.igem.org/Team:Queens_Canada',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Glacial Gladiators: Bifunctional Biofilms for Arctic Bioremediation\\n',\n",
       "  'abstract': 'Biofilms are often maligned because of their roles in antibiotic-resistant infections and dental plaque. However, biofilms also offer an attractive platform for the design of self-assembling biomaterials programmed for specific functionality. The amyloid protein CsgA accounts for the majority of the proteinaceous component, curli nanofibres, of Escherichia coli biofilms. CsgA has been shown to be tolerant of C-terminal fusions, allowing CsgA endowed with diverse peptide domains to be secreted and self-assembled extracellularly similar to normal curli nanofibres. We present the genetic engineering of CsgA to create a biofilm that binds ice and degrades hydrocarbons. A type I antifreeze protein, AFP8, will be fused to CsgA for ice binding, and a PA-14 adhesin domain will be fused, via the SpyTag-SpyCatcher system, to bind the hydrocarbon-degrading bacterium Marinobacter hydrocarbonoclasticus. Thus, the end-product will be a bifunctional biofilm capable of establishing itself on Arctic ice to degrade toxic hydrocarbons present in oil spills.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2210': {'team_id': '2210',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2210',\n",
       "  'team_name': 'RDFZ-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:RDFZ-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Mobile Surfactant Factory \\x96 Combating Oil Spill With Engineered Bacillus subtilis\\n',\n",
       "  'abstract': 'Soil contamination due to crude oil causes environmental and health-related problems. Our project engineers Bacillus subtilis that function as surfactin producing units to remediate contaminated soils. Surfactin is a biosurfactant that can emulsify hydrophobic organic compounds and, in turn, enhance the biodegradation process. To synthesize and export surfactin more efficiently, we overexpress sfp, the 4\\x92-phosphopantetheinyl-transferase, and YerP, a surfactin efflux pump. In addition, lmrA, a multidrug resistance transporter from Lactococcus lactis, is mutated and tested for higher surfactin specificity. We want our product to provide a greener and safer alternative to methods such as heat treatment and leaching. Biosurfactants and the introduction of Bacillus subtilis should have fewer impacts on soil microbiome, and should be more effective than relying on bioremediation alone. We hope that our project can contribute to the use of B. subtilis as a chassis in synthetic biology, and explore new methods of utilizing multi-drug resistance factors.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2338': {'team_id': '2338',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2338',\n",
       "  'team_name': 'REC-CHENNAI',\n",
       "  'wiki': 'http://2017.igem.org/Team:REC-CHENNAI',\n",
       "  'region': 'Asia',\n",
       "  'country': 'India',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Latarcoli: A knight on guard against food pathogens\\n',\n",
       "  'abstract': 'The rise of the super bugs poses an imminent threat that looms large for the very existence of humanity, catching us off guard and weaving the web of extensive antibiotic resistance. Antimicrobial peptides have found a meteoric acclamation amongst research scientists owing to their lineament of resisting the resilience of bacteria. Latarcins are antimicrobial peptides found in spider venom. Latarcin-2a(M-Zodatoxin) is a 26 amino acid, highly cationic peptide that lyses microbes via a discrete, membrane-specific carpet mechanism. A proposed mutation to its native form eliminates its existing haemolytic and cytolytic activity thus rendering it innocuous to humans. In an attempt to overcome the lethal effects of Latarcin on our chassis itself, a quorum sensing mechanism will be incorporated into our host. Having identified a niche in the food preservative sector, we plan to enclose this engineered strain in a food wrapper to extend the shelf life of food.\\r\\n\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2390': {'team_id': '2390',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2390',\n",
       "  'team_name': 'RHIT',\n",
       "  'wiki': 'http://2017.igem.org/Team:RHIT',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Measurement',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'B-Fine: Characterization of Vitamin B9 and B12 Binding Riboswitches\\n',\n",
       "  'abstract': 'Enabling the ease of access to proper nutrition is vital for developing a healthy human population. Understanding the elements of nutrition will aid in engineering solutions to current -and future- nutrition concerns. One set of vitamins where deficiencies are presently a concern, especially among pregnant women, are vitamins B9 and B12. The primary goal of this project was to characterize and document riboswitches responsive to vitamins B9 and B12. Five riboswitches regulated by either vitamin B9 or B12 were selected and synthesized. E. coli were engineered to express the mRNA regulating GFP expression constitutively using the well-documented promoter BBa_J23106. The GFP expression when exposed to varied levels of the ligand were measured over time through fluorescence measurements, and the binding affinity of the ligand to each riboswitch was determined through isothermal titration calorimetry. These data and subsequent documentation will aid in providing a usable, documented part for further use.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2194': {'team_id': '2194',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2194',\n",
       "  'team_name': 'Rice',\n",
       "  'wiki': 'http://2017.igem.org/Team:Rice',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'HexaTri: Bioremediation of Chromium(VI) Contaminated Wastewater via Engineered Chromium(VI)-Reducing Shewanella oneidensis\\n',\n",
       "  'abstract': 'A recent analysis of America\\x92s drinking water for carcinogenic hexavalent chromium (CrVI) revealed over 218 million Americans consume CrVI at levels exceeding a de minimis lifetime cancer risk. Our team attempted to solve this problem by engineering Shewanella oneidensis to reduce CrVI to its less toxic form (CrIII) in contaminated wastewater. We enhanced reduction via expression of a mutated chromate reductase enzyme (chrR6) and increased chromate permeability via over-expression of sulfate transporter and binding proteins (cysP,U,W,A and sbp). Addressing biocontainment, we included a \\x93kill switch\\x94 where low CrVI levels signaled expression of a \\x93toxin\\x94 (BamHI). Laboratory work has been completed in model organism Escherichia coli; future work includes characterizing the circuit in S. oneidensis. To assess real-world feasibility, we developed bioreactor- and cell-scale simulations of our engineered bacteria in an activated sludge secondary treatment system aimed at determining the kinetics of Cr(VI) remediation and cell death.\\n',\n",
       "  'award': 'Nominated for Best Environment Project',\n",
       "  'medal': 'gold'},\n",
       " '2503': {'team_id': '2503',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2503',\n",
       "  'team_name': 'RPI_Troy_NY',\n",
       "  'wiki': 'http://2017.igem.org/Team:RPI_Troy_NY',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Selective modification of yeast MFE2 gene improves the efficiency of medium-chain length sophorolipid production.\\n',\n",
       "  'abstract': 'Sophorolipids are a group of amphipathic compounds synthesized by yeasts, including our selected chassis Starmerella bombicola. These compounds are among the most industrially viable biosurfactants. As surfactants, sophorolipids are superior to current synthetic alternatives largely due to their low toxicity and ready biodegradability. Anti-cell proliferation and anti-microbial assays suggest that medium-chain length sophorolipids possess therapeutic potential. However, their higher cost of production poses a major barrier to commercial use. Even when grown on a substrate largely composed of fatty acids, a significant portion of the fatty acids are diverted from sophorolipid production due to competition with the β-oxidation pathway. Fatty acids diverted into the β-oxidation pathway undergo irregular cleavage, lowering sophorolipid yield and creating a heterogeneous mixture of products. Full suppression of this pathway would result in undesirable metabolic disruption. Selective suppression of this pathway through genetic engineering could reduce production costs through improving the yield of useful chain lengths.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2334': {'team_id': '2334',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2334',\n",
       "  'team_name': 'SCU-WestChina',\n",
       "  'wiki': 'http://2017.igem.org/Team:SCU-WestChina',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Blocking the Urate Storm in the Blood\\n',\n",
       "  'abstract': 'Hyperuricemia and refractory gout are caused by the high urate concentration in the blood. Considering there are still debates on whether the drug should be used in the asymptomatic hyperuricemia patients and the drug resistance of the refractory gout patients, new approaches are eagerly needed. We constructed the urate metabolic pathway in the probiotic E. coli Nissle 1917 and applied it in the gut to reduce the urate concentration in the blood indirectly. In addition, we built a dialysis-like device combined with modified bacteria to utilize the urate directly in the blood. Our solutions provide a suitable, long-term and non-drug treatment for asymptomatic hyperuricemia patients and an ultimate treatment for the refractory gout patients.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2360': {'team_id': '2360',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2360',\n",
       "  'team_name': 'SCUT-China_A',\n",
       "  'wiki': 'http://2017.igem.org/Team:SCUT-China_A',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'A reporter device based on SRRz lysis gene applied to heavy matel ions detection\\n',\n",
       "  'abstract': '  Our team tries to build a group of heavy metal bio-sensors: We standardize the lysis gene SRRz and the β-galactosidase gene, to build a visible and low-cost reporter which can detect chemical substances in aqueous solution qualitatively, and in a certain concentration range, it can detect the chemical substances quantitatively. Subsequently, we will link some specific inducible promoters of heavy metal ions to this reporter and transform these devices into Escherichia coli BL21 to verify the feasibility of the reporter and construct engineering bacteria that can rapidly detect heavy metal ions in aqueous solution.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2366': {'team_id': '2366',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2366',\n",
       "  'team_name': 'SCUT-China_B',\n",
       "  'wiki': 'http://2017.igem.org/Team:SCUT-China_B',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Foundational Advance(P)',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': ' iGEL-an Integrate HydroGel by Protein Brick\\n',\n",
       "  'abstract': 'Nowadays there are lots of problems in the world,heavy metals,industry pollution and hard study, and biological tech.have plenty of potential to solve these. We know proteins play a vital roles in biology.Therefore we want to start with synthesizing protein bio-parts,and making it into a biological reaction platform which named \"Bio-Go\".It would be a special hydrogel carrying two kinds of molecules or cells. With its special physical properties,gels have been widely used in many fields like industry,medication etc.and this year we want to build a bio-affinity and self-assembly hydrogel. Thanks to the concept of synthetic biology we could connect spy-system and coiled-coil system scarlessly and turn it into standardized protein part. With it we built a new hydrogel network, and explored its initial application in and in industry research and medication. It could be a self-fixed co-cartelized system for industry,cell 3D culture for research and even 3D printing tissue for medication.\\n'},\n",
       " '2325': {'team_id': '2325',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2325',\n",
       "  'team_name': 'SCUT-FSE-CHINA',\n",
       "  'wiki': 'http://2017.igem.org/Team:SCUT-FSE-CHINA',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Robust E.coli for Open Culture in Industrial Fermentation Processes\\n',\n",
       "  'abstract': 'Microbial fermentation is an important component of industrial biotechnology and is increasingly popular, with products ranging from bulk chemicals to bioactive molecules. However, despite advance in biotechnology and improvement in fermentation control, microbial contamination and phages infection during fermentation process still remain major concerns worldwide. This is due to the widespread distribution of microorganism as well as phages and the consequent negative economic impact caused by frequent sterilization. In an effort to avoid fermentation failure and even to make these processes more economical, we try to introduce a \\x93robust strain\\x94 for developing the fermentation process in the open (unsterile) culture. Here, we introduce the metabolic pathways of formamide and phosphite into the host to fit our special designed medium so that the unexpected microorganism could not exist. Additionally, applying CRISPR/Cas9 system makes the host attain phage resistance ability. Eventually, an open fermentative process can be achieved. \\r\\n\\n',\n",
       "  'award': 'Nominated for Best New Application Project',\n",
       "  'medal': 'gold'},\n",
       " '2276': {'team_id': '2276',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2276',\n",
       "  'team_name': 'SCU_China',\n",
       "  'wiki': 'http://2017.igem.org/Team:SCU_China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Rhythmic Production of Melatonin in E.coli\\n',\n",
       "  'abstract': \"Nowadays, insomnia and circadian rhythm disorders are increasingly plaguing individuals' daily life. According to NIH reports, around 50-70 million U.S. adults are suffering from sleep insufficiency and poor-quality sleep. The circadian rhythm in human is predominantly under the regulation of melatonin, a hormone produced by pineal gland periodically. This year we want to use engineered E.coli to mimic this process in human. We propose to construct a melatonin biosynthesis pathway in E.coli to produce melatonin, while simultaneously couple this process with periodically-optimized repressilator, a synthetic genetic oscillator with higher precision, with the intention to render melatonin production resemble mammalian periodicity in E.coli. \\n\",\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2239': {'team_id': '2239',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2239',\n",
       "  'team_name': 'SDSZ-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:SDSZ-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Automatic Synthesis of UDCA (Effective Ingredient of Bear Bile) Using Immobilized Enzymes\\n',\n",
       "  'abstract': 'Bear bile, a widely employed Chinese traditional medicine, has significant pharmaceutical effects on treating\\xa0primary biliary cholangitis, hepatitis C, allotransplantation rejection, primary sclerosing cholangitis, and acute calculous cholecystitis. In order to find an alternative for extracting bile from living Asian black bear, our team works on the biological synthesis of UDCA( Ursodeoxycholic Acid, the effective ingredient of bear bile). By employing CBD(cellulous binding domain) to immobilize enzymes (7α-HSDH, 7β-HSDH, GDH, and LDH) on cellulous, we are able to convert CDCA(Chenodeoxycholic Acid), found in goose bile, into UDCA. Furthermore, we have designed a machine equipped with reaction efficiency measuring system and enzyme-adding controlling system. Our automatic biological synthesis will not only be more efficient but also be cheaper than the tradition extraction and the chemical approach. This approach relieves the pain of patients and the bile bears.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2449': {'team_id': '2449',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2449',\n",
       "  'team_name': 'SDU-Denmark',\n",
       "  'wiki': 'http://2017.igem.org/Team:SDU-Denmark',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Denmark',\n",
       "  'track': 'Energy',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'PowerLeaf \\x96 a bacterial solar battery\\n',\n",
       "  'abstract': 'The PowerLeaf introduces a novel solution for long-term storage of solar energy, thus becoming an alternative to solar cells. This is accomplished without the use of environmentally harmful resources. The device is designed to resemble a plant leaf, which is meant to provide a nature-in-city ambience. This hypothetical implementation of the PowerLeaf in an urban environment, was developed through public engagement and collaboration.The bacterial solar battery is composed of an energy storing unit (1), and an energy converting unit (2). The energy storing unit (1) is defined by a genetically engineered Escherichia Coli, which fixates carbon dioxide into the chemically stable polymer cellulose. A light sensing system activates dormancy during nighttime, to reduce energy lost by metabolism. The energy converting unit (2) uses genetically engineered Geobacter Sulfurreducens to consume the stored cellulose. Retrieved electrons are transferred by optimized nanowires to an anode resulting in an electrical current. \\n',\n",
       "  'award': 'Nominated for Best Energy Project',\n",
       "  'medal': 'gold'},\n",
       " '2321': {'team_id': '2321',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2321',\n",
       "  'team_name': 'SDU_CHINA',\n",
       "  'wiki': 'http://2017.igem.org/Team:SDU_CHINA',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Cancer Slayer ─ An invincible opponent of PD-L1\\n',\n",
       "  'abstract': 'Non-small cell lung cancer (NSCLC) is the leading cause of cancer-related death among the world with low overall survival rate. In NSCLC, immunotherapy has been indicated as a potential therapy for treating in situ solid tumor. Previous research has indicated that tumor cells can express programmed death-1 ligand (PD-L1) to diminish T-cell effector functions and therefore to achieve immune escape. In our project, gene edition will be incorporated with immunotherapy to eradicate the immune escape occurred in NSCLC. By constructing plasmid with Crispr-Cas 9 system targeting the gene coding for PD-L1, the expression of PD-L1 will be inhibited to restore immune system supervision. To ensure the biosafety, another plasmid with Crsipr-Cas 9 system to cut off the housekeeping gene of two plasmids will be also constructed to suicide both plasmids when PD-L1 expression was not detectable. Better therapeutic effects of immunotherapy to conquer the NSCLC are hoped to be achieved.\\n',\n",
       "  'award': 'Nominated for Best Therapeutics Project',\n",
       "  'medal': 'bronze'},\n",
       " '2291': {'team_id': '2291',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2291',\n",
       "  'team_name': 'SECA_NZ',\n",
       "  'wiki': 'http://2017.igem.org/Team:SECA_NZ',\n",
       "  'region': 'Asia',\n",
       "  'country': 'New Zealand',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Frozen in Thyme\\n',\n",
       "  'abstract': 'With an ever-growing world population, having sustainable and reliable crops for food production\\r\\nis becoming increasingly important. However, every year millions of dollars\\x92 worth of produce is damaged, lost, or never produced because of frosts. Frost damages new shoots and buds of crop plants through the formation of ice crystals within the tissues, which rupture the surrounding cells. As a result, new plant and fruit growth is severely inhibited. Despite promising research into frost resistance mechanisms, the majority of producers still utilise costly, and often ineffective, traditional methods of frost avoidance. Our team seeks to introduce a variety of frost resistance genes into the model organisms Arabidopsis thaliana and Escherichia coli for characterisation. This will provide insight into the varying ability of frost resistance genes to protect model organisms at sub-zero temperatures, ultimately leading to the production of frost tolerant crops.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Entrepreneurship',\n",
       "  'medal': ''},\n",
       " '2453': {'team_id': '2453',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2453',\n",
       "  'team_name': 'sfu_bc_canada',\n",
       "  'wiki': 'http://2017.igem.org/Team:sfu_bc_canada',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': None,\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': ''},\n",
       " '2315': {'team_id': '2315',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2315',\n",
       "  'team_name': 'Shanghaitech',\n",
       "  'wiki': 'http://2017.igem.org/Team:Shanghaitech',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'A multilayer signal-processing system based on Quorum Sensing  (Multilayer-QS)\\n',\n",
       "  'abstract': 'In synthetic biology, building complex logic circuits are often difficult, especially in a single cell population. It requires many non-crosstalking information processing units such as quorum sensing (QS) factors, to work within a cell without significant interference. However, these non-crosstalking QS factors are very limited. Furthermore, even for simple circuits, one has to create de novo which can be difficult for beginners. It would be ideal to use modularized parts assemble directly into circuits. Therefore, we aimed to create a multilayer signal-processing system by using compartmentalized QS factors. We have provided proof-of-concept data and modeling to show that our system would allow faithful information flow between bacterial population. Also, we aimed to build QS part libraries to allow easily switching of parts, changing signal inputs and outputs without de novo cloning. Thus, our system may not only increase signal-processing power, but also make it more friendly to synthetic biologists.\\n',\n",
       "  'award': 'Nominated for Best Foundational Advance Project;Nominated for Best Education & Public Engagement;Best Hardware;Nominated for Best Hardware;Nominated for Best New Composite Part',\n",
       "  'medal': 'gold'},\n",
       " '2508': {'team_id': '2508',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2508',\n",
       "  'team_name': 'Sheffield',\n",
       "  'wiki': 'http://2017.igem.org/Team:Sheffield',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Hardware',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'BrightBiotics \\x96 Monitoring bacterial growth to advise on antibiotic choices\\n',\n",
       "  'abstract': 'With increasing use, and especially misuse of antibiotics, microorganisms keep becoming more and more resistant to them. This phenomenon is called antimicrobial resistance, or AMR.  As current diagnostic devices are not fast enough, this has resulted in the overprescription of antibiotics, leading to an increase in AMR. The aim of our project, BrightBiotics, was to create a cheap, rapid and user-friendly way to faster advise healthcare professionals, on what antibiotics to use. Our device, the BrightBiotics System, does this by measuring bacterial growth, as turbidity, in the presence of different antibiotics. Once the patient\\x92s sample has been prepared and put in the device, photodiodes detect changes in turbidity, and the data is sent via WiFi to the Cloud. The healthcare professionals can then check the results allowing them to make a more informed decision about which antibiotic to give the patient.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2200': {'team_id': '2200',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2200',\n",
       "  'team_name': 'Shenzhen_SFLS',\n",
       "  'wiki': 'http://2017.igem.org/Team:Shenzhen_SFLS',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Targeting the Mutant BRAF in Melanoma Cells by CRISPR/Cas9 Technology\\n',\n",
       "  'abstract': 'Melanoma is the most malignant type of skin cancer with high metastasis potential and a low survival rate. It is reported that about 60% of melanomas contain a mutation in the v-raf murine sarcoma viral oncogene homolog B (BRAF), and V600E (1799T>A) variation in BRAF is the main type of mutations in the cancer tissues, which plays a critical role in carcinogenesis of melanoma. In our project, we aim to disrupt the mutant BRAF in the two melanoma cell lines (A375 and G361) by CRISPR/Cas9 technology. The data showed that the system significantly inhibited the proliferation and migration, and induced apoptosis in the two cell lines, which suggest that we could target a specific oncogene and achieve personalized therapy for different types of cancer only by simply changing the sequence of a sgRNA.\\n',\n",
       "  'award': 'Nominated for Best Wiki, High_school;Best Presentation, High_school;Nominated for Best Presentation, High_school;Nominated for Best Poster, High_school',\n",
       "  'medal': 'bronze'},\n",
       " '2507': {'team_id': '2507',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2507',\n",
       "  'team_name': 'SHSBNU_China',\n",
       "  'wiki': 'http://2017.igem.org/Team:SHSBNU_China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Noninvasive gut inflammation detector\\n',\n",
       "  'abstract': \"Thiosulfate and tetrathionate are two kinds of chemical compound which would be produced during gut inflammation. Until now, scientists are able to detect gut inflammation using utilized two-system detector, which includes detector and reporter part, gained from marine Shewanella species and present the result by expressing sfGFP gene. Considering this method can be further improved, SHSBNU_China team worked on changing the reporter part to let the results be presented more clearly and visibly without specially-produced ultraviolet light, instead, the E.coli would change into a different color even in anaerobic environment. Furthermore, to make the result being collected more easily, we developed a kind of pill, where the E.coli is stored, with special walls that would only allow small molecules to get through. This system also has a functional potential that it could be further modified to do some treatment to the inflammation once it's detected. \\n\",\n",
       "  'award': 'Nominated for Best Presentation, High_school;Nominated for Best Model, High_school;Nominated for Best Part Collection, High_school',\n",
       "  'medal': 'gold'},\n",
       " '2472': {'team_id': '2472',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2472',\n",
       "  'team_name': 'SIAT-SCIE',\n",
       "  'wiki': 'http://2017.igem.org/Team:SIAT-SCIE',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Tardi-Guards\\n',\n",
       "  'abstract': 'Genetically engineered organisms designed by iGEM teams have the potential to serve in wide range of fields. And when it comes to application, resilience of these organisms is an important factor that needs our attention. Hence this year, our project seeks to improve the stress resistance ability of engineered organisms. Our solution was built upon Tardigrades, an organism famous for their extraordinary ability to survive in harsh conditions. The resilience comes from some unique protective proteins. We express these proteins in vivo, test its ability to increase the survivability of bacteria under water deficient and radiation conditions, eventually it will be used on eukaryotic cells like yeast, multicellular organisms like nematode or even mammal cells and organs. Our products can be easily used by many teams whose engineered organism have to work in extreme environments, with numerous potential use such as in culture collection and protection of transplant organs. \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2311': {'team_id': '2311',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2311',\n",
       "  'team_name': 'SiCAU-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:SiCAU-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'A Sensitive Positive Feedback Detector\\n',\n",
       "  'abstract': 'Positive feedback system can be found in everywhere, existed widely in creature contained from Lac operon of prokaryotes to human beings, and it has been used to construct various attractive circuits such bistable state, oscillators and other inconceivable systems.\\r\\nIn our project this year, the significant performance of sensitivity, which researchers has not concerned to, will be developed in positive feedback loop established by LuxR/LuxI quorum sensing system. But this loop will be open because of its background expression. The AiiA, which can hydrolyze the AHL, will be control the background expression, together with the Lac operon being the system main switch.\\r\\nFurthermore, the input defined by user will have a negative correlation to the opening time. Hence, there need a model to describe this relation and even realize quantitative determination.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2285': {'team_id': '2285',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2285',\n",
       "  'team_name': 'SJTU-BioX-Shanghai',\n",
       "  'wiki': 'http://2017.igem.org/Team:SJTU-BioX-Shanghai',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Palette\\n',\n",
       "  'abstract': 'In our project, Palette, we develop a system which can detect multiple target substances and can read the result by naked eyes. We choose chromoproteins to provide rich colors for indication of multiple signals. Furthermore, by mixing two types of chromoproteins, we can create a third color taken as the characterization of the relative abundance of two signal molecules. We employed small transcription activating RNAs (STAR) to decrease the leaky expression of chromoproteins which we found in our experiment.\\r\\nWe also introduced test paper and smartphone into our project to make our system more user friendly. With these designs, even ordinary people can easily tell the concentration of target substances. \\r\\nTo test the system, we choose heavy metal ions as an example. Since heavy metal pollution is an urgent problem, we hope our application can be used widely. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2211': {'team_id': '2211',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2211',\n",
       "  'team_name': 'SJTU-Software',\n",
       "  'wiki': 'http://2017.igem.org/Team:SJTU-Software',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Software',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'BAT: BiobrickAssist Technology, a search & evaluation engine for biobricks\\n',\n",
       "  'abstract': 'As many accessable biobricks are not well annotated or documented even on some official sites, researchers especially novices are hard to find biobricks of high quality. Thus we have developed a web application named BAT(BiobrickAssist Technology) with Node.js. BAT mainly functions as a search and evaluation engine for biobricks. We are aimed to create a platform for researchers to learn and evaluate. Based on our work in 2016, BAT can do evaluation at different levels and evaluate completely new biobricks, and this year we do some optimization for the scoring algorithm. Users may search biobricks by categories, submission time etc. Considering the auto-evaluation may not be perfect all the time, we have been developing a small but useful community like Q/A for users to manually select best biobricks. If a researcher has used a biobrick, he(she) can score it on our site and leave a short comment.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2393': {'team_id': '2393',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2393',\n",
       "  'team_name': 'SKLPB_China',\n",
       "  'wiki': 'http://2017.igem.org/Team:SKLPB_China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': None,\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': ''},\n",
       " '2224': {'team_id': '2224',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2224',\n",
       "  'team_name': 'SMS_Shenzhen',\n",
       "  'wiki': 'http://2017.igem.org/Team:SMS_Shenzhen',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Pesticide Mate\\n',\n",
       "  'abstract': \"This year, SMS_Shenzhen develops a pesticide mate which can reduce copious quantities of pesticides using and is free from side effects to the environment. Scale insects are pests which threaten agriculture and gardening industry. Because of its harmfulness and tenacity, a large amount of contact pesticide is used yearly to repress its outbreak. In this project, E. coli was applied to selectivity degrade the thick shell of scale insects. Three kinds of enzymes are expressed in E. coli to degrade the chitin, protein and wax shell, which make it possible for pesticide to penetrate the shell. To avoid hurting plants\\x92wax-protein surface, a sucrose-induced enzyme release system is introduced, which utilizes the insect's honeydew secretion. Though pests may not be killed directly by E. coli, there is a huge decrease in pesticides needed in the killing process, which saves much money and brings numerous benefits to environment.\\n\",\n",
       "  'award': 'Best Presentation, High_school;Nominated for Best Presentation, High_school;Nominated for Best Poster, High_school;Nominated for Best Integrated Human Practices, High_school;Nominated for Best Education & Public Engagement, High_school',\n",
       "  'medal': 'gold'},\n",
       " '2244': {'team_id': '2244',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2244',\n",
       "  'team_name': 'SSTi-SZGD',\n",
       "  'wiki': 'http://2017.igem.org/Team:SSTi-SZGD',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': ' Guardian of the wheatland \\n',\n",
       "  'abstract': '  The aim of this project is to combine optogenetics and biotechnology, by using genetically modified organism as the core carriers, to develop a novel method for degrading pesticide residues in contaminated soil. We employed a novel light inducible/repressive system that can efficiently overexpress heterogenous hydrolases that are able to degrade chemical pesticides organophosphorus and parathion-methyl residues in soil . By using light control gene expression system, protein products can be produced without the need of using toxic and expensive chemical inducers (i.e. IPTG). In addition, we plan to develop a device that integrates monitoring and data-sharing functions that help applying this system to on-site pesticide residue detection and degradation. We hope to achieve a microbial degradation method with low toxicity, low cost and high efficiency that serve a positive role in maintaining ecological balance and sustaining development in China .\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2485': {'team_id': '2485',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2485',\n",
       "  'team_name': 'Stanford-Brown',\n",
       "  'wiki': 'http://2017.igem.org/Team:Stanford-Brown',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Mars: Getting there and staying there\\n',\n",
       "  'abstract': 'Planetary exploration requires a balance between preemptive planning and financial feasibility.  The risk of mid-mission equipment failure, power shortages, or supply depletion incentivizes precautionary measures, but the financial strain of sending unnecessary mass into space limits this practice. \\r\\n\\r\\nTo balance the two, our team explored the advantages of biological solutions, namely the self-sustaining abilities of low-mass organisms, to make planetary exploration more self-sufficient and economical. Prioritizing repair over replacement, we are developing self-healing materials embedded with Bacillus subtilis. For longer-lasting energy, we are designing a \\x93biobactery\\x94 using linearly oriented E. coli to generate power. For renewable materials, we are engineering bacteria to synthesize and degrade rubber. Individually, these projects offer sustainable alternatives for repair, power, and materials. But when combined, these consolidated insights can provide us with the power to get to Mars and resources to sustain us while we\\x92re there.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2235': {'team_id': '2235',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2235',\n",
       "  'team_name': 'Stockholm',\n",
       "  'wiki': 'http://2017.igem.org/Team:Stockholm',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Sweden',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'PROlung \\x96 a mucus degrading lung probiotic\\n',\n",
       "  'abstract': 'Until recently, the mucus layer of our lungs has remained unexplored. Many respiratory diseases such as asthma, chronic obstructive pulmonary disease (COPD) and cystic fibrosis are characterized by an excessive accumulation of thick mucus. Afflicted people suffer repeated lung infections and breathing difficulties caused by clogged airways, resulting in severely reduced quality of life. We address this issue by developing an unprecedented probiotic approach that self-regulates mucus thickness to protect and promote respiratory health. Our engineered bacteria sense the pathologically altered osmotic pressure caused by thickened mucus, thus triggering the expression of mucus degrading enzymes. For a proof-of-concept we degrade mucus, envisioning to clear the airways and consequentially remove entrapped pathogens and harmful particles. As pioneers in the field of lung probiotics, we challenge conventional treatments by exploring the lung microbiome as a potential solution to manage mucus-related respiratory diseases.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Entrepreneurship, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2196': {'team_id': '2196',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2196',\n",
       "  'team_name': 'Stony_Brook',\n",
       "  'wiki': 'http://2017.igem.org/Team:Stony_Brook',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Bacterioassassins: Development of hybrid bacteriocins to Target Methicillin-resistant Staphylococcus aureus\\n',\n",
       "  'abstract': 'MRSA is a prevalent threat due to its high resistance against multiple antibiotics, prompting health professionals to find alternative treatment options. Bacteriocins are ribosomally synthesized antimicrobial peptides produced by bacteria to selectively kill other strains of bacteria in their surrounding environment. Our study involves creating a hybrid of the bacteriocins, Lacticin Z and Aureocin A53 (or Epidermicin NI01) by connecting them with three glycine residues. These bacteriocins have shown effective inhibitory activity against MRSA strains; they form pores to penetrate the membrane of the pathogenic strain. We tested if the hybrid bacteriocins will have a synergistic effect and compared their cytotoxicity over time to that of the individual bacteriocins against MRSA. This was done by conducting two assays: minimum inhibitory concentration and the spot-on-lawn. Through development of hybrids, we aim to optimize the activity of bacteriocins and treat bacterial infections, particularly those with antibiotic resistance.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2497': {'team_id': '2497',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2497',\n",
       "  'team_name': 'Stuttgart',\n",
       "  'wiki': 'http://2017.igem.org/Team:Stuttgart',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Light up the Pipe  \\n',\n",
       "  'abstract': 'The clogging of drains and pipe systems by hair and fat is a serious problem in industry and private households. Currently many blockages are dissolved by toxic and reactive chemicals like sodium hydroxide and chlorine compounds. We want to engage this problem in a more sustainable way by developing a biological cleaner based on a holistic approach using E. coli. Our microbial system is targeted on producing and secreting enzymes to break down hair, fat and other pollutants. By optimizing the secretion of the selected enzymes, we are avoiding enzyme purification which can save valuable money and time. Additionally we want to produce a scent from the existing waste as an indicator of successful degradation. Involving mathematical modeling of enzymatic kinetics and degradation processes will support the experimental work. Concluding, we hope to reduce chemical waste and deal with an everyday problem more sustainably.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2216': {'team_id': '2216',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2216',\n",
       "  'team_name': 'SUIS_Alpha_Shanghai',\n",
       "  'wiki': 'http://2017.igem.org/Team:SUIS_Alpha_Shanghai',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'E-coli?? E-cOILi: Recycled Waste Cooking Oil Treatment System.\\n',\n",
       "  'abstract': 'Recycled waste cooking oil exhibits amplified volumes of Malondialdehyde (MDA), a mutagenic and carcinogenic substance which reacts with DNA to form adducts to deoxyguanosine and deoxyadenosine. The illegal practice returning this recycled waste oil back into the food industry can result in the ingestion of unsafe quantities of MDA. Malondialdehyde can be broken down into benign metabolites via the sequential enzymatic action of aldehyde dehydrogenase (specifically mitochondrial ALDH2) and malonic semialdehyde oxidative decarboxylase. The products of the reaction are acetyl-CoA with the release of CO2. We designed a genetic circuit which expresses these two enzymes in equal amounts and is supports the expression system of E.coli, rendering our device useful for the development of a biological treatment system which could form part of standard protocols relating to the treatment of waste oil in industrial oil refinery plants. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2492': {'team_id': '2492',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2492',\n",
       "  'team_name': 'SUSTech_Shenzhen',\n",
       "  'wiki': 'http://2017.igem.org/Team:SUSTech_Shenzhen',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'C.elegans in Skinner Box \\x97 the study of learning behavior based on optofluidics\\n',\n",
       "  'abstract': 'What is the distinction between the conditioned reflex of Pavlov\\x92s dogs and the behavioral reinforcement of Skinner\\x92s pigeons? What is the determination of the formation of behaviors and how to control it? Taking the response to alcohol of C.elegans as an example, a platform of optofluidics is established to provide deeper insights into these questions.\\r\\n\\r\\nExpression of two channelrhodopsins in the olfactory receptor neuron pair provides worms with the preference and aversion to specific wavelengths, and the lights are employed to reinforce their addictive or abstemious attitude to alcohol. The neo-behaviorism theory is expected to be verified in C.elegans, demonstrating the learning capability of model organisms based on both behavioral observation and quantification at the molecular level. \\r\\n\\r\\nHopefully, downstream neurons of the new-learned behavior will be revealed by using this platform, and it depicts a future of training human brains through optogenetics.\\n',\n",
       "  'award': 'Nominated for Best Foundational Advance Project;Nominated for Best Hardware;Nominated for Best New Basic Part',\n",
       "  'medal': 'gold'},\n",
       " '2379': {'team_id': '2379',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2379',\n",
       "  'team_name': 'SVCE_CHENNAI',\n",
       "  'wiki': 'http://2017.igem.org/Team:SVCE_CHENNAI',\n",
       "  'region': 'Asia',\n",
       "  'country': 'India',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'ReguloGEM - the ideal regulatory BioBrick \\n',\n",
       "  'abstract': 'Synthetic biologists over the years have regulated protein expression using two types of regulators - transcriptional and translational. While transcriptional regulators are easily composable and are capable of regulating multiple genes(operons) and complex genetic circuits, they are difficult to engineer. On the other hand, translational regulators are not capable of regulating multiple genes and complex genetic circuits but can be easily engineered de novo using predictive thermodynamic models. Hence an ideal regulator is found wanting, one which is capable of regulating multiple genes simultaneously and at the same time easily engineerable.\\r\\n\\r\\nReguloGEM provides such a regulator based on the tnaC operon. This regulator termed the Adaptor is capable of converting the translational regulatory property of riboswitches into transcriptional regulation. The Adaptor will be tested using temperature and pH sensitive riboswitches. We have also built a machine learning based tool that is capable of predicting the strength of sigma 70 promoters.\\r\\n\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2417': {'team_id': '2417',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2417',\n",
       "  'team_name': 'Sydney_Australia',\n",
       "  'wiki': 'http://2017.igem.org/Team:Sydney_Australia',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Australia',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Designing Insulin that is Single-Chain and Open-source (DISCO)\\n',\n",
       "  'abstract': 'Insulin is used to treat diabetes, which affects about 415 million people globally today. Currently, the control of the insulin market by a few large pharmaceutical companies has kept insulin prices very high, beyond the reach of many around the world who rely on insulin for survival.\\r\\n\\r\\nThis year, the University of Sydney iGEM team aims to address this problem of worldwide insulin inaccessibility. Our project involves using synthetic biology to develop an insulin manufacturing system that is cost efficient and simple, using the bacterial species Escherichia coli and Bacillus subtilis. We plan to have these microbes express proinsulin, the native human insulin, as well as our own newly designed open-source single-chain insulin, which we named \\x91Winsulin\\x92. We hope to develop a novel, cost efficient, and optimised pipeline for the production of proinsulin and Winsulin - a small step in making insulin affordable for all.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Poster, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2298': {'team_id': '2298',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2298',\n",
       "  'team_name': 'SYSU-CHINA',\n",
       "  'wiki': 'http://2017.igem.org/Team:SYSU-CHINA',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Stem Cell Woundplast\\n',\n",
       "  'abstract': 'Our goal this year is to provide a novel therapy for Asherman\\x92s Syndrome (AS) with engineered mesenchymal stem cells (MSCs). AS is characterized by adhesions in the endometrium, and often associated with dilation and curettage of the intrauterine cavity. Current treatment remains ineffective for severe adhesions, which may lead to infertility, repeated miscarriages, obstetric complications, even endometriosis. Our Woundplast aims at treating AS with MSCs in a safe and effective manner. To target and maintain engineered MSCs to the wound, we use Pluronic F-127, an FDA approved material as supporting matrix. Besides, we engineered MSCs to over-express PVDF, bFGF and VEGF individually or in combination to further promote wound healing. We test our idea in vitro with rat model. More importantly, we believe our project has the potential to be used in other types of wound healing, not just AS.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2214': {'team_id': '2214',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2214',\n",
       "  'team_name': 'SYSU-Software',\n",
       "  'wiki': 'http://2017.igem.org/Team:SYSU-Software',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Software',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'S-DIN\\x97\\x97Search engine and Design platform for Inspiration with Network analysis.\\n',\n",
       "  'abstract': 'With an exponential accumulation of circuit designs in synthetic biology, it becomes time-consuming to ponder out how to utilize the previous works in solving a new problem. This year, we create S-DIN for the ocean of projects facilitated by the power of network analysis and recommendation algorithms. It contains two main parts: Intelligent Search Engine and Embedded Design Platform. By drawing digital users portrait, our search engine can help users specify their needs and recommend related projects & parts. For the first time in synthetic biology, we introduced the big-data analysis to analyze the complex network of projects, parts, and topics, which supports our recommendation and search results in a global way. The embedded design platform allows you to search and edit a previous work simultaneously and related circuits & parts are recommended based on interaction database. Finally, the mathematical performance of circuits design will be simulated by our platform. \\r\\n\\n',\n",
       "  'award': 'Best Software Project;Nominated for Best Software Project;Nominated for Best Model;Nominated for Best Applied Design',\n",
       "  'medal': 'gold'},\n",
       " '2418': {'team_id': '2418',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2418',\n",
       "  'team_name': 'Szeged_SA_RMG',\n",
       "  'wiki': 'http://2017.igem.org/Team:Szeged_SA_RMG',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Hungary',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'METHUNGENY - Methane Biosensor Project\\n',\n",
       "  'abstract': 'Methane is a significant substance both in the environment and in the industry, thus indicating its presence has a great importance. Our aim is to develop a methane-biosensor. Our plan is to genetically modify a methanotroph bacterium (Methylococcus capsulatus) to produce lactate from methane or methanol. During its metabolism this bacterium produces pyruvate in the presence of methane. After having inserted the lactate-dehydrogenase gene, pyruvate is converted to have enough lactate for an enzymatic assay to change color when excess amount of methane is present in the environment of the bacteria. In the beginning the bacterium is cultivated in NMS medium containing methanol. For the genetic modification we use the lactate-dehydrogenase gene from Bacillus coagulans. First we transform E. coli with the cloned pMHE conjugation vector, which will transfer the LDH gene into M. capsulatus. The lactate concentration of the transformed bacterial medium will be determined with a lactate-assay.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2232': {'team_id': '2232',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2232',\n",
       "  'team_name': 'SZU-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:SZU-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'CON-cure-CRETE\\n',\n",
       "  'abstract': \" This year we designed a self-healing system for concrete. When there is a microcrack our system can be switched on and concrete can start to heal themselves. We transfered gerA gene to be our biosensor. When there is liquid L-alanine. GerA receptor can be induced so the whole chain can be started. we improved the alkali resistance by transfer nhaC. We put the spores of our Bacillus subtilis into microcapsules, along with  nutrients and L-alanine powder. And mix the microcapsule with concrete. When there is a microcrack, the tension of the wall breaking will also break the microcapsule and the water will infiltrate, after the germination that induced by L-alanine. The Carbonic anhydrase gene that we transfered will let CO2 hydrate to produce CO32-, which then binds with free Ca + in the environment to form calcium carbonate sediment,so the microcracks are filled.The rebars inside won't rust.\\n\",\n",
       "  'award': 'Best Manufacturing Project;Nominated for Best Manufacturing Project',\n",
       "  'medal': 'gold'},\n",
       " '2349': {'team_id': '2349',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2349',\n",
       "  'team_name': 'Tartu_TUIT',\n",
       "  'wiki': 'http://2017.igem.org/Team:Tartu_TUIT',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Estonia',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Yeasthylene\\n',\n",
       "  'abstract': 'Ethylene is the building block of many chemical compounds. Its polymer, polyethylene, is the main component in numerous plastic materials. Due to its versatility, the demand for ethylene has been increasing during the last decade. As it is derived from petroleum, which is a non-renewable source, it is necessary to find an environmentally friendly way of producing ethylene to satisfy this demand. In this project, two yeast strains with completely different roles will be genetically modified to produce ethylene from sucrose. The focus of the project is set to make those strains dependent on each-other and to provide a balanced growth. This approach represents a more efficient method than cloning the whole pathway into one population, especially when longer heterologous pathways will be used in the future to produce more complex chemicals. Energy gain due to lower metabolic burden and balanced co-factor metabolism will result in higher production rates.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2229': {'team_id': '2229',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2229',\n",
       "  'team_name': 'TAS_Taipei',\n",
       "  'wiki': 'http://2017.igem.org/Team:TAS_Taipei',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Taiwan',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'NANOTRAP; Nanoparticle removal from wastewater systems\\n',\n",
       "  'abstract': 'The small size of nanoparticles is both an advantage and a problem. Their high surface-area-to-volume ratio enables novel medical, industrial, and commercial applications. However, their small size also allows them to evade conventional filtration during water treatment, posing health risks to humans, plants, and aquatic life. Our project aims to remove nanoparticles using two approaches: 1)  bind citrate-capped nanoparticles with the membrane protein proteorhodopsin and 2) trap nanoparticles using E. coli biofilm produced by overexpressing two regulators -- OmpR234 and CsgD. We envision integrating our trapping system in both rural and urban wastewater treatment plants to efficiently capture all nanoparticles before treated water is released into the environment.\\r\\n\\n',\n",
       "  'award': 'Best Wiki, High_school;Nominated for Best Wiki, High_school;Nominated for Best Presentation, High_school;Nominated for Best Poster, High_school;Nominated for Best Integrated Human Practices, High_school;Nominated for Best Part Collection, High_school;Finalist, High_school;Nominated for Finalist, High_school',\n",
       "  'medal': 'gold'},\n",
       " '2383': {'team_id': '2383',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2383',\n",
       "  'team_name': 'TCFSH_Taiwan',\n",
       "  'wiki': 'http://2017.igem.org/Team:TCFSH_Taiwan',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Taiwan',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Detecoli : A biomonitor sticker on freights\\n',\n",
       "  'abstract': 'DETECOLI\\x97a word we created by combining \\x93detection\\x94 and \\x93E. coli\\x94\\x97which changes color in environments of excess sunlight or high temperatures. The detection should be visualized by discriminating the alteration of three chromoproteins (cjBlue, RFP, BFP) expressed by E. coli with the regulatory circuit of temperature and UV light. We aim to build up a quantitative experiment for observing the color of E. coli. For the practical application we aim to place E.coli on stickers and attach the stickers on the product before delivery, thus able to monitor the whole process of transportation. DETECOLI alerts consumers to possible deterioration or contamination by changing color, and serves as a guarantee of quality.\\r\\n\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2471': {'team_id': '2471',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2471',\n",
       "  'team_name': 'Tec-Chihuahua',\n",
       "  'wiki': 'http://2017.igem.org/Team:Tec-Chihuahua',\n",
       "  'region': 'Latin America',\n",
       "  'country': 'Mexico',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Erwinions: Quenching the fire out of Fire Blight\\n',\n",
       "  'abstract': \"Erwinia amylovora causes fire blight disease worldwide in some important crops such as apple, roses, pear and most Rosaceae's family members. For example, the largest Latin American apple producer has 3,000 hectare from which 50% are estimated to have the disease. This iGEM edition, Tec-Chihuahua presents its proposal to address this environmental/economical issue by using synthetic biology techniques to synthesize three enzymes that might inhibit most, if not all, of the virulence factors. The use of N-Acyl homoserine lactonase would directly affect the AHLs by hydrolyzing the main quorum sensing molecule. Then, the Cyclic-di-GMP phosphodiesterase would linearize the c-di-GMP avoiding the formation of biofilm while encouraging motility. Nevertheless, Tec-Chihuahua proposes to arrest flagellar rotation with a glycosyltransferase. As these are intracellular proteins, the pathogen should be genetically modified and tested hoping for a descent. Afterwards, the commercial and technical viability of a theoretical biocontrol would be developed as real proposal.\\n\",\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2246': {'team_id': '2246',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2246',\n",
       "  'team_name': 'TecCEM',\n",
       "  'wiki': 'http://2017.igem.org/Team:TecCEM',\n",
       "  'region': 'Latin America',\n",
       "  'country': 'Mexico',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'We are an Undergraduate Team not Overgrad.\\n',\n",
       "  'title': 'Silencing multiple Diaphorina citri genes using siRNA for control of Huanglongbing disease in citrus plantations.\\n',\n",
       "  'abstract': 'Huanglongbing disease (HLB) is caused by Candidatus liberibacter asiaticus (CLas), a phloem-limited bacteria, transmitted by the Asian citrus psyllid  Diaphorina citri (ACP).  Current HLB control methods include insecticides and antibiotics that present short-term solutions, that are not specific and may cause a negative impact in plants and the final product. The use of RNAi technology has been widely used in research of gene silencing. Team TEC CEM designed siRNAs targeting four different D. citri genes: Abnormal wing disc (Awd), Wingless (WNT), Superoxide dismutase 1 (CSD1) and Rac-like GTP-binding protein 1 (RacI) to prevent CLas infection and the spread of HLB.  RNAi has been proposed as a control mechanism for D. citri, but has only been tested using long dsRNA, therefore we developed siRNAs to maximize the specificity, reducing off-targeting.  To ensure effective delivery, siRNAs will be encapsulated using nanotechnology for their direct application to citrus plants. \\r\\n\\r\\n\\r\\n\\r\\n\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2520': {'team_id': '2520',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2520',\n",
       "  'team_name': 'TECHNION-ISRAEL',\n",
       "  'wiki': 'http://2017.igem.org/Team:TECHNION-ISRAEL',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Israel',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': ' Tolegen: A preventative treatment for autoimmune disease and allergies\\n',\n",
       "  'abstract': 'Autoimmune disease and allergies are an increasingly common phenomenon in the Western world. An estimated 73 million people suffer from allergies and autoimmune disease in the United States. To date, many of these diseases lack treatment and pose an incredible financial burden on both the patients and society.\\r\\nWe intend to design a preventative treatment for allergies and autoimmune disease by utilizing the innate system of Central Tolerance. We\\x92ve designed a plasmid based platform that will allow inducible expression and display of target antigens on the membrane of Hematopoietic Stem Cells (HSCs). Hematopoietic progenitor cells (HPC-7) will be transfected with our plasmid and we will attempt to show apoptosis in an immature B cell model (WEHI-231) closely approximating the process of Central Tolerance. In the future this technology may be used to engineer HSCs harvested from cord blood and allow for a cost effective preventative treatment.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2434': {'team_id': '2434',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2434',\n",
       "  'team_name': 'TecMonterrey_GDA',\n",
       "  'wiki': 'http://2017.igem.org/Team:TecMonterrey_GDA',\n",
       "  'region': 'Latin America',\n",
       "  'country': 'Mexico',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'PHAgave\\n',\n",
       "  'abstract': '\"PHAgave\" aims to synthesize polyhydroxyalkanoates (PHA), a form of bioplastic naturally produced by bacterial fermentation, from the available carbon sources in residues from the tequila production process through a recombinant E. coli strain. The study compares the metabolic speed for the PHA synthesis between Pseudomonas putida KT2440 and a recombinant E. coli with the genes for PHA synthesis (ACC, fabG, phaG, phaC1 and phaC2) for a continuous bioplastic production. This stage also includes the extraction of the PHAs from the intracellular medium in a more sustainable way than a chemical lysis.The main objective is to make the process efficient, clean, and with a high yield. The obtained bioplastic has multiple applications; for example, it can be used for the creation of disposable laboratory equipment and containers due to its resistance to heat and oils. A business model for the commercialization of our bioplastic is included.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2408': {'team_id': '2408',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2408',\n",
       "  'team_name': 'Tel-Hai',\n",
       "  'wiki': 'http://2017.igem.org/Team:Tel-Hai',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Israel',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Wonder Wine\\n',\n",
       "  'abstract': 'The purpose of our project is to improve upon the quality of wine nutritional value production by way of genetically engineered yeast, aimed at several objectives. The first is spoilage of the wine due to the undesired presence of the Brettanomyces yeast. We intend to target this obstacle by secretion of Brett-specific toxins by our \"designer\" yeast. Second, enhancing the health benefits of wine, specifically antioxidant content, by increasing presence of reservatrol. Third, lowering the glycemic index of wine while maintaining its flavor and quality by introducing the Miraculin molecule.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2407': {'team_id': '2407',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2407',\n",
       "  'team_name': 'Tianjin',\n",
       "  'wiki': 'http://2017.igem.org/Team:Tianjin',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Romantic Switcher\\n',\n",
       "  'abstract': 'We developed a novel gene switcher based on the mating-type switch and the mating behavior of Saccharomyces cerevisiae, namely \\x93Mating Switcher\\x94. Two groups of MATa haploid yeast with different functional genes initially work separately. Then by activating the inducible promoter in one group, the MATa yeast in this group will become MATα yeast to mate with another group. With vika/vox system, the original function can be shut off, and new function can be launched after mating. \\r\\nWe applied our mating-type switcher to a controllable system of heavy metal enrichment for further disposal. To realize the separation of different ions in polluted water, we designed a gene circuit enabling the adsorption of different metal ions (Cu2+, Cd2+ ) in chronological order. And thanks to the flexibility and feasibility of our mating-type switcher, there will be more applications waiting us to exploit and develop.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2328': {'team_id': '2328',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2328',\n",
       "  'team_name': 'TJU_China',\n",
       "  'wiki': 'http://2017.igem.org/Team:TJU_China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Utilizing a Novel Infrared Fuorescent System to Track Intestinal Bacteria in Real Time\\n',\n",
       "  'abstract': 'In recent years, the researchers have revealed the key role of intestinal microbiota. There is an increasing number of evidences indicating that gut flora can really influence our thinking, mood, behavior, and feelings. However, there exists difficulties in tracking intestinal bacteria in a living body. This year we achieve this goal by using a novel fluorescent system. Several important enteropathogens and probiotics are on our list, including facultative anaerobe and obligate anaerobe, like EHEC O157:H7, Bifidobacterium longum and so on. We construct different shuttle vectors to express this fluorescent system and successfully prove that it works well. Through the expression system, bacteria with fluorescence can be detected in a living body, making in vivo imaging come true. \\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2279': {'team_id': '2279',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2279',\n",
       "  'team_name': 'TMMU-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:TMMU-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Development of Quorum Sensing Tool Kit for Gram-positive Bacteria\\n',\n",
       "  'abstract': 'Gram-positive bacteria comprise various kinds of microbes. Quorum sensing (QS) system can play diverse roles in response to bacteria population density, which makes it an intriguing tool for synthetic biologists. Most of the QS systems are relied on the N-acyl homoserine lactones (AHLs) based QS tool kit of Gram-negative bacteria, however, QS tool kit for Gram-positive bacteria has rarely reported. In this project, we want to develope a QS tool kit for Gram-positive bacteria. We will build this tool kit based on the Agr system from S.aureus, the PlcR-PapR system from Bacillus cereus, and the AimR-AimP system from the Bacillus subtilis bacteriophage Phi3T. We are going to test the utility of this tool kit in Bacillus subtilis and Lactococcus lactis. This QS tool kit will facilitate synthetic biologists to construct more sophisticated systems, both in Gram-positive bacteria and mixed microbial consortia.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2496': {'team_id': '2496',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2496',\n",
       "  'team_name': 'TNCR_Korea',\n",
       "  'wiki': 'http://2017.igem.org/Team:TNCR_Korea',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Korea, Republic Of',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Transformation of Dipeptidyl peptidase-4(DPP-4) to Intestinal Gut Microflora: A possible alternative of \\x93Gluten-free Diet\"\\n',\n",
       "  'abstract': 'Gluten-related disorders, including celiac disease(CD) and non-celiac gluten sensitivity, are found worldwide. Currently available therapy for CD patients, permanent gluten-free diet(GFD), has drawbacks of restricted nutrition and lifelong remission. Thus, we considered a non-dietary therapy that utilizes synthetic biology targeting the gut microflora. Dipeptidyl peptidase-4(DPP-4) gene, which is known to decompose the gluten, was inserted to the iGEM provided vectors through infusion cloning based on the 3A Assembly method. Western blot analysis using Flag-tag was conducted to verify the expression of the target gene. Because excessive expression of DPP-4 may induce stress and inflammation, we added Anderson promoter that differentially regulates gene expression in three levels to achieve optimal expression of DPP-4 of an individual. The transformed dominant gut bacteria can be embedded into a bacteria-favorable yogurt for the subsequent probiotic administration. This study proposes a potential novel treatment of gluten-related disorders through recombinant gut bacteria\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2505': {'team_id': '2505',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2505',\n",
       "  'team_name': 'TokyoTech',\n",
       "  'wiki': 'http://2017.igem.org/Team:TokyoTech',\n",
       "  'region': 'Asia',\n",
       "  'country': 'Japan',\n",
       "  'track': 'Information Processing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Coli Sapiens: Co-Culture System of Human Cells and Bacteria Sustained by Cross-Kingdom Talk\\n',\n",
       "  'abstract': 'Gene therapy has been expected in cancer therapy for years. An interesting therapy for cancer using anaerobic bacteria as a carrier has been developed, but after the anaerobic cancer region is diminished, the bacteria cannot stay there anymore. If anti-cancer bacteria can stay in affected area, they promptly respond to cancer recurrence. Co-existence of bacteria and host cells should be quite difficult in our body or human cell culture systems, because bacteria grow so fast. It is important to control bacterial proliferation in them. So, we try to establish a new living system that human cells control the population of bacteria by engineering the both cells by creating two signaling pathways of 1) Bacteria-Mammals and 2) Bacteria-Plants. We expect that this system will lead to a new experimental approach and a new medical therapy. Moreover, we imagine about \"A boundary between cellular groups and living organisms\" with general public.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2374': {'team_id': '2374',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2374',\n",
       "  'team_name': 'Tongji_China',\n",
       "  'wiki': 'http://2017.igem.org/Team:Tongji_China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'A new use of UAS/Gal4 system for pest control\\n',\n",
       "  'abstract': 'The spread of many diseases is related to mosquitoes, such as dengue, Zika, and chikungunya, which pose a threat to human health. And many pests also have a negative impact on agricultural production.\\r\\n\\r\\nOur project is focusing on addressing the growing number of pest-induced disasters and outbreaks by changing sexual orientation. We use Drosophila melanogaster as chassis organisms, and use the classic UAS/Gal4 system to regulate this character. What\\x92s more, we also build models to simulate the effects.\\r\\n\\r\\nCompared to other traditional ways, this method is heritable and continuable, and causes less impact on the ecosystem. In addition, the gene will be activated in summer, when pests thrived. We hope that our project will provide a new way to solve the problems caused by pest.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2469': {'team_id': '2469',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2469',\n",
       "  'team_name': 'Toronto',\n",
       "  'wiki': 'http://2017.igem.org/Team:Toronto',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Light, Lac and LOV: A light-regulated switch for control of gene expression\\n',\n",
       "  'abstract': 'iGEM Toronto has developed a genetic switch that will allow stringent control of transcriptional activity using LacILOV, a novel light-regulated transcriptional modulator, and cI, a viral repressor. Using blue light as an input, our tool will enable users to manipulate desired gene outputs. As a proof of the utility of our switch, we propose a system that will permit spatiotemporal control of CRISPR activity using blue light. By integrating anti-CRISPR proteins and sgRNAs as our outputs, we demonstrate the applicability of our design to CRISPR-Cas9 gene editing technology. Given the foundational nature of our project, we have devised a guide that will aid researchers in considering the ethical, legal and socio-economic implications of the various applications of our system. Using human gene editing as an example, we aim to create a framework that facilitates reflexive dialogue, incorporating feedback from various potential stakeholders.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2369': {'team_id': '2369',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2369',\n",
       "  'team_name': 'TP-CC_San_Diego',\n",
       "  'wiki': 'http://2017.igem.org/Team:TP-CC_San_Diego',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Cancer Research Utilizing CRISPR based ecDNA Modification\\n',\n",
       "  'abstract': 'Cancer, a genetic disease resulting in uncontrollable cell growth, is mostly caused by somatic mutations acquired throughout an individual\\x92s lifetime. Because it induces the increased expression of growth related genes, oncogene amplification is one of the driving forces of cancer cell replication.  Recently, it was discovered that some oncogenes resided on extrachromosomal DNA (ecDNA). Like the DNA on chromosomes, ecDNAs are double stranded. A key difference, however, is the circular shape of ecDNAs; they are able to randomly distribute because they do not have centromeres, which increases heterogeneity in daughter cells. This can cause the cancerous tumors to develop faster resistance to current treatments. To target the ecDNA, we used CRISPR technology to create double strand breaks specifically in the ecDNA. Because ecDNA causes oncogene copy number to increase exponentially, utilizing CRISPR to create breaks in ecDNA decreases cancer cells\\x92 replication speed.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2227': {'team_id': '2227',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2227',\n",
       "  'team_name': 'Troyanos-UDEM_UFU-MX',\n",
       "  'wiki': 'http://2017.igem.org/Team:Troyanos-UDEM_UFU-MX',\n",
       "  'region': 'Latin America',\n",
       "  'country': 'Mexico',\n",
       "  'track': None,\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': ''},\n",
       " '2247': {'team_id': '2247',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2247',\n",
       "  'team_name': 'Tsinghua',\n",
       "  'wiki': 'http://2017.igem.org/Team:Tsinghua',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'YeasyAFT: An easy-to-use biosensor of aflatoxin based on yeast two-hybrid assay \\n',\n",
       "  'abstract': 'We engineered yeasts to be biosensors of aflatoxin (AFT), a carcinogen to human. Often produced by molds when foods decay, AFT is easily found in everyday foods such as grains, oil and milk. However, its detection remains expensive and inconvenient, requiring special equipment. We developed a biosensor of aflatoxin easy for household use which gives a warning of AFT levels beyond safety limit in oil and milk. Utilizing the yeast two-hybrid (Y2H) assay, we fused two partial antibodies (single-chain variable fragments) targeting aflatoxin with Y2H elements to drive the expression of the reporter gene in response to AFT. The reporter is a hexose transporter gene linking the level of AFT detected to the concentration of glucose in medium, easily read out by a glucometer. Thus we combine the sensitivity of antibody reaction, flexibility of Y2H assay and convenience of glucometer readout to facilitate household detection of AFT in everyday foods.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2250': {'team_id': '2250',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2250',\n",
       "  'team_name': 'Tsinghua-A',\n",
       "  'wiki': 'http://2017.igem.org/Team:Tsinghua-A',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Information Processing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'E.coli War\\n',\n",
       "  'abstract': 'During a war, what makes one side outcompete the other? Is it the more powerful weapons, more sufficient logistic supply  or better cooperation and communication? Our bacteria can tell! Here we devise an interesting interactive system with E.coli from different \\x91sides\\x92, with each of them incorporated with functionally independent roles including warrior, farmer and beggar, based on certain gene circuit and the mechanism of quorum sensing. By combining experimenting and modeling, we monitor the ratio and capability of each category of individuals to obtain the factors to facilitate the victory of a group. With our efforts to explore synthetic biology as our tool, more ecological rules will be tested and more stories about inter- and inner-species cooperation and competition will be uncovered. From these, we can have a better understanding at the rules of competition or \\x91warfare\\x92 in the ecosystem as well as in the human society.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2356': {'team_id': '2356',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2356',\n",
       "  'team_name': 'TU-Eindhoven',\n",
       "  'wiki': 'http://2017.igem.org/Team:TU-Eindhoven',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Netherlands',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'GUPPI: Gelation Using Protein Protein Interactions\\n',\n",
       "  'abstract': 'Cancer is a major cause of death worldwide. With our project, GUPPI, we hope to set the basics of a system that can encapsulate the cancerous tissue to prevent metastasis in an early stage. GUPPI utilizes Protein Protein Interactions and is inspired by the formation of membraneless organelles by multivalent interactions. A designed protein scaffold and its binding partner, both having sequential repeating units, will respond to an inducer and form a gel-like structure. We envision that later on, GUPPI can respond to extracellular conditions that will act as an inducer of the gelation to specifically target and encapsulate the desired tissue. The GUPPI system has some major advantages, such as the tunability of the protein\\x92s multivalency and the possible adaptation of interactions. Furthermore, a rule-base-model is developed to predict, verify and characterize the wet-lab experiments and act as an additional support.\\n',\n",
       "  'award': 'Nominated for Best New Application Project',\n",
       "  'medal': 'gold'},\n",
       " '2306': {'team_id': '2306',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2306',\n",
       "  'team_name': 'TUDelft',\n",
       "  'wiki': 'http://2017.igem.org/Team:TUDelft',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Netherlands',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'CASE13A - Cutting our way through the antibiotic resistance problem\\n',\n",
       "  'abstract': 'The existing methods to detect the presence of a specific RNA sequence in a sample are laborious and require both trained personnel and sophisticated lab equipment. Inevitably, these restrictions limit the development of diagnostic detection methods based on specific RNA sequences. Here, we aimed to overcome these limitations by using the CRISPR-Cas protein Cas13a. Once this recently characterized protein binds to its complementary target, it unspecifically cleaves all surrounding RNA. Utilizing this property we designed a system for the detection of antibiotic resistance genes in agricultural pathogens, aiming to combat the global rise of antibiotic resistance. We developed a procedure to extract biological samples and detect RNA sequences of interest by converting the collateral cleavage activity of Cas13a to a read-out visible to the naked eye. Through ongoing interaction with direct stakeholders, we made substantial progress towards making not only durable components but a cheap, safe and reliable detection method.\\n',\n",
       "  'award': 'Best New Application Project, Overgrad;Nominated for Best New Application Project, Overgrad;Nominated for Best Wiki, Overgrad;Best Presentation, Overgrad;Nominated for Best Presentation, Overgrad;Nominated for Best Poster, Overgrad;Best Integrated Human Practices, Overgrad;Nominated for Best Integrated Human Practices, Overgrad;Best Education & Public Engagement, Overgrad;Nominated for Best Education & Public Engagement, Overgrad;Nominated for Best Model, Overgrad;Best Measurement, Overgrad;Nominated for Best Measurement, Overgrad;Best Entrepreneurship, Overgrad;Nominated for Best Entrepreneurship, Overgrad;Nominated for Best Applied Design, Overgrad;Best New Composite Part, Overgrad;Nominated for Best New Composite Part, Overgrad;Best Part Collection, Overgrad;Nominated for Best Part Collection, Overgrad;Finalist, Overgrad;Nominated for Finalist, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2372': {'team_id': '2372',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2372',\n",
       "  'team_name': 'Tuebingen',\n",
       "  'wiki': 'http://2017.igem.org/Team:Tuebingen',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'The Advanced Trojan Horse - a new approach for designing resistance-specific antibiotics\\n',\n",
       "  'abstract': 'The rising amount of microbial resistances urge for innovation and new therapeutics. Many substances already available in the lab are potent as antibiotics but cannot be used in humans due to toxicity or problems in delivery.\\r\\nTherefore, we used aminocoumarins and created a new derivative via a semi-biosynthetic synthesis approach. The introduction of specific modifications changes the pattern of interaction with proteins while also improving the chemical properties of aminocoumarins. Our modification specifically targets ß-lactam resistant pathogens. For proof of principle, we developed a suitable test system consisting of an E.coli-based pathogen model and a toxicity assay. Additionally, we performed an in silico prediction of changes in binding affinity to target proteins.\\r\\nIn conclusion, we created an antibiotic that is only activated by ß-lactam resistant pathogens itself via enzymatic cleavage while not affecting other cells in the human body - the advanced Trojan horse is set free.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2416': {'team_id': '2416',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2416',\n",
       "  'team_name': 'Tufts',\n",
       "  'wiki': 'http://2017.igem.org/Team:Tufts',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': None,\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': ''},\n",
       " '2267': {'team_id': '2267',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2267',\n",
       "  'team_name': 'TUST_China',\n",
       "  'wiki': 'http://2017.igem.org/Team:TUST_China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Research into Increasing Yields of Bacterial Cellulose via Methods of Synthetic Biology\\n',\n",
       "  'abstract': '   Bacterial cellulose is a novel nano-material synthesized by microbiology. BC has a unique three-dimensional net structure with high degree of biological histocompatibility and degradability. Therefore, it has a wide range of application.However,low yield and high cost of producing BC makes it  way behind reaching  practical need of production and application.Hence, we intended to implement synthetic biology to improve the yield of bacterial cellulose.\\r\\n  \\r\\n1.Refinement of fermentation conditions.\\r\\nIt has been proved by research that G.xylinus can reach its highest yield of bacterial cellulose  when  pH=4.8.Therefore, we intend to control the continually descending pH of the environment during fermentation and maintain its pH to around 4.8.\\r\\n  \\r\\n2.Partial alternation of carbon flux distribution.\\r\\n  \\r\\n  We hope we can redirect and improve the carbon flux distribution to the pathway of producing bacterial cellulose via over-expressing some key enzymes during the fermetation of G.xylinus.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2380': {'team_id': '2380',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2380',\n",
       "  'team_name': 'TU_Darmstadt',\n",
       "  'wiki': 'http://2017.igem.org/Team:TU_Darmstadt',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'ChiTUcare\\n',\n",
       "  'abstract': 'Over the course of the last decades wound infections have become a major issue in daily medical routine. In this context we present a method combining continuous wound screening with next generation wound care. To accomplish fast wound screening, we introduce a tool to detect individual bacterial proteases. Here, non-toxic fluorophores are linked to chitosan oligomers by sequence-specific peptide linkers within a hydrogel matrix. This method excels by being painless, quick and non-invasive. Chitosans show antimicrobial and wound-healing supportive properties, making chitosan oligomers excellent plaster materials and fluorophor-carriers. De novo biosynthesis in E. coli makes it possible to produce chitosan oligomers with different defined deacetylation patterns which differ in their bioactivity and are therefore useful for various applications for instance plant protection and medical materials.\\r\\nFor this purpose different chitindeacetylases, with unique deacetylation patterns can be individually expressed in E. coli by orthogonal regulation via a split-T7-RNAP regulation system. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2273': {'team_id': '2273',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2273',\n",
       "  'team_name': 'TU_Dresden',\n",
       "  'wiki': 'http://2017.igem.org/Team:TU_Dresden',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'EncaBcillus \\x96 It\\x92s a trap! \\n',\n",
       "  'abstract': 'Synthetic biology wants to go beyond the pure biological by integrating concepts from chemistry or physics into the living world. At this interphase, our project wants to introduce Peptidosomes as a new fundamental approach for generating and applying encapsulated bacteria. \\r\\nThese spheres possess advantageous properties like stability in different media and a mesh-like structure that allows for the selective exchange of compounds via diffusion. Therefore, we are able to benefit from the entrapped cells\\x92 abilities, while ensuring that they are not released into their surroundings.\\r\\nUsing the powerful genetics of Bacillus subtilis and its secretory capabilities we demonstrate communication and cooperation between separately encapsulated bacterial populations as well as the environment. Peptidosomes can be further enhanced by incorporating magnetic or biological beads \\x96 which can be functionalized with proteins \\x96 into their peptide-based shell.\\r\\nWith this unique setup, we provide a whole new universe of applications to the iGEM community.\\n',\n",
       "  'award': 'Nominated for Best Foundational Advance Project, Overgrad;Nominated for Best Poster, Overgrad;Nominated for Best New Basic Part, Overgrad;Nominated for Best Part Collection, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2389': {'team_id': '2389',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2389',\n",
       "  'team_name': 'UAlberta',\n",
       "  'wiki': 'http://2017.igem.org/Team:UAlberta',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'The RISE System: Recombinant protein Interaction Screening and Enrichment System\\n',\n",
       "  'abstract': 'Protein therapeutics are a front line approach to treatments for cancer, infections, autoimmune disorders, and other diseases. Developing new protein-based therapeutics require methodologies, particularly directed evolution,  that enable the engineering of protein-protein interactions with high specificity and affinity. Due to the iterative mutagenesis required for directed evolution, optimizing protein interactions can become tedious. To enhance enrichment of desired variants Team UAlberta designed a protein-protein interaction assay based on the reconstitution of adenylate cyclase in Escherichia coli to accelerate  screening for successful variants. We have built two constructs to act as reporters for our system. The first contains genes required for gas vesicle formation and bacterial buoyancy while the second encodes for fluorescent proteins. Our current efforts are focused on characterizing and validating our system using well-characterized protein interactions and our reporters. \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2402': {'team_id': '2402',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2402',\n",
       "  'team_name': 'UBonn_HBRS',\n",
       "  'wiki': 'http://2017.igem.org/Team:UBonn_HBRS',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Germany',\n",
       "  'track': 'Therapeutics(P)',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': 'Dna Dental Care\\n',\n",
       "  'abstract': 'Brushing your teeth. An everyday routine. Yet some of us still get caries, no matter how meticulous our oral hygenie plan is. It is time for a new innovative approach in the fight against caries. We aspire to achieve this goal by expressing surface proteins of the cariogenic pathogen Streptococcus mutans in Escherichia coli. We will perform SELEX on these proteins to obtain aptamers, which specifically bind to S. mutans and prevent the formation of biofilm. And thus the formation of Caries.\\n'},\n",
       " '2287': {'team_id': '2287',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2287',\n",
       "  'team_name': 'UCAS',\n",
       "  'wiki': 'http://2017.igem.org/Team:UCAS',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'We Fit Fish\\n',\n",
       "  'abstract': 'Aquatic products provide Chinese people with gourmet foods as well as high-quality proteins. In China, freshwater aquaculture products contribute up to 68% to total aquaculture products. Nevertheless, in freshwater aquaculture, there exists a major problem of excessive ammonia nitrogen in pond water, which is toxic to aquatic creatures. Although ammonia nitrogen concentration is critical in water quality evaluation, few farming ponds in China are equipped with ammonia monitoring systems, causing fish diseases and reduced aquaculture production. This year, UCAS iGEM team engineered E.coli to sense ammonia in pond water and alert farmers to high ammonia concentration. Furthermore, to remove the excess nitrogen from water, we introduced human xanthine oxidase to transfer ammonia into uric acid precipitation. Consequently, breeding density can be raised and farming water can be recycled, thus reducing costs and increasing benefits. Our uric acid-producing microorganism can also serve as a platform to select effective drugs for gout.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2344': {'team_id': '2344',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2344',\n",
       "  'team_name': 'UCC_Ireland',\n",
       "  'wiki': 'http://2017.igem.org/Team:UCC_Ireland',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Ireland',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'MOOnshine; biosensors for antibiotics and methanol\\n',\n",
       "  'abstract': 'To ensure consumer safety, all processed food and beverages are subject to rigorous testing for contaminants that are hazardous to human health. This testing can be expensive, inaccessible and time-consuming. Since these sophisticated tests are often limited to large-scale producers, local dairy farmers, microbreweries and home-brewers remain vulnerable to penalties and poisoning should the level of contaminants in their products fall outside the regulatory guidelines. Our iGEM project aims to create a biosensor, specifically to detect antibiotic residues in milk and methanol in alcohol. The universal readout will incorporate a blue chromoprotein, AmilCP, which will correlate with the concentration of the contaminant and be quantified with a portable colourimetric device linked to smartphones. A cell-free system will be utilised to circumvent the risks associated with the use of genetically-modified live bacteria outside the laboratory.\\r\\n \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2428': {'team_id': '2428',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2428',\n",
       "  'team_name': 'UChicago',\n",
       "  'wiki': 'http://2017.igem.org/Team:UChicago',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Expression of Centromeric Activity by a Chromosomal Integration Plasmid in E. coli and Pichia pastoris\\n',\n",
       "  'abstract': 'Centromeric plasmids combine the stability of a chromosome and the flexibility of a plasmid to create the perfect tool for the biotechnical industry. Pichia pastoris, a strain of yeast often used in this field, lacks a centromeric plasmid, which limits the research that can be done with this species. The University of Chicago iGEM Team, GeneHackers, is conducting research to create a centromeric plasmid for Pichia pastoris. Such a plasmid that will be useful for a multitude of purposes, including bioengineering and industrial applications. We aim to find and incorporate the minimal sequence within the genome of Pichia pastoris that allows our modified version of the iGEM plasmid psB1C3 to behave as part of the yeast\\x92s chromosomes. The combination of DNA from both E. coli and Pichia pastoris will lead to the creation of a yeast centromeric plasmid that can act as a shuttle vector between the two organisms.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2289': {'team_id': '2289',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2289',\n",
       "  'team_name': 'UChile_Biotec',\n",
       "  'wiki': 'http://2017.igem.org/Team:UChile_Biotec',\n",
       "  'region': 'Latin America',\n",
       "  'country': 'Chile',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Bimatox: A biosensor made of aptazyme to help people detect marine toxins in their environment\\n',\n",
       "  'abstract': 'BiMaTox is a biosensor that detects marine toxins that are produced during harmful algal blooms, also known as red tides. Of these toxins, saxitoxin is the most-deadly, as it attacks the human nervous system impeding synapse formation. The biosensor consists of a cell-free cellulose matrix device that displays a color in the presence of the toxin.  This color is produced by the aptazymes which are contained within the device. The aptazyme consist of a specific toxin aptamer connected to a DNAzyme. When the toxin binds to its specific aptamer, the peroxidase activity of DNAzyme is triggered and produces the oxidation of a compound called ABTS, which generates a color that is readily visible to the human eye. The device is constructed in such a way that it is easy and simple to use, with the aim that, for example, fishermen can know when there are toxins in their fishing area.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2425': {'team_id': '2425',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2425',\n",
       "  'team_name': 'UChile_OpenBio-CeBiB',\n",
       "  'wiki': 'http://2017.igem.org/Team:UChile_OpenBio-CeBiB',\n",
       "  'region': 'Latin America',\n",
       "  'country': 'Chile',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Greenhardtii Project\\n',\n",
       "  'abstract': 'During the year 2015, the worldwide mean of atmospheric CO2 concentration surpassed the threshold of 400 ppm and will keep increasing. But can CO2 be thought of as an exploitable resource?\\r\\nGreenhardtii Project is an initiative that searches to generate a green microalgae with optimized capacity of carbon uptake, using this as cellular fuel to use it as a production platform of desired biomolecules.\\r\\n\\r\\nThe optimization of the Calvin Cycle in our Greenhardtii platform (Green + Chlamydomonas reinhardtii) is produced by the expression of a cyanobacteria enzyme. Moreover, test of the kinetic behavior of a regulable promoter and pathways inhibition will be made in a mathematical model.\\r\\n\\r\\nTo Greenhardtii Project, linking science with society is indispensable, so the design of a photobioreactor is being developed, in order to propose instalations in one of the sectors with the worst air standards in Chile \\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2332': {'team_id': '2332',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2332',\n",
       "  'team_name': 'UCL',\n",
       "  'wiki': 'http://2017.igem.org/Team:UCL',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Light Induced Technologies (LIT): Advancing the Optical Control of Cellular Mechanisms\\n',\n",
       "  'abstract': 'We are developing optogenetic tools to improve two areas of human life: architecture and regenerative medicine. In contrast to current mechanical and chemical methods for gene control, our light-based strategies allow for the precise spatiotemporal and non-invasive control of complex gene circuits.\\r\\n\\r\\nTo minimize the usage of electricity for illumination and fossil fuels for building materials, we are testing the feasibility of optogenetics to provide an eco-friendly and sustainable alternative.  We are engineering bacterial cells that i) control their bioluminescence in response to day/night cycles and ii) can be directed by light to form physical structures out of biodegradable plastic.\\r\\n\\r\\nAdditionally, to address the scalability and speed issues in tissue engineering, we are developing an optical guidance system for inter-cell adhesion and directed gene expression of mammalian cells for tissue generation. This will enable fast in situ cell adhesion for tissue-specific organ healing.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2341': {'team_id': '2341',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2341',\n",
       "  'team_name': 'UCLouvain',\n",
       "  'wiki': 'http://2017.igem.org/Team:UCLouvain',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Belgium',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'BactaSun : Detecting UV with bacteria\\n',\n",
       "  'abstract': 'Nowadays, exposition to harmful UV rays has led to an ever growing number of skin cancer cases, amongst other sun-related diseases. Our aim is to design a biobadge detecting excessive UV exposure and therefore warn us to seek sun protection should it become necessary.\\r\\n\\r\\nThis badge would work as a capsule holding E. coli cells, changing colours as the UV intensity increases. Therefore, we investigated two approaches using photocaged tyrosine (o-nitrobenzyl tyrosine). In both cases, UV-rays will release the tyrosine and enhance a reporter signal. (1) Starting with a tyrosine auxotroph E. coli strain, a reporter RFP will be synthetized once tyrosine is liberated from its cage. (2) Using a photocaged peptide and a specific transcription factor called ComR, we also aim at UV-controlling the expression of the reporter gene.\\r\\n\\r\\nThe capsule would also work as a safe and reliable containment, destroying the engineered microorganisms once the biobadge is discarded. \\r\\n\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2386': {'team_id': '2386',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2386',\n",
       "  'team_name': 'UConn',\n",
       "  'wiki': 'http://2017.igem.org/Team:UConn',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Energy',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'An Algaeneious Approach to Continuous Cultures for Biofuel Production\\n',\n",
       "  'abstract': 'Biofuels are a promising, nearly carbon-neutral alternative fuel source, often derived from algal lipid production. Previous methods of fuel harvest have relied on destructive means of extraction, but we aim to upregulate the excretion of lipids, allowing for potential harvest by physical separation. Our goal is to enhance the algal lipid production and extracellular transport in Nannochloropsis oceanica, a well characterized species with high lipid content. This will be achieved by upregulating the endogenous lipid production enzymes of the cell with high expression promoters and transfecting algae with an ATP binding cassette transporter from Arabdopsis thaliana. Next steps will include developing a system to physically separate excreted lipid from the algal biomass, while maintaining a productive continuous culture.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2455': {'team_id': '2455',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2455',\n",
       "  'team_name': 'UCopenhagen',\n",
       "  'wiki': 'http://2017.igem.org/Team:UCopenhagen',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Denmark',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Incell: a platform for synthetic endosymbiosis\\n',\n",
       "  'abstract': 'Incell is a new synthetic biology platform with near future applications in research, industry and services. We are rewriting nature\\x92s code for endosymbiosis and transforming an evolutionary phenomenon into a technology compatible with standard biological parts.\\r\\nOur vision is to produce synthetic host\\x96endosymbiont systems. We set out with a trinity of experiments intrinsic to the synthetic reconstruction of endosymbiosis. First, creating and sustaining dependence between a host and its endosymbionts by fulfilling the amino acid requirement of an auxotrophic host. Next, to build a modular system of cell-penetrating peptides for protein transport of host nuclear encoded proteins into an endosymbiont, recapitulating a crucial feature of the natural process. Finally, regulating the number of endosymbionts within a host using a CRISPR-Cas system for control of replication.\\r\\nFurther ahead we see a safe, customisable, sustainable technology providing biological solutions to present and future challenges in biotechnology, agriculture and medicine.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2223': {'team_id': '2223',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2223',\n",
       "  'team_name': 'UCSC',\n",
       "  'wiki': 'http://2017.igem.org/Team:UCSC',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Bugs Without Borders\\n',\n",
       "  'abstract': 'Much of the world struggles with inadequate access to essential medicines and nutrition due to high pharmaceutical prices and unreliable distribution. Our solution is to decentralize production of these resources by engineering the robust cyanobacterium, Arthrospira platensis, to produce essential medicines and supplements self-sustainably, photosynthetically, and on-site at healthcare facilities. However, due to insufficient research into the genetics of A. platensis, we have undertaken two separate engineering endeavors in the metabolically similar Synechococcus elongatus PCC 7942 to produce acetaminophen and human-usable vitamin B12. The genes for acetaminophen production, 4ABH and nhoA, and those for B12 production, ssuE and bluB, have been transformed into PCC 7942 with the aim of method validation and subsequent migration to A. platensis, once a genetic system has been established.  In addition, we are performing a whole genome sequencing of A. platensis UTEX 2340 to further this research and its potential medical applications.\\n',\n",
       "  'award': 'Nominated for Best Manufacturing Project',\n",
       "  'medal': 'bronze'},\n",
       " '2212': {'team_id': '2212',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2212',\n",
       "  'team_name': 'UC_San_Diego',\n",
       "  'wiki': 'http://2017.igem.org/Team:UC_San_Diego',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'SynEco: A Xenobiotics-derived Co-culture System of S. elongatus and E. coli for Applications in Bioproduction\\n',\n",
       "  'abstract': 'Currently, the biofuel industry uses glucose as feedstock for E. coli. Methods to prevent contamination require expensive process sterilization or excess antibiotics dosage which become ineffective over time by promoting antibiotic resistance. In our project, we use a xenobiotic approach to engineer autotrophic cyanobacteria, Synechococcus elongatus PCC 7942, to produce the rare sugar D-tagatose in a five-step enzymatic pathway. After detecting tagatose via HPLC, we will seek to engineer a transporter for tagatose secretion, a mechanism that is currently not known. The cyanobacteria will be harnessed in conjunction with genetically engineered E. coli that can metabolize tagatose; by using a multifaceted approach, we could leverage xenobiotic technology in a unique way to create a novel production platform that uses tagatose as a nutrient source and an anti-contamination agent. Because our system utilizes photosynthetic cyanobacteria to cheaply produce the rare sugar carbon source, our co-culture system is both self-sustainable and cost-effective.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2286': {'team_id': '2286',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2286',\n",
       "  'team_name': 'UESTC-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:UESTC-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': ' Tobacco. Degradation. TCP.\\n',\n",
       "  'abstract': \"1,2,3-Trichloropropane (TCP) is a new organic pollutant which has been introduced into our environment as a consequence of industrial waste disposal and widespread open use in agriculture.It's reported that TCP is intended to be pathogenic,carcinogenic and quite persistent in environment.To solve the pollution of TCP,the predecessors have tried to carry out TCP degradation by using E.coli and immobilized enzymes.Although its degradation efficiency is pretty high,long-term sustained environmental remediation outcomes couldn't be achieved.So as to solving this problem,this year we choose tobacco to design a phytoremediation system for TCP. Three kind of enzymes,DhaA in Rhodococcus sp, HheC and EchA derived from Agrobacterium radiobacter AD1,are transformed into tobacco to degrade the TCP into non-toxic and harmless glycerin in the soil and sewage,and to achieve the sustained environmental remediation.\\n\",\n",
       "  'award': 'Best Plant Synthetic Biology;Nominated for Best Plant Synthetic Biology',\n",
       "  'medal': 'gold'},\n",
       " '2352': {'team_id': '2352',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2352',\n",
       "  'team_name': 'UFlorida',\n",
       "  'wiki': 'http://2017.igem.org/Team:UFlorida',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Tryptophol Synthesis for Mitigating Amphibian Chytridiomycosis\\n',\n",
       "  'abstract': 'Amphibian populations worldwide are threatened by the disease Chytridiomycosis, which is caused by the aquatic fungal pathogen Batrachochytrium dendrobatidis (Bd). Chytridiomycosis often leads to death; however, a few species of amphibians are resistant to the Chytridiomycosis infection. The resistance is conferred by symbiotic bacteria present on their skin that produce antifungal metabolites. The UFlorida iGEM team seeks to develop a new treatment for Chytridiomycosis based on these antifungal metabolites. One such antifungal compound, tryptophol, has been shown to be effective at combating Bd at low doses. Tryptophan is converted to tryptophol by three enzymes, Aromatic Amino Acid Aminotransferase II, 2-oxo Acid Decarboxylase, and Alcohol Dehydrogenase I. Our team modified E.coli with the genes comprising the pathway for tryptophol synthesis. The UFlorida team expects to demonstrate that the tryptophol from the genetically modified E. coli will show antifungal activity against the Chytridiomycota family as an antifungal treatment for amphibians. \\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2392': {'team_id': '2392',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2392',\n",
       "  'team_name': 'UGA-Georgia',\n",
       "  'wiki': 'http://2017.igem.org/Team:UGA-Georgia',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Development of an Aptameric Biosensor for Aflatoxin B1 in Peanuts\\n',\n",
       "  'abstract': 'The University of Georgia\\x92s 2017 iGEM team aims to engineer a biosensor for the detection of aflatoxin B1, a harmful mycotoxin peanut contaminant. As Georgia produces more peanuts than any other state in the United States, UGA iGEM recognizes that aflatoxin B1 contamination in peanuts is a regionally important issue. While other iGEM teams have developed detection methods for aflatoxin B1, these methods lack high specificity. To achieve a highly-specific biosensor, UGA iGEM has incorporated an aflatoxin B1 aptamer that has previously been used for an in vitro detection assay. By mimicking this system in vivo, UGA iGEM is developing a cost-effective two-component riboswitch aflatoxin biosensor. This technology could be utilized in the manufacturing process to cut costs in recognizing infected peanuts. Moreover, by creating an affordable aptasensor, this will both make peanut crop growth more accessible to developing countries and decrease aflatoxin-related deaths worldwide.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2424': {'team_id': '2424',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2424',\n",
       "  'team_name': 'UiOslo_Norway',\n",
       "  'wiki': 'http://2017.igem.org/Team:UiOslo_Norway',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Norway',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Achieving lasing in a GFP protein solution and yeast cells\\n',\n",
       "  'abstract': 'A bio-laser, which is a laser with a biological gain medium, was first described in 2011. In our project, we want to create and explore such a bio-laser. To achieve this, we will first attempt to use an external light source and mirrors to get stimulated light amplification in a solution of fluorescent proteins, and use that as a functional proof of concept. The second part is to transform S. Pombe yeast cells with said fluorescent protein, and try to achieve lasing in a solution of live yeast cells.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2427': {'team_id': '2427',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2427',\n",
       "  'team_name': 'UIOWA',\n",
       "  'wiki': 'http://2017.igem.org/Team:UIOWA',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Development of a 3-Hydroxypropionic Acid Biosensor\\n',\n",
       "  'abstract': 'Many industrial manufacturing processes utilize the metabolite 3-hydroxypropionic acid (3-HP) as a platform chemical for the synthesis of complex biofuels and plastics. Current companies engineering bacteria to overproduce 3-HP utilize high-performance liquid chromatography to assay concentrations, which is a precise method for established bioreactors but time-consuming for testing experimental strains. Recent studies identified 3-HP responsive genes in Pseudomonas denitrificans and Pseudomonas putida, which could be used as a viable biosensor for real-time monitoring of 3-HP concentrations in vivo. Our ongoing research project utilizes the 3-HP responsive genes found in P. putida and P. denitrificans as biological reporters which express luciferase in the presence of 3-HP. We will then adapt this system to Bacillus subtilis, which has shown potential as a 3-HP producer for industrial processes due to its high tolerance for concentrations of 3-HP that are toxic in other microorganisms.  Our biosensor is a useful tool for metabolic engineering research.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2468': {'team_id': '2468',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2468',\n",
       "  'team_name': 'UIUC_Illinois',\n",
       "  'wiki': 'http://2017.igem.org/Team:UIUC_Illinois',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Sweet Giblets: A Homemade Gibson Assembly Recipe\\n',\n",
       "  'abstract': 'The aim of this project is to lower the cost of a common recombinant DNA technique, the Gibson Assembly. \\r\\n\\tThe concept revolves around the construction and expression of plasmids with DNA polymerase and DNA ligase as the insert genes. The constructed plasmids for these two proteins are transformed into DH5α cells for high cloning and then transformed into BL21(DE3) for high gene expression and production of the proteins of interest. \\r\\nCell lysate from transformed BL21(DE3) cells is used as DNA ligase and DNA polymerase and the base level of exonuclease present naturally in the cell lysate serves as the T5 exonuclease. Using different combinations of the cell lysates for the DNA ligase and DNA polymerase, a plasmid with known sequence and properties is assembled to test the concept.  Sequence results obtained after each trial are recorded as a measure of success or a need to modify the lysate ratios. \\r\\n\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2215': {'team_id': '2215',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2215',\n",
       "  'team_name': 'ULaVerne_Collab',\n",
       "  'wiki': 'http://2017.igem.org/Team:ULaVerne_Collab',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'S.O.S! Save Our Seas!\\n',\n",
       "  'abstract': 'Corals are responsible for sustaining a quarter of the ocean\\x92s biodiversity. However, due to natural biochemical processes and human activities, ocean temperatures are rising by 1-2% every year. This results in coral bleaching, the loss or expulsion of zooxanthellae from its tissue, due to the accumulation of reactive oxygen species (ROS) produced from photosystem I. ROSs can be converted to less toxic chemicals by superoxide dismutases (SODs), however it does not react as quickly with ROSs as does other chemicals in the cell. Our research aims to reduce the production of ROSs by characterizing three SODs. We will determine which SOD is most efficient based on its placement in the cell under abiotic stresses, such as light and heat. We will also create a chassis for Symbiodinium and construct biobricks for each individual part. In addition, a controlled water tank will be created to simulate environmental conditions.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2477': {'team_id': '2477',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2477',\n",
       "  'team_name': 'UMaryland',\n",
       "  'wiki': 'http://2017.igem.org/Team:UMaryland',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'An Apeeling Solution to Panama Disease In Bananas\\n',\n",
       "  'abstract': \"Panama disease, caused by the fungus Fusarium oxysporum, poses a major threat to the world's banana population. Experts confirm that the most common banana cultivar could be wiped out, resulting in major agricultural and economic consequences. Our goal is to combat F. oxysporum with transgenic bacteria that sense fusaric acid, a toxin released by F. oxysporum, and respond by producing and secreting thaumatin like protein (TLP), an antifungal agent found in plants. Transferring the final genetic circuitry to Bacillus amyloliquefaciens, a symbiont associated with the roots of the banana plant, will provide a soil additive capable of protecting banana crops from this threatening disease. This system can be tuned with a Cas9 mutagenesis screening method we developed. Finally, to increase awareness of the role of synthetic biology in protecting food and water, we fabricated an inexpensive \\x93lab-in-a-box\\x94 system suitable for high schools along with a bacterial based metal detection system.\\n\",\n",
       "  'award': 'Nominated for Best Hardware',\n",
       "  'medal': 'bronze'},\n",
       " '2430': {'team_id': '2430',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2430',\n",
       "  'team_name': 'UMBC',\n",
       "  'wiki': 'http://2017.igem.org/Team:UMBC',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment(P)',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': ''},\n",
       " '2236': {'team_id': '2236',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2236',\n",
       "  'team_name': 'UNBC-Canada',\n",
       "  'wiki': 'http://2017.igem.org/Team:UNBC-Canada',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'MRSAway\\n',\n",
       "  'abstract': 'Antibiotic resistance is one of the largest issues facing modern medicine today. Of particular importance is the lethal Methicillin-resistant Staphylococcus aureus, or MRSA. We propose to use sRNA-mediated gene silencing as a substitute for antibiotics to target an array of essential genes in MRSA: mecA, secA, glmM, and ddl. First, we identified a dsRNA-binding chaperone, Hfq, which acts to stabilize the sRNA-mRNA duplex in order to recruit RNase III to degrade the dsRNA. We then designed complimentary sRNAs for each mRNA target and tested the affinity of Hfq to our custom designed (AU)7A binding region on the sRNAs using a fluorescence polarization assay. The number of ribonucleotides cleaved from the binding region of Hfq via RNase III was determined via alkaline hydrolysis and long-format PAGE. Finally, Hfq and sRNA were co-transformed on a single vector to test gene knockdown efficiency.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2499': {'team_id': '2499',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2499',\n",
       "  'team_name': 'UNC-Asheville',\n",
       "  'wiki': 'http://2017.igem.org/Team:UNC-Asheville',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'TCEasy\\n',\n",
       "  'abstract': 'Degradation of Trichloroethylene through novel metabolic pathway using soluble methane monoxygenase and haloacid dehalogenase transformed e.coli.   \\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2461': {'team_id': '2461',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2461',\n",
       "  'team_name': 'UNebraska-Lincoln',\n",
       "  'wiki': 'http://2017.igem.org/Team:UNebraska-Lincoln',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Engineering E. coli to Reduce Methane Emissions in Cattle\\n',\n",
       "  'abstract': \"The excessive production of methane by cattle is harmful to both the environment and the cattle industry. The reduction of nitrate to nitrite by the rumen microbiota has been shown to compete with the methanogenesis process for hydrogen. Nitrate is an effective feed additive, but in large quantities it causes nitrate poisoning. To combat this we have engineered E. coli to express a nitrite reductase protein. When introduced into the cows' gut microbiome, it will make the cows resistant to nitrate poisoning. Seaweed has also been found to effectively reduce methane emissions when used as a cattle feed supplement. Bromoform was found to be the compound in seaweed that inhibits methanogenesis. We studied the possibility of producing bromoform directly in the cows\\x92 gut by engineering E. coli to produce a bromoperoxidase enzyme. In further efforts we hope to replace recurring feed additive purchases with a one-time inoculation of food-grade bacteria.\\n\",\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2343': {'team_id': '2343',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2343',\n",
       "  'team_name': 'UNIFI',\n",
       "  'wiki': 'http://2017.igem.org/Team:UNIFI',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Italy',\n",
       "  'track': 'Art & Design',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'The sound of coli\\n',\n",
       "  'abstract': 'Since \\x9290 biotechnologies have been highly debated at political and social level, strongly influencing public perception and mass media.\\r\\nOur aim is to investigate the relationship between public perception and biotechnologies and managing to speak to the broadest audience we chose the universal language of music, an innovative science art.\\r\\nWe make use of bacterial oscillator related to quorum sensing molecules. In particular the idea is to create three E. coli strains, expressing three fluorescent proteins under an oscillating gene regulation circuit controlled by three molecular inducers. The patterns of the emitted fluorescence by the co-culture will be used as input signal for a colour-to-sound translation software. The end will be a \"son et lumière\" show, which will serve as demonstration of the complexity and beauty of the fascinating biomolecular world. We want to demonstrate that such hidden beauty can be disclosed only by the use of biotechnological tools.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2521': {'team_id': '2521',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2521',\n",
       "  'team_name': 'United-NFLS-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:United-NFLS-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Withdrawn',\n",
       "  'section': 'This team has withdrawn. It will appear in normal team listings.\\n',\n",
       "  'title': '',\n",
       "  'abstract': ''},\n",
       " '2284': {'team_id': '2284',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2284',\n",
       "  'team_name': 'UNOTT',\n",
       "  'wiki': 'http://2017.igem.org/Team:UNOTT',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Key. coli: next generation security - protect your gems with germs!\\n',\n",
       "  'abstract': 'Numerous critical issues have begun to emerge affecting digital password security. Companies are increasingly turning to physical strategies, involving biometric and digital keys, to secure client accounts. Synthetic biology offers significant unprecedented security opportunities through synthetically generated biometrics. Consequently, we have developed a randomly assorting, fluorescent bacterial key. Separate genes encoding three distinct fluorescent proteins are expressed using different promoters that are capable of inhibition by dCas9. Null and functional sgRNA DNA modules were endowed with identical sticky ends to compete with one another during ligation, generating an ON/OFF fluorescent state and variance for distinguishable keys. Each combination is modelled from lab data, illustrating discernibility. We have designed a safe, portable device for storage of E. coli, paired to a streamlined authentication procedure tailored to be immune to current hacking frameworks. This system is scalable to include any type of protein, synonymously expanding the number of combinations and improving security.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2327': {'team_id': '2327',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2327',\n",
       "  'team_name': 'uOttawa',\n",
       "  'wiki': 'http://2017.igem.org/Team:uOttawa',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Exploring gRNA-modulated genetic networks\\n',\n",
       "  'abstract': 'Despite the fundamental role transcription factors play, they place many constraints on experimental designs due to limited types available. Synthetic guide RNA can be modulated to match existing promoters, allowing for targeted regulation without the need for DNA manipulation. Inspired by Gander\\x92s study, \\x93Digital logic circuits in yeast with CRISPR-dCas9 NOR gates\\x94, we have decided to build a NOR gate, whose output is controlled via inducible production of gRNA-complexed dCas9. First, we will create four different strains of yeast, each representing a stationary state. The system has two different gRNA; the expression of either results in the inhibition or transcription of yeGFP. Finally, we will create another strain containing a NOR gate, whose different output states can be modified by controlling the production of both gRNAs using different inducible promoters. Thus, our project verifies regulated recruitment using synthetic guide RNA as a viable alternative to conventional transcriptional regulatory modules.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2504': {'team_id': '2504',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2504',\n",
       "  'team_name': 'UPMC_PARIS',\n",
       "  'wiki': 'http://2017.igem.org/Team:UPMC_PARIS',\n",
       "  'region': 'Europe',\n",
       "  'country': 'France',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'The BioMaker Factory, Synthetic biology for access to health care\\n',\n",
       "  'abstract': 'Realizing that the lack of access to health care is a major problem in developing countries, we decided to create an automated user-friendly mobile factory able to produce therapeutic molecules for multiple diseases. A heterologous recombinant protein expression system will make E. coli bacteria produce antigen-binding (Fab) fragments of antibodies under an optogenetic regulation thus the production. Our smart box will automatically manage the expression of active biological substances as well as their purification thanks to a computer that will control every step through a software specially designed for our box.\\r\\n\\r\\n\\x93The BioMaker Factory\\x94 will be available to humanitarian organizations and local facilities to tackle the lack of access to health care in areas in need. \\r\\n\\r\\nThanks to the BioBricks system and the work currently done by researchers and industries to develop bioproduction, we are now confident that \\x93The BioMaker Factory\\x94 will offer a wide range of applications.\\r\\n\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2423': {'team_id': '2423',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2423',\n",
       "  'team_name': 'Uppsala',\n",
       "  'wiki': 'http://2017.igem.org/Team:Uppsala',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Sweden',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Crafting Crocin\\n',\n",
       "  'abstract': \"Crocin is part of a beta-carotene pathway that leads to synthesis of saffron, which gives the compound it\\x92s beautiful crimson colour. In recent years Crocin has caught the attention of researchers worldwide as a potential treatment for various degenerative diseases such as cancer and Alzheimer\\x92s. Besides treating diseases we also see potential of Crocin being used as a dye in different industries. The Zeaxanthin pathway is in the registry as well the beta-carotene pathway, but due to that the Zeaxanthin plasmid is quite large and unstable we decided to use lambda red recombineering to transfer it from the plasmid to the chromosome to make it more stable. When producing a compound synthetically and potentially cheaper than the actual plant, it is important to take the ethical aspects into account. That's why ethics has been a huge part of our project and with our Ethics manual hope to inspire more teams. \\n\",\n",
       "  'award': 'Best Manufacturing Project, Overgrad;Nominated for Best Manufacturing Project, Overgrad',\n",
       "  'medal': 'gold'},\n",
       " '2397': {'team_id': '2397',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2397',\n",
       "  'team_name': 'UrbanTundra_Edmonton',\n",
       "  'wiki': 'http://2017.igem.org/Team:UrbanTundra_Edmonton',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Cleaning an Interplanetary Toxin: Converting Perchlorate into a Breathable Alternative\\n',\n",
       "  'abstract': ' Perchlorate is a natural toxin that exists on Mars at high concentrations of ~0.5-1% and on Earth at significant concentrations of ~0.15-0.25%. However, perchlorate is also a by-product contaminant of several industrial processes. Certain soil bacteria degrade perchlorate to oxygen and chloride using the enzymes perchlorate reductase and chlorite dismutase. By exploiting this two-step enzyme pathway, our team hopes to bioremediate Martian soil for future human settlement and potential applications on Earth.                                                                                             \\r\\n                                                                                                   \\r\\n Last year, our high school team UrbanTundra 2016 successfully expressed chlorite dismutase in E. coli, and showed that it could convert the chlorite intermediate to chloride and oxygen. This year, our plans are two-fold: 1) to complete the pathway by expressing perchlorate reductase, a challenging problem involving a large two-subunit membrane protein requiring multiple cofactors, and 2) the design of a self-contained bioreactor for the enzyme pathway that can be used on both planets.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2222': {'team_id': '2222',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2222',\n",
       "  'team_name': 'USMA-West_Point',\n",
       "  'wiki': 'http://2017.igem.org/Team:USMA-West_Point',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Detecting chemicals with engineered olfactory receptors through microelectrode array readings\\n',\n",
       "  'abstract': 'Current state-of-the-art inorganic hardware sensors for biological and chemical agent detection are highly tailored for specific chemicals and find difficulty when used to detect compounds outside of a highly defined analyte set. Olfactory receptors are G Protein Coupled Receptors (GPCRs) that discriminate thousands of odorants based on genetic sequences that in the presence of a ligand cause cells to generate an electric potential that is measurable using microelectrode arrays (MEAs). Here, we modify HT-22 cells by adding individual olfactory receptors plasmids via nucleofection. A bioreactor was designed with a peristaltic pump system allowing for media to flow across a MEA cultured with neurons which enables the controlled addition of liquid samples for action potential measurement. Analysis of modified neurons serve as a representative model for exploiting the sensitivity and selectivity of native olfactory systems to be used as rapid detection systems for applications in security and medical & health capacities.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2199': {'team_id': '2199',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2199',\n",
       "  'team_name': 'USNA_Annapolis',\n",
       "  'wiki': 'http://2017.igem.org/Team:USNA_Annapolis',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Editing the Human Microbiome: Preventing Aerosolized Peptide Proteins\\n',\n",
       "  'abstract': 'Intercellular communication in biofilm microbial communities is a well-known natural phenomenon. Recently, it has been reported that cells in biofilms may also communicate long distances via ion gradients, mimicking neuronal networks. We believe such electrical communication could be important in pathogenicity of air-borne environmental toxicants if they interfere with natural ion fluxes in the human respiratory microbiome. If a microorganism were engineered to respond to an ionic change, the respiratory microbiome could detect and respond to an environmental toxicant exposure in real-time. Our project will demonstrate a cation responsive genetic sensor by regulating expression of GFP using the sodium responsive transcriptional regulator, NhaR. We will use the ionophore monensin, known to create a change in Na+ equilibrium across the cell membrane, as a proxy to imitate an ionic response within a biofilm. This will ultimately provide bioengineers a method to create alternate probiotic sensors for cell-communication and possibly ion homeostasis. \\r\\n\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2486': {'team_id': '2486',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2486',\n",
       "  'team_name': 'USP-Brazil',\n",
       "  'wiki': 'http://2017.igem.org/Team:USP-Brazil',\n",
       "  'region': 'Latin America',\n",
       "  'country': 'Brazil',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': ' BioTrojan: Combining paratransgenesis and Synthetic Biology approaches to combat mosquito-borne diseases\\n',\n",
       "  'abstract': 'Paratransgenesis can be defined as a set of strategies to eliminate a pathogen from vector populations through the usage of genetically modified symbionts, thus controlling vector-borne diseases. For our iGEM Project, we have focused on generating a versatile molecular toolkit for endogenous detection and elimination of mosquito-borne pathogens. We have selected Pantoea agglomerans, a ubiquitous bacterium which is enriched in the midgut microbiota of anophelines, as a novel chassis for targeting malarial parasites. Two coupled genetic circuits have been designed (i) for sensing malarial infection biomarkers in the blood ingested by mosquitoes and (ii) for subsequent production/secretion of anti-Plasmodium synthetic peptides. A third module, consisting in an endogenous bacterial killer-switch was designed to control the population dynamics of the genetically engineered symbiont. This is the first study combining the conceptual frameworks of both paratransgenesis and Synthetic Biology, bearing great potential for the generation of novel approaches for combating mosquito-borne diseases.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2242': {'team_id': '2242',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2242',\n",
       "  'team_name': 'USTC',\n",
       "  'wiki': 'http://2017.igem.org/Team:USTC',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'PELICAN\\n',\n",
       "  'abstract': 'Bio-manufacturing is a type of manufacturing that utilizes biological systems to produce commercially important biomaterials. However, it can\\x92t be scaled up and put into practice so easily mainly due to the unstable productivity. So, to make bio-manufacturing a more practical technology for synthesis, we build up a more stable and efficient synthesis platform based on bio-cathode. There are 3 systems in our project. To enable E.coli to transfer the extracellular electrons into the cytoplasm as NADH, we construct the Mtr CAB system, the first system we have. Mtr CAB, a protein complex located on the outer membrane, can transfer electrons from the outside of the membrane into the periplasm. The second system is a photoelectric system, generating high energy electrons with CdS nanoparticles as a light harvester. Lastly, we introduce an enzyme as an indicator to demonstrate our project\\x92s flexibility and efficiency.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2241': {'team_id': '2241',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2241',\n",
       "  'team_name': 'USTC-Software',\n",
       "  'wiki': 'http://2017.igem.org/Team:USTC-Software',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Software',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Biohub 2.0\\n',\n",
       "  'abstract': \"Biohub 2.0 is a synthetic biological platform devoting for more efficient ideas sharing and colliding. With a more friendly and more convenient web-based interface, users can browse the parts interested them and rate the parts impressed them easily. If inspiration or questions popped out during studying, one can leave a comment under the specific part and communicate with other users. Experiment experience can also be published for later referencing. More than a community, the platform is also a well-designed plugin system, allowing splendid field-related algorithms to be integrated into it. Currently it carries BioBrick Manager, Biobless and ABACUS as default plugins, but users can develop and upload their own ones freely with the help of Biohub's documentation.\\n\",\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2519': {'team_id': '2519',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2519',\n",
       "  'team_name': 'UST_Beijing',\n",
       "  'wiki': 'http://2017.igem.org/Team:UST_Beijing',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Cyclase of Nature & Pangu Algorithm\\n',\n",
       "  'abstract': 'Out of 500 some existing amino acids nature only picks 20 as major building blocks to make proteins. To analyze these 20 molecules, a Matlab computer algorithm is scripted to measure physical-chemical distance among them, which is then summed up to score homologous proteins. We called the program Pangu, a Chinese legendary figure, to reflect the shared-common ancestor of life on earth. We hope programs like Pangu would supplement standard methods of analyzing phylogenetic relationships between proteins and species.\\r\\n\\r\\nGuided by Pangu algorithm, we will synthesize an artificial squalene cyclase in cultured human cells to study basic physiology of cholesterol and related analogs.  We hope the result will help to develop new methods to identify micro-nutrients importance for human health.   \\r\\n\\r\\nLast but not least, we will continue our 2016 iGEM program iGUT, an E.coli-expressed beta-glucosidase for notoginseng processing to improve bio-availability.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2522': {'team_id': '2522',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2522',\n",
       "  'team_name': 'US_AFRL_CarrollHS',\n",
       "  'wiki': 'http://2017.igem.org/Team:US_AFRL_CarrollHS',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Engineered Microbes to Sense and Respond to ETEC\\n',\n",
       "  'abstract': 'Every year, Enterotoxigenic Escherichia coli (ETEC), the most common form of traveler\\x92s diarrhea, affects thousands of deployed warfighters. The goal is to engineer non-pathogenic E. coli to sense ETEC, respond to its presence, and package it in a cellulose matrix to enable environmental detection of ETEC. We created two plasmids: \\x91sense-respond\\x92; and \\x91packaging\\x92.  The sense-respond plasmid sensed Auto-Inducer 2 (AI-2), a quorum sensing molecule created by most ETEC strains, by expressing LsrR which switches on the Lsr promoter.  Activation of the Lsr promoter expresses Super-Folder Green Fluorescent Protein (sfGFP), indicating the presence of ETEC. The packaging plasmid expresses  a fusion protein consisting of curli fibers and cellulose binding domains. These modified surface proteins permit the bacteria to bind to cellulose, encapsulating the sense-response module.  We envision this genetically engineered machine to be deployed in both the internal and external environment to detect ETEC. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2451': {'team_id': '2451',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2451',\n",
       "  'team_name': 'UT-Knoxville',\n",
       "  'wiki': 'http://2017.igem.org/Team:UT-Knoxville',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Expanding Our Aromatic Waste Degradation Platform \\n',\n",
       "  'abstract': 'Crude oil processing produces toxic byproducts, such as\\xa0benzene, toluene, and xylenes (BTX) capable of contaminating groundwater and soil\\xa0leading to potentially\\xa0serious health risks for\\xa0both humans and wildlife. Thus, the effective removal of these hazardous organic compounds is imperative\\xa0towards protecting natural resources.\\xa0While it is possible to manually clean contaminated sites, such efforts are costly and time consuming.\\xa0Our project\\xa0seeks to degrade the\\xa0toluene-based\\xa0contaminants into valuable aromatic aldehydes by expressing\\xa0the\\xa0xyl\\xa0ortho\\xa0pathway from\\xa0Pseudomonas putida\\xa0in\\xa0E. coli.\\xa0As a continuation of our 2016\\xa0project,\\xa0the 2017 UT-Knoxville\\xa0iGEM\\xa0team\\xa0aims to\\xa0use synthetic biology and metabolic engineering techniques to further develop our bioremediation strain. First, regulatory elements sensitive to the presence of aromatic hydrocarbons are engineered to fine-tune protein production. Second, overexpression of broadly specific efflux pumps intend to increase organic solvent tolerance and alleviate toxic effects. And third, exploring enzymatic homologs can expand our product library. Our project aims to increase production titers and develop a more robust microorganism suitable for manufacturing and bioremediation needs. \\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2351': {'team_id': '2351',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2351',\n",
       "  'team_name': 'Utrecht',\n",
       "  'wiki': 'http://2017.igem.org/Team:Utrecht',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Netherlands',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'OUTCASST: Out-of-cell CRISPR Activated Sequence-specific Signal Transducer\\n',\n",
       "  'abstract': 'We aim to create a DNA detection kit for diagnosis of diseases caused by microorganisms, such as Chagas disease. Current test-kits for Chagas disease lack in specificity or sensitivity and require trained personnel and a well-equipped laboratory, which are often not available in rural areas. Our system, called OUTCASST, detects pathogen DNA, allowing for direct diagnosis in a simple, robust and inexpensive manner that can be used in absence of laboratory equipment. It works through DNA binding and colocalization of catalytically inactive Cas9 and Cpf1 fused to membrane proteins and exposed to the extracellular medium. To assemble OUTCASST, we generated inactive Cas9 and Cpf1 variants and checked for functionality. In parallel, several modelling techniques were employed to identify venues for optimization. OUTCASST can be adapted easily to detect any desired DNA sequence and is therefore a valuable addition to the pool of toolkits for disease diagnosis. \\n',\n",
       "  'award': 'Nominated for Best Integrated Human Practices',\n",
       "  'medal': 'gold'},\n",
       " '2490': {'team_id': '2490',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2490',\n",
       "  'team_name': 'U_of_Guelph',\n",
       "  'wiki': 'http://2017.igem.org/Team:U_of_Guelph',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'One OXCellent FRC\\x92n Project\\n',\n",
       "  'abstract': 'The University of Guelph\\x92s project focuses on the development of an enzyme-based cleaning method for the removal of calcium oxalate scale (commonly called beerstone) from the inside of beer brewing vats. Beerstone is incredibly insoluble and difficult to remove, with common cleaning methods involving the use of caustic acids. Our team hopes to use the enzymes Formyl-CoA Transferase (FRC) and Oxalyl-CoA decarboxylase (OXC) from Oxalobacter formigenes\\x92s oxalate degrading metabolism to break down beerstone. This year\\x92s aspect of the project focused on cloning frc and oxc into E.coli DH5a using pET-28a. This required conducting site directed mutagenesis to add the PstI cut site to the pET-28a vector. Future experimentation will include cloning into BL21, expressing and characterizing FRC and OXC, and the development of an enzyme-based cleaning solution. \\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2269': {'team_id': '2269',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2269',\n",
       "  'team_name': 'Valencia_UPV',\n",
       "  'wiki': 'http://2017.igem.org/Team:Valencia_UPV',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Spain',\n",
       "  'track': 'Food & Nutrition',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'ChatterPlant\\n',\n",
       "  'abstract': 'Urban overpopulation, climate change and natural resources decrement are threatening food security. Ensuring season-less, accessible and local food production promotes a sustainable agriculture. Valencia_UPV provides a whole new system to control plant physiology at both genetic and environmental level.\\r\\n\\r\\nChatterPlant is a SynBio-based solution that works as plant-human interface allowing a bidirectional communication. First, a root-specific modular optogenetic circuit enables control on plants´ endogenous gene expression (e.g flowering). Then, a sensor circuit with color coded output provides specific information of stress conditions, accelerating corrective measures. The genetic setup is complemented with a hardware device, ChatterBox, specially designed to control plant\\x92s growth conditions.\\r\\n \\r\\nChatterPlant\\x92s possibilities can be improved gathering Plant SynBio knowledge. PlantLabCo is an open-access online platform which aims to unify Plant SynBio researchers\\x92 work. Individual results can be published, supported by a modeling software tool integrated to ease the mathematical models\\x92 generation of genetic circuits.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Poster;Nominated for Best Applied Design;Nominated for Best Software Tool',\n",
       "  'medal': 'gold'},\n",
       " '2259': {'team_id': '2259',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2259',\n",
       "  'team_name': 'Vilnius-Lithuania',\n",
       "  'wiki': 'http://2017.igem.org/Team:Vilnius-Lithuania',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Lithuania',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'SynORI \\x96 a framework for multi-plasmid systems\\n',\n",
       "  'abstract': 'Gene copy number serves as a fundamental parameter in the dynamics of synthetic gene circuits, but is often not explicitly considered. Coupled with transcriptional and translational regulation, copy number control would offer an effective coordination and increased dynamic range of multiple gene expression. Therefore, we modified the ColE1 replicon to develop a synthetic origin of replication - SynORI - which enables the alteration of plasmid copy number. SynORI framework also incorporates a multi-plasmid regulation system based on uniquely barcoded regulatory RNA molecules, allowing to co-maintain different-group plasmids at preselected copy numbers in a standardized manner. In case certain plasmids are chosen to have a low copy number, an active partitioning system will minimize the risk of plasmid loss and increase the stability of our system. SynORI enables the creation of more complex metabolic pathways, smart assembly of protein complexes and a more precise information processing in synthetic biology.\\n',\n",
       "  'award': 'Nominated for Best Foundational Advance Project;Nominated for Best Wiki;Nominated for Best Presentation;Nominated for Best Poster;Nominated for Best Education & Public Engagement;Best New Basic Part;Nominated for Best New Basic Part;Best New Composite Part;Nominated for Best New Composite Part;Best Part Collection;Nominated for Best Part Collection;Finalist;Nominated for Finalist',\n",
       "  'medal': 'gold'},\n",
       " '2272': {'team_id': '2272',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2272',\n",
       "  'team_name': 'Virginia',\n",
       "  'wiki': 'http://2017.igem.org/Team:Virginia',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Sewage, PD\\n',\n",
       "  'abstract': 'Current wastewater treatment methods are complex and often difficult to maintain. During the biological nutrient removal process, sludge composed of co-cultures of nitrifying and denitrifying bacteria converts ammonia and nitrites into inert nitrogen gas. Proper treatment of wastewater is important because the release of ammonia and nitrites poses health risks to humans. These toxic chemicals also fuel detrimental water eutrophication. Unfortunately, reaching optimal efficiency of the predominant nitrifier, Nitrosomonas europaea, requires aeration, which is costly for treatment facilities. Here we present a biological device that reduces aeration requirements and eliminates the need for co-cultures. We use a denitrifying bacterium Paracoccus denitrificans as a chassis for a device that contains a nitrification circuit taken from the genome of N. europaea. The addition of amoCAB, haoA, and the associated cytochrome genes creates a complete nitrogen removal system. Upon implementation, our device reduces the operating costs of wastewater treatment plants. \\r\\n\\n',\n",
       "  'award': 'Nominated for Best Environment Project;Nominated for Best New Basic Part',\n",
       "  'medal': 'gold'},\n",
       " '2387': {'team_id': '2387',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2387',\n",
       "  'team_name': 'Wageningen_UR',\n",
       "  'wiki': 'http://2017.igem.org/Team:Wageningen_UR',\n",
       "  'region': 'Europe',\n",
       "  'country': 'Netherlands',\n",
       "  'track': 'Diagnostics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Mantis: Modular antigen-based test for infectious diseases\\n',\n",
       "  'abstract': 'Over the last two decades epidemics of infectious diseases have caused major harm to the world population. These outbreaks often originate from developing countries, where diagnosis can be problematic. The Mantis project aims to detect antigens in blood samples of patients using an in vivo bacterial system, allowing for efficient diagnosis even in rural areas. Mantis is fast, robust, modular, and requires little equipment. Moreover, the whole-cell system allows for more affordable production and longer storage compared to current systems. Detection is based on affinity bodies, an antibody mimetic that can be produced by bacteria. Through rational design, the system can be adapted to detect a wide range of pathogens. The addition of the affinity body to a bacterial receptor will rapidly generate a clear fluorescent signal, measured by our portable, 3D-printed diagnostic device directly in the field. This way, Mantis will help diseases come to light.\\n',\n",
       "  'award': 'Best Wiki, Overgrad;Nominated for Best Wiki, Overgrad;Nominated for Best Presentation, Overgrad;Nominated for Best Integrated Human Practices, Overgrad;Nominated for Best Model, Overgrad;Nominated for Best Applied Design, Overgrad;Nominated for Best Hardware, Overgrad;Nominated for Best New Composite Part, Overgrad;iGEMers Prize',\n",
       "  'medal': 'gold'},\n",
       " '2463': {'team_id': '2463',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2463',\n",
       "  'team_name': 'Warwick',\n",
       "  'wiki': 'http://2017.igem.org/Team:Warwick',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Application of optogenetic control mechanisms for manipulating biopolymer synthesis in advancing 3D printing technologies\\n',\n",
       "  'abstract': 'By providing a well-defined, biocompatible surface coating, the risk of bone and dental implant failure will be greatly reduced. We aim to accomplish this by controlling the spatial production of extra-cellular cellulose with light. Our modified E.coli builds on previous teams work, utilising a transmembrane protein complex, which upon exposure to red light, phosphorylates a promoter and initiates the synthesis cascade. Using this technology, our team will be able to build a 3D printer where living bacteria act as the bioink. Our team will then be able to produce cellulose structures, featuring micrometer pores, which mimic the surface of broken bone, for implants. This structure has been shown to promote osseointegration, helping to reduce overall failure rates. \\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2391': {'team_id': '2391',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2391',\n",
       "  'team_name': 'Washington',\n",
       "  'wiki': 'http://2017.igem.org/Team:Washington',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Contact email: uwigem [a] uw [d] edu\\n',\n",
       "  'title': 'Viva Violacein - An Affordable Real-Time Metabolics Tracker\\n',\n",
       "  'abstract': 'Advances in synthetic biology have lead to new, useful metabolic pathways that can be used to produce metabolites on a commercially-viable scale. However, management of these cultures is time-consuming and labor-intensive, and measuring levels of metabolites often involves prohibitively expensive analytics, such as HPLC. Our project aims to overcome these problems by providing a low-cost, automated turbidostat bioreactor that analyzes a yeast culture in real time and corrects inputs to maintain culture conditions.\\r\\nTo test our system, we use the violacein metabolic pathway, regulated with inducible promoters, to yield four visually distinct pigments. An open-source Raspberry Pi computer collects color and opacity information about the culture, and software analyzes the color and gradually introduces the inducers to keep the culture\\x92s production stable. \\r\\nBy combining biological, software, and hardware systems, our design can generate previously-unavailable visual data in certain biosynthesis processes, and has a wide range of applications in industrial fermentation.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2195': {'team_id': '2195',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2195',\n",
       "  'team_name': 'WashU_StLouis',\n",
       "  'wiki': 'http://2017.igem.org/Team:WashU_StLouis',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Operation: Ultraviolet\\n',\n",
       "  'abstract': 'Due to numerous climate effects, photosynthetic organisms are being damaged due to increased levels of harmful UV-B radiation. Luckily, many organisms exist that have a natural resistance to this kind of radiation. Using genes from the tardigrade species Ramazzottius Varieornatus, the bacteria species Deinococcus Radiodurans, and a strain of cyanobacteria, we hope to induce resistance to UV-B radiation damage. In order to demonstrate proof of these genes\\x92 utility, our first step was to transform the genes into E. Coli. From there, we planned to test these transformed cells in our Environmental Simulation System by irradiating the cells and comparing their growth to that of a control. So far, we have found that E. Coli cells with Dsup are substantially more resistant to UV-B radiation, and we have also recorded preliminary data on uvsE as well. Currently, we are transforming the four genes into cyanobacteria and testing them under UV-B light.\\n',\n",
       "  'award': 'Nominated for Best Environment Project;Nominated for Best Hardware',\n",
       "  'medal': 'gold'},\n",
       " '2475': {'team_id': '2475',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2475',\n",
       "  'team_name': 'Waterloo',\n",
       "  'wiki': 'http://2017.igem.org/Team:Waterloo',\n",
       "  'region': 'North America',\n",
       "  'country': 'Canada',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'Prions be Lit: Functional Amyloid as a Biological Tool\\n',\n",
       "  'abstract': 'Prions are perhaps most famous for their implication in neurodegenerative diseases. However, there are also proteins that bear strong similarities to prions while not being associated with an infectious disease. These proteins have been deemed \\x93functional prions.\\x94 Here, engineered functional prions in Saccharomyces cerevisiae provide a proof of concept for using prions as a tool in synthetic biology to co-localize different proteins. These engineered proteins have prion-like aggregative behaviour, as well as a fluorescent tag. They will be used to test the viability of using engineered prions to bring and keep proteins in close proximity to each other while maintaining their function.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2420': {'team_id': '2420',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2420',\n",
       "  'team_name': 'Westminster_UK',\n",
       "  'wiki': 'http://2017.igem.org/Team:Westminster_UK',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'The detection of quorum-sensing signalling molecules from Pseudonomas Putida via a novel biosensor\\n',\n",
       "  'abstract': 'AHL is a quorum-sensing molecule secreted by bacteria and is essential in biofilm formation. Biofilm is a collection of extracellular proteins and DNA which decreases sensitivity of bacteria to antibiotics. Antibiotic resistance is emerging as a world crisis with the NHS spending roughly £1 billion just on treating nosocomial infections. Multidrug resistant (MDR) gram negative bacteria such as Pseudomonas in particular are emerging as increasingly problematic bacterial species, especially in secondary infections. Therefore, we are aiming to develop strategies to help counter biofilm formation from the molecular level using synthetic biology, with the specific genes ppuR, ppuI and RsaL in Pseudomonas Putida being involved in the formation of the quorum sensing molecule AHL. A Biosensor can be developed to identify and decrease production of AHL, causing decreased biofilm formation and an increased sensitivity of the bacteria to antibiotics. Subsequent applications in pharmacology and well as the development of biocontainment devices.\\n',\n",
       "  'award': '',\n",
       "  'medal': ''},\n",
       " '2462': {'team_id': '2462',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2462',\n",
       "  'team_name': 'WHU-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:WHU-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Wow(WHU-iGEM operate wastewater)\\x97\\x97biodegradation of halogenated phenol in wastewater\\n',\n",
       "  'abstract': 'Widely used in agriculture and various industries, synthetic phenolic compounds commonly exist in wastewater and have been troublesome in wastewater treatment due to their remarkable stability and acute toxicity. They can accumulate through food chains and serve as mutagens and carcinogens to people and other organisms. Among them, halogenated phenolic compounds are notablely more toxic and less bio-degradable. Even when the number of halogen atoms increases, the toxicity of the whole molecule is raised as well . To deal with this serious problem, our team aims to construct a Hybrid Membrane Bioreactor to assist the degradation of halogenated phenolic compounds in wastewater. In this device, we intend to apply an engineered Bacillus megaterium which can mainly express an original reductive dehalogenase RdhANP. And we will also make some genetic changes on this B. megaterium to scale up its efficiency in treating the wastewater containing halogenated phenols.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'sivel'},\n",
       " '2333': {'team_id': '2333',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2333',\n",
       "  'team_name': 'William_and_Mary',\n",
       "  'wiki': 'http://2017.igem.org/Team:William_and_Mary',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Foundational Advance',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Modular Control of Gene Expression Speed using Protein Degradation Tags\\n',\n",
       "  'abstract': 'It has become apparent that a fundamental principle of cellular signal processing is the encoding of information within the temporal dynamics of regulatory circuits. For synthetic circuits to achieve the versatility and effectiveness of endogenous circuits, it is necessary to develop simple, effective methods to control a circuit\\x92s speed. Current approaches to speed control are too complex for widespread use\\x97 we lack a modular system that allows one to \\x93swap out\\x94 a sequence to predictably change a gene\\x92s speed just as swapping an RBS can change the gene\\x92s expression strength. To address this need, we developed a BioBrick suite of degradation tags associated with an E. coli-orthogonal protease, providing the parts and characterization necessary for controlling genetic circuit speed in a simple, modular, and predictable way. Additionally, we created a searchable database of previous iGEM teams\\x92 outreach projects, designed to better enable future teams to build upon previous outreach efforts.\\r\\n\\n',\n",
       "  'award': 'Nominated for Best Foundational Advance Project;Nominated for Best Presentation;Nominated for Best Poster;Nominated for Best Education & Public Engagement;Best Model;Nominated for Best Model;Best Measurement;Nominated for Best Measurement;Nominated for Best New Composite Part;Nominated for Best Part Collection;Finalist;Nominated for Finalist',\n",
       "  'medal': 'gold'},\n",
       " '2452': {'team_id': '2452',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2452',\n",
       "  'team_name': 'WLC-Milwaukee',\n",
       "  'wiki': 'http://2017.igem.org/Team:WLC-Milwaukee',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Phage Gauge: A novel method of detecting Escherichia coli in  contaminated water. \\n',\n",
       "  'abstract': 'Escherichia coli infection is a common problem when water resources such as drinking water, beaches, or irrigation systems are contaminated with human waste. Infections by these bacteria can lead to severe gastrointestinal problems and even death if left untreated. Therefore, it is necessary to have water testing methods that can rapidly and accurately determine whether water sources contain these harmful pathogens.\\r\\n     The WLC-iGEM team is working on developing a water testing kit that provides fast and accurate results using components of the Lambda Phage tail that are specific for their target bacteria. These bacteriophage tail components have been cloned and will be purified. By attaching a horseradish peroxidase enzyme (that will produce color when a substrate is added) to these bacteriophage components, E. coli should be able to be detected when present in a water sample. The high specificity and high concentration of bacteriophages should provide accurate and rapid results.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2513': {'team_id': '2513',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2513',\n",
       "  'team_name': 'Worldshaper-Nanjing',\n",
       "  'wiki': 'http://2017.igem.org/Team:Worldshaper-Nanjing',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Self-sinking Algae for CO2 Sequestration\\n',\n",
       "  'abstract': \"Increasing level of carbon dioxide in earth's atmosphere is the main reason of global warming. Thus, it has become important to slow down the accumulation rate or reduce the amount of carbon dioxide in atmosphere. Here, we hope to develop an efficient biological carbon dioxide sequestration system using Synchronous sp. PCC 7002. The capability of CO2 capturing and storage of the algae was improved by inserting foreign genes, encoding rubisco, and starch synthase. The modified strain was also designed to have an expression of a metal binding protein on its pilus to increase weight and deactivate pilus slowly, which would finally cause the alga to sink to the sea bottom permanently so as to cut off the carbon from being reused.\\n\",\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2514': {'team_id': '2514',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2514',\n",
       "  'team_name': 'Worldshaper-Wuhan',\n",
       "  'wiki': 'http://2017.igem.org/Team:Worldshaper-Wuhan',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'miR-21 Sponge in Colorectal Cancer for Diagnosis and Treatment\\n',\n",
       "  'abstract': 'Colorectal cancer is one of the most common cancers in the world. The current method for early diagnosis of colorectal cancer remains as a big limitation. The current options for treating colorectal cancer include chemotherapy, which attack all fast-growing cells. MicroRNAs are noncoding single strand small RNA molecules which act as regulators of gene expression and control many cellular processes. Recently, miR-21 is reported to have high sensitivity and specificity in identifying colorectal cancer. Meanwhile, miR-21 can be served as a therapeutic target by targeting many oncogenes in colorectal cancer. That had inspired us of using it as the new biomarker in diagnosing and treatment of colorectal cancer. Our project constructed a mir-21 sponge containing marker gene of GFP and luc for the detection and inhibition of mir-21 in cell lines, which offers a non-invasive and highly sensitive approach for early diagnosis and treatment of colorectal cancer in the future.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2512': {'team_id': '2512',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2512',\n",
       "  'team_name': 'Worldshaper-XSHS',\n",
       "  'wiki': 'http://2017.igem.org/Team:Worldshaper-XSHS',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'High School',\n",
       "  'kind': 'High School',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'High_School\\n',\n",
       "  'title': 'Portable low-cost bio-detector for dissolved oxygen, phosphorus or nitrogen in water\\n',\n",
       "  'abstract': 'Hometown Hangzhou is widely known by water-related UNESCO World Heritages \\x96 West Lake and Grand Canal, however, severe water pollution problem caused by many reasons also exist. Hence, we hope to provide an easy-operating and low-cost tool for public to monitor water quality around. \\r\\nWe designed a portable water quality bio-detector prototype based on E.coli strains which were constructed to detect dissolved oxygen, phosphorus or nitrogen in water respectively. The oxygen sensitive vgb promoter and a GFP reporter constitute Oxygen detector 1.0. To enhance the expression of GFP, version 2.0 contains a vgb promoter, a T7 RNA polymerase gene, a T7 promoter and a GFP gene. For nitrogen, the PyeaR promoter was used to response to different concentrations of nitrate, nitrite and nitric, with a BFP reporter gene. For phosphate, a plasmid consisting of an \\x91external phosphate sensing promoter\\x92 to sense the phosphate concentration and a RFP gene to report. \\r\\n\\n',\n",
       "  'award': 'Nominated for Best New Composite Part, High_school',\n",
       "  'medal': 'gold'},\n",
       " '2413': {'team_id': '2413',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2413',\n",
       "  'team_name': 'WPI_Worcester',\n",
       "  'wiki': 'http://2017.igem.org/Team:WPI_Worcester',\n",
       "  'region': 'North America',\n",
       "  'country': 'United States',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Go(a)t Lead? Bacterial Detection and Bioremediation of Lead Contamination in Drinking Water\\n',\n",
       "  'abstract': 'Lead contamination in drinking water is a major problem across the U.S. Our project aims to improve lead testing and treatment by developing a lead biosensor and colorimetric lead assay, as well as a lead-binding probiotic. Our lead biosensor improves the cost and efficiency of lead testing by producing specific chromoproteins that indicate benchmark levels of lead contamination in a sample. These benchmarks were confirmed by our assay. We are conducting proof-of-principle demonstrations of the biosensor in Escherichia coli, and will ultimately transform it into the Generally Recognized As Safe organism Bacillus subtilis. The probiotic, Lactobacillus rhamnosus, offers an emergency prophylactic solution for treatment by absorbing lead from the gastrointestinal tract after consumption of contaminated water. To achieve this, we are using selective pressure to evolve a probiotic with enhanced lead-binding capacity. Our project will improve the accessibility of lead detection and bioremediation for the general population.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2309': {'team_id': '2309',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2309',\n",
       "  'team_name': 'XJTLU-CHINA',\n",
       "  'wiki': 'http://2017.igem.org/Team:XJTLU-CHINA',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Therapeutics',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'Grenadier Guards \\x97\\x97 using antimicrobial peptides to fight against Staphylococcus aureus\\n',\n",
       "  'abstract': 'Staphylococcus aureus develops resistance to various antibiotics and becomes increasingly difficult to be eliminated due to antibiotic overuse. Patients with intestinal S. aureus colonization tend to suffer from fecal incontinence, diarrhea and so on. Hence, our project aims to make the most of antimicrobial peptides (AMPs), which have a reputation for their efficient antimicrobial activity against bacteria, as new therapeutics. We genetically engineer Lactococcus lactis (Grenadier Guards) to closely detect the presence of S. aureus and throw AMPs (grenades) to eradicate them within a short time after infection. The mechanism includes utilizing the S. aureus\\x92s own quorum sensing system as a sensor, or using the cells\\x92 osmoregulatory system to control the expression of the AMP genes in tandem repeats, which can produce a relatively high quantity level, subsequently Lactococcus lactis will undergo autolysis controlled by a toggle switch to release AMPs to attack S. aureus, thus relieving the symptoms.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2310': {'team_id': '2310',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2310',\n",
       "  'team_name': 'XMU-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:XMU-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Environment',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'A chip based device for sensitive and in situ detection of several contaminants in water\\n',\n",
       "  'abstract': 'With the increasing amount of industrial wasted water being discharged into water areas, various kinds of harmful ions have had a great impact on the ecological environment and daily drinking water, but the degree of detection of these ions is limited in some special institutions and instruments, which cannot be portable, real-time and general. Thus, we plan to develop a miniature instrument to achieve these goals. \\r\\nTo achieve a trace detection, we designed a genetic system which can amplify electrochemical signals or optical signals. At the same time, we designed a device based on micro-fluid chip to concentrate our engineered bacteria on improving the sensitivity of detection following the principles of modularity, cheapness and easy operation.\\r\\nLast but not least, our system provides an innovative and satisfying platform of detection for other molecules by simply changing corresponding promoters.\\r\\n\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2489': {'team_id': '2489',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2489',\n",
       "  'team_name': 'York',\n",
       "  'wiki': 'http://2017.igem.org/Team:York',\n",
       "  'region': 'Europe',\n",
       "  'country': 'United Kingdom',\n",
       "  'track': 'Hardware',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Overgraduate\\n',\n",
       "  'title': 'QWACC: a Quicker Way to Analyse Co-Cultures\\n',\n",
       "  'abstract': 'Co-culturing of microorganisms is an extremely promising approach, in Biology, for understanding natural and synthetic cell population interactions, and for applications in Industrial Biotechnology including manufacturing and drug research. However, the maintenance of stable and productive co-cultures is technically challenging, expensive, and can be time consuming. We aim to develop a Digital In-line Holographic Microscope (DIHM), along with associated software, that will be able to monitor co-culture counts in a closed loop, in real time, and inexpensively. To complement the microscope, we will also develop a synthetic co-culture. This comprises Chlamydomonas reinhardtii and Escherichia coli, two visually distinct organisms. Chlamydomonas will be engineered to export maltose to feed E. coli, creating a sustainable production platform. We believe that our DIHM and co-culture is a promising start to streamlining the development of co-cultures for industrial applications.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'},\n",
       " '2207': {'team_id': '2207',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2207',\n",
       "  'team_name': 'ZJU-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:ZJU-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'New Application',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'The Guardian Trichoderma\\n',\n",
       "  'abstract': ' Trichoderma is a genus of fungi that has been used as biocontrol agents for many years.Although these species have a considerable inherent ability of antagonizing phytopathogen,like mycoparasitism,bacteriolysis,antibiotic production etc., the practical application is still limited.Our project has tried to solve this problem in a synthetic biological way. Trichoderma.atroviride is our selected guardian,we use it as our chassis, making it can respond to specified volatile organic compounds(VOC) and then synthesize new VOC to realize the communication between guardians just like QS in bacteria.We further build a hardware to detect the healthy condition of plants based on machine learning. We also express ferritin and TRPV1 proteins to try to make the guardians respond to medium waves emitted by the hardware when it estimates the plants are attacked, and then our guardians will protect the plants from phytopathogen,nematodes,or even herbivores.We use tobaccos and its nature enemy Phytopathora nicotianae to elaborate our works.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'gold'},\n",
       " '2277': {'team_id': '2277',\n",
       "  'team_url': 'http://igem.org/Team.cgi?team_id=2277',\n",
       "  'team_name': 'ZJUT-China',\n",
       "  'wiki': 'http://2017.igem.org/Team:ZJUT-China',\n",
       "  'region': 'Asia',\n",
       "  'country': 'China',\n",
       "  'track': 'Manufacturing',\n",
       "  'kind': 'Collegiate',\n",
       "  'application': 'Accepted',\n",
       "  'section': 'Undergraduate\\n',\n",
       "  'title': 'LiGeM: A Light-induced Genetically Engineered Machine for Cell Disruption\\n',\n",
       "  'abstract': 'In fermentation industry, numerous valuable products are intracellular metabolites. Thus, the cell disruption step is essential for product extraction. However, traditional cell disruption methods (including sonication and homogenization) usually need additional equipments and are laborious/time-consuming. To improve the efficiency of cell disruption and intracellular products extraction, a lysozyme-encoding gene was successfully integrated into a modified blue-light-controlled gene switch which was evaluated through eGFP gene expression. Subsequently, self-lysis of E. coli cells was assessed under blue light. To control the expression of lysis gene regulated by blue light, a model of light-emitting device was constructed and a layout of fermentor with light-emitting device was designed. Moreover, we made an economic analysis after consulting several bio-factories. It indicated that our design not only reduce the cost in cell disruption but also simplify operation process. Taken together, our project showed an exciting potential for cell disruption and intracellular products extraction in fermentation industry.\\n',\n",
       "  'award': '',\n",
       "  'medal': 'bronze'}}"
      ]
     },
     "execution_count": 23,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "team_dict"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "metadata": {},
   "outputs": [],
   "source": [
    "json.dump(team_dict,open('team_2017.json','w'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
